<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Nicholas C Wu">
<meta name="author" content="Tomás Villada-Cadavid">
<meta name="author" content="Christopher Turbill">

<title>Supplementary Information</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="supplementary_information_files/libs/clipboard/clipboard.min.js"></script>
<script src="supplementary_information_files/libs/quarto-html/quarto.js"></script>
<script src="supplementary_information_files/libs/quarto-html/popper.min.js"></script>
<script src="supplementary_information_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="supplementary_information_files/libs/quarto-html/anchor.min.js"></script>
<link href="supplementary_information_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="supplementary_information_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="supplementary_information_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="supplementary_information_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="supplementary_information_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#supplementary-methods" id="toc-supplementary-methods" class="nav-link active" data-scroll-target="#supplementary-methods">Supplementary Methods</a>
  <ul class="collapse">
  <li><a href="#respirometry-set-up" id="toc-respirometry-set-up" class="nav-link" data-scroll-target="#respirometry-set-up">Respirometry set-up</a></li>
  <li><a href="#calculations-and-conversation" id="toc-calculations-and-conversation" class="nav-link" data-scroll-target="#calculations-and-conversation">Calculations and conversation</a></li>
  <li><a href="#quantifying-pd" id="toc-quantifying-pd" class="nav-link" data-scroll-target="#quantifying-pd">Quantifying <em>Pd</em></a></li>
  <li><a href="#cave-microclimate" id="toc-cave-microclimate" class="nav-link" data-scroll-target="#cave-microclimate">Cave microclimate</a></li>
  </ul></li>
  <li><a href="#process-data" id="toc-process-data" class="nav-link" data-scroll-target="#process-data">Process Data</a>
  <ul class="collapse">
  <li><a href="#load-and-clean-data" id="toc-load-and-clean-data" class="nav-link" data-scroll-target="#load-and-clean-data">Load and clean data</a></li>
  <li><a href="#snps-and-genetic-relatedness" id="toc-snps-and-genetic-relatedness" class="nav-link" data-scroll-target="#snps-and-genetic-relatedness">SNPs and genetic relatedness</a></li>
  <li><a href="#prepare-tree-for-analysis" id="toc-prepare-tree-for-analysis" class="nav-link" data-scroll-target="#prepare-tree-for-analysis">Prepare tree for analysis</a></li>
  </ul></li>
  <li><a href="#analysis" id="toc-analysis" class="nav-link" data-scroll-target="#analysis">Analysis</a>
  <ul class="collapse">
  <li><a href="#model-output" id="toc-model-output" class="nav-link" data-scroll-target="#model-output">Model output</a></li>
  </ul></li>
  <li><a href="#figures" id="toc-figures" class="nav-link" data-scroll-target="#figures">Figures</a>
  <ul class="collapse">
  <li><a href="#figure-1---study-sites" id="toc-figure-1---study-sites" class="nav-link" data-scroll-target="#figure-1---study-sites">Figure 1 - Study sites</a></li>
  <li><a href="#figure-2---tsk-tmr-and-tewl" id="toc-figure-2---tsk-tmr-and-tewl" class="nav-link" data-scroll-target="#figure-2---tsk-tmr-and-tewl">Figure 2 - <em>T</em><sub>sk</sub>, TMR, and TEWL</a></li>
  <li><a href="#figure-3---phylogeny" id="toc-figure-3---phylogeny" class="nav-link" data-scroll-target="#figure-3---phylogeny">Figure 3 - Phylogeny</a></li>
  </ul></li>
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a>
  <ul class="collapse">
  <li><a href="#session-information" id="toc-session-information" class="nav-link" data-scroll-target="#session-information">Session Information</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<div class="quarto-title-block"><div><h1 class="title">Supplementary Information</h1><button type="button" class="btn code-tools-button dropdown-toggle" id="quarto-code-tools-menu" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi"></i> Code</button><ul class="dropdown-menu dropdown-menu-end" aria-labelelledby="quarto-code-tools-menu"><li><a id="quarto-show-all-code" class="dropdown-item" href="javascript:void(0)" role="button">Show All Code</a></li><li><a id="quarto-hide-all-code" class="dropdown-item" href="javascript:void(0)" role="button">Hide All Code</a></li><li><hr class="dropdown-divider"></li><li><a id="quarto-view-source" class="dropdown-item" href="javascript:void(0)" role="button">View Source</a></li></ul></div></div>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Authors</div>
    <div class="quarto-title-meta-contents">
             <p>Nicholas C Wu <a href="mailto:nicholas.wu.nz@gmail.com" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0002-7130-1279" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
             <p>Tomás Villada-Cadavid <a href="mailto:t.cadavid@westernsydney.edu.au" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0002-1743-0694" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
             <p>Christopher Turbill <a href="mailto:c.turbill@westernsydney.edu.au" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-9810-7102" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">25th May 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<p><a href="https://github.com/nicholaswunz/bat-resp">GitHub Repository</a>: The Quarto code to produce the HTML file can be downloaded from the <em>Code</em> drop-down menu (top right).</p>
<p>This supplementary file contains the <em>R</em> workflow for processing and analysing the raw data, and creating figures for the manuscript titled “<strong>Local climate and genetic influence on intraspecific patterns in torpor physiology</strong>”.</p>
<hr>
<section id="supplementary-methods" class="level1">
<h1>Supplementary Methods</h1>
<section id="respirometry-set-up" class="level2">
<h2 class="anchored" data-anchor-id="respirometry-set-up">Respirometry set-up</h2>
<p><img src="Fig%20S1%20-%20Respirometry%20setup.png" class="img-fluid"></p>
<p><strong>Fig. S1</strong> | Schematic diagram of the flow-through respirometry set up to measure oxygen consumption, carbon dioxide production, and evaporative water loss of seven torpored <em>Miniopterus orianae oceanensis</em> and one control blank chamber. The mass flow controllers were placed outside the temperature-controlled incubator to change the flow rate during the experiment manually.</p>
</section>
<section id="calculations-and-conversation" class="level2">
<h2 class="anchored" data-anchor-id="calculations-and-conversation">Calculations and conversation</h2>
<p>The rate of oxygen consumption (<span class="math inline">\(\dot{V}\textrm{O}_2\)</span>; ml O<sub>2</sub> min<sup>-1</sup>) was calculated following <span class="citation" data-cites="Lighton2019">Lighton (<a href="#ref-Lighton2019" role="doc-biblioref">2019</a>)</span> in <a href="#eq-vo2" class="quarto-xref">Equation&nbsp;1</a>:</p>
<p><span id="eq-vo2"><span class="math display">\[
\dot{V}\textrm{O}_2 = \frac{\textrm{FR}^\textrm{'}_{\textrm{i}}[(F_{\textrm{i}}\textrm{O}_{2}-F_{\textrm{e}}\textrm{O}_{2})-F_{\textrm{e}}\textrm{O}_{2}(F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}-F^\textrm{'}_{\textrm{i}}\textrm{CO}_{2})]}{(1-F_{\textrm{e}}\textrm{O}_{2})}
\tag{1}\]</span></span></p>
<p>where <span class="math inline">\(\textrm{FR}^\textrm{'}_{\textrm{i}}\)</span> is the incurrent dry flow rate corrected for water vapour (<span class="math inline">\(\textrm{FR}^\textrm{'}_{\textrm{i}} = \textrm{FR}_{\textrm{i}} \times \textrm{BP}/(\textrm{BP}-\textrm{WVP})\)</span>). <span class="math inline">\(F_{\textrm{i}}\textrm{O}_{2}\)</span> and <span class="math inline">\(F_{\textrm{e}}\textrm{O}_{2}\)</span> is the dry incurrent (baseline) and excurrent O<sub>2</sub> gas concentration, respectively. <span class="math inline">\(F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}\)</span> and <span class="math inline">\(F^\textrm{'}_{\textrm{i}}\textrm{CO}_{2}\)</span> is the dry” equivalent of the excurrent and incurrent CO2 gas concentration corrected for water vapour, respectively. The rate of carbon dioxide production (<span class="math inline">\(\dot{V}\textrm{CO}_2\)</span>; ml CO<sub>2</sub> min<sup>-1</sup>) was calculated as <a href="#eq-vco2" class="quarto-xref">Equation&nbsp;2</a>:</p>
<p><span id="eq-vco2"><span class="math display">\[
\dot{V}\textrm{CO}_2 = \frac{\textrm{FR}^\textrm{'}_{\textrm{i}}[(F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}-F^\textrm{'}_{\textrm{i}}\textrm{CO}_{2})-F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}(F_{\textrm{i}}\textrm{O}_{2}-F_{\textrm{e}}\textrm{O}_{2})]}{(1-F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2})}
\tag{2}\]</span></span></p>
<p>Metabolic rate (MR; kJ h<sup>-1</sup>) was calculated from <span class="math inline">\(\dot{V}\textrm{O}_2\)</span> and <span class="math inline">\(\dot{V}\textrm{CO}_2\)</span> as <a href="#eq-mr" class="quarto-xref">Equation&nbsp;3</a>:</p>
<p><span id="eq-mr"><span class="math display">\[
\textrm{MR} = 0.06 \times (3.941 \times \dot{V}\textrm{O}_2 + 1.106 \times \dot{V}\textrm{CO}_2) \times 4.184
\tag{3}\]</span></span></p>
<p>The evaporative water loss (EWL) or <span class="math inline">\(\dot{V}\textrm{H}_2\textrm{O}\)</span> (µg H2O min<sup>-1</sup>) was calculated as <a href="#eq-ewl" class="quarto-xref">Equation&nbsp;4</a>:</p>
<p><span id="eq-ewl"><span class="math display">\[
\dot{V}\textrm{H}_2\textrm{O} = \textrm{FR}_{\textrm{i}} \left( \frac{\textrm{WVP}}{T \times R_{\textrm{w}}} \right)
\tag{4}\]</span></span></p>
<p>where WVP is the water vapour pressure (kPa) minus the baseline WVP, <span class="math inline">\(T\)</span> is temperature in Kelvin (°C + 273.15), and <span class="math inline">\(R_{\textrm{w}}\)</span> is the gas constant for water vapour (461.5 J kg<sup>−1</sup> K<sup>−1</sup>). <span class="math inline">\(\dot{V}\textrm{H}_2\textrm{O}\)</span> was converted to mg H<sub>2</sub>O h<sup>-1</sup> by multiplying by 0.06. Prior to calculating <span class="math inline">\(\dot{V}\textrm{O}_2\)</span>, <span class="math inline">\(\dot{V}\textrm{CO}_2\)</span>, and <span class="math inline">\(\dot{V}\textrm{H}_2\textrm{O}\)</span>, the baseline drift and time lag of the fractional gas concentrations were corrected for.</p>
</section>
<section id="quantifying-pd" class="level2">
<h2 class="anchored" data-anchor-id="quantifying-pd">Quantifying <em>Pd</em></h2>
<p>To finalise with Anna</p>
</section>
<section id="cave-microclimate" class="level2">
<h2 class="anchored" data-anchor-id="cave-microclimate">Cave microclimate</h2>
<p>The local air temperature and relative humidity outside and inside the cave was recorded to monitor differences in microclimate conditions available for cave-roosting bats. An external HOBO logger (MX2302A; Onset, MA, USA) mounted with a radiation shield (RS3-B; Onset) was placed outside ~20 m away at 1.5 m height from the cave entrance to log external microclimate at 30 second intervals. An internal HOBO logger (MX2301A; Onset) was placed inside the cave above ground in the same chamber where bats were observed roosting to log cave microclimate at 30 second intervals. Cave surface temperature and relative humidity was measured with an infrared thermometer (TG56 IR 30:1; FLIR, OR, USA) and HOBO logger on an 8 m extendable pole (MX2302A) every 10 m inside the cave or where bats were found. Cave surface temperature were measured in three replicates at three time points, May, July, and September. Vapour pressure deficit (kPa) was calculated as the difference between the saturation water vapour pressure (<span class="math inline">\(e_s\)</span>) and the actual water vapour pressure (<span class="math inline">\(e_a\)</span>). <span class="math inline">\(e_s\)</span> was calculated following the Clausius-Clapeyron equation <span class="citation" data-cites="Stull2000">(<a href="#ref-Stull2000" role="doc-biblioref">Stull, 2000</a>)</span>.</p>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="supplementary_information_files/figure-html/Fig%20S2-1.png" class="img-fluid quarto-figure quarto-figure-center figure-img" width="864"></p>
</figure>
</div>
</div>
</div>
<p><strong>Fig. S2</strong> | Distribution of measured (<strong>a</strong>) cave surface temperatures (°C), (<strong>b</strong>) cave relative humidity (RH; %), and (<strong>c</strong>) calculated cave vapour pressure deficit (VPD; kPa) from Jenolan and Yessabah during the 2023 Austral winter period (May–September). Surface temperature, RH, and VPD with bats present were coloured <span style="color:#000000;"><strong>black</strong></span>, while surface temperature, RH, and VPD without bats present were coloured <span style="color:#999999;"><strong>grey</strong></span>. Dashed lines represented the experimental exposure temperatures which are within the expected temperature range in the caves.</p>
</section>
</section>
<section id="process-data" class="level1">
<h1>Process Data</h1>
<p>Metadata of the imported respirometry data for analysis.</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 13%">
<col style="width: 86%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Name</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">pop</td>
<td style="text-align: left;">Unique identifier of population source (Jenolan, Yessabah).</td>
</tr>
<tr class="even">
<td style="text-align: left;">date_exp</td>
<td style="text-align: left;">The date when the experiment was conducted in day/month/year</td>
</tr>
<tr class="odd">
<td style="text-align: left;">ID</td>
<td style="text-align: left;">Unique identifier of the individual.</td>
</tr>
<tr class="even">
<td style="text-align: left;">ID_gene</td>
<td style="text-align: left;">Unique identifier of the biopsy sample for genetic analysis</td>
</tr>
<tr class="odd">
<td style="text-align: left;">sex</td>
<td style="text-align: left;">Sex of the individual (female, male)</td>
</tr>
<tr class="even">
<td style="text-align: left;">FA_mm</td>
<td style="text-align: left;">Forearm length in millimetre (mm).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">mass_g</td>
<td style="text-align: left;">Body mass in grams (g).</td>
</tr>
<tr class="even">
<td style="text-align: left;">temp_exp</td>
<td style="text-align: left;">Set air temperature exposure in degrees Celcius (°C).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">temp_act</td>
<td style="text-align: left;">Recorded air temperature in degrees Celcius (°C).</td>
</tr>
<tr class="even">
<td style="text-align: left;">RH_treatment</td>
<td style="text-align: left;">Recorded relative humidity in percentage (%).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">chamber_n</td>
<td style="text-align: left;">Unique identifier of the metabolic chamber</td>
</tr>
<tr class="even">
<td style="text-align: left;">setFR_ml_min</td>
<td style="text-align: left;">Set flow rate (ml/min).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">BP_kPa</td>
<td style="text-align: left;">Recorded barometric pressure (kPa).</td>
</tr>
<tr class="even">
<td style="text-align: left;">FR_adj_ml_min</td>
<td style="text-align: left;">Adjusted-flow rate (ml/min).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">T_start</td>
<td style="text-align: left;">Start time of the respirometry measurments as year-month-day hour:min:sec.</td>
</tr>
<tr class="even">
<td style="text-align: left;">T_end</td>
<td style="text-align: left;">End time of the respirometry measurments as year-month-day hour:min:sec.</td>
</tr>
<tr class="odd">
<td style="text-align: left;">t_tor_h</td>
<td style="text-align: left;">Time between onset of torpor and respirometry measurments.</td>
</tr>
<tr class="even">
<td style="text-align: left;">VO2_ml_min</td>
<td style="text-align: left;">Calculated rate of oxygen consumption in millilitres per minute (ml/min).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">VCO2_ml_h</td>
<td style="text-align: left;">Calculated rate of carbon dioxide production in millilitres per minute (ml/min).</td>
</tr>
<tr class="even">
<td style="text-align: left;">EWL_mg_h</td>
<td style="text-align: left;">Calculated rate of evaporative water loss in milligrams per minute (mg/h).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Tsk_C</td>
<td style="text-align: left;">Calculated skin surface temperature in degrees Celcius (°C).</td>
</tr>
<tr class="even">
<td style="text-align: left;">VO2_ml_h</td>
<td style="text-align: left;">Calculated rate of oxygen consumption in millilitres per hour (ml/h).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">VCO2_ml_h</td>
<td style="text-align: left;">Calculated rate of carbon dioxide production in millilitres per hour (ml/h).</td>
</tr>
<tr class="even">
<td style="text-align: left;">VO2_ml_g_h</td>
<td style="text-align: left;">Mass-specific oxygen consumption in ml/g/h.</td>
</tr>
<tr class="odd">
<td style="text-align: left;">VCO2_ml_g_h</td>
<td style="text-align: left;">Mass-specific carbon dioxide production in ml/g/h.</td>
</tr>
<tr class="even">
<td style="text-align: left;">EWL_mg_g_h</td>
<td style="text-align: left;">Mass-specific evaporative water loss in mg/g/h.</td>
</tr>
<tr class="odd">
<td style="text-align: left;">lnMass</td>
<td style="text-align: left;">The natural logarithm of body mass.</td>
</tr>
<tr class="even">
<td style="text-align: left;">MR_kJ_h</td>
<td style="text-align: left;">Calculated metabolic rate in kilojoules per hour (kJ/h).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">MR_J_h</td>
<td style="text-align: left;">Calculated metabolic rate in joules per hour (J/h).</td>
</tr>
<tr class="even">
<td style="text-align: left;">MR_W</td>
<td style="text-align: left;">Calculated metabolic rate in Watts (W).</td>
</tr>
<tr class="odd">
<td style="text-align: left;">MR_mW</td>
<td style="text-align: left;">Calculated metabolic rate in milliWatts (mW).</td>
</tr>
<tr class="even">
<td style="text-align: left;">t_diff</td>
<td style="text-align: left;">Difference between skin temperature and actual air temperature in degrees Celcius (°C).</td>
</tr>
</tbody>
</table>
</div>
</div>
<section id="load-and-clean-data" class="level2">
<h2 class="anchored" data-anchor-id="load-and-clean-data">Load and clean data</h2>
<p>Load and clean the raw data for analysis. <strong>Table S1</strong> summarises data on torpor physiology traits by site, sex, and temperature exposure.</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Respirometry data</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>resp_dat <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="fu">file.path</span>(data_path, <span class="st">"data/resp_raw_dat.csv"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="sc">-</span><span class="fu">c</span>(file_name, FR_ml_min, transmitter_n, freq_hz)) <span class="sc">%&gt;%</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(cond <span class="sc">==</span> <span class="st">"dry"</span> <span class="sc">&amp;</span> ID <span class="sc">!=</span> <span class="fu">c</span>(<span class="st">"MAM-18"</span>, <span class="st">"MAM-16"</span>)) <span class="sc">%&gt;%</span> <span class="co"># remove individuals not in torpor</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">pop         =</span> <span class="fu">factor</span>(pop),</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a>                <span class="at">VO2_ml_h    =</span> VO2_ml_min <span class="sc">*</span> <span class="dv">60</span>,</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a>                <span class="at">VCO2_ml_h   =</span> VCO2_ml_min <span class="sc">*</span> <span class="dv">60</span>,</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>                <span class="at">VO2_ml_g_h  =</span> VO2_ml_h <span class="sc">/</span> mass_g,</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a>                <span class="at">VCO2_ml_g_h =</span> VCO2_ml_h <span class="sc">/</span> mass_g,</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a>                <span class="at">EWL_mg_g_h  =</span> EWL_mg_h <span class="sc">/</span> mass_g,</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a>                <span class="at">lnMass      =</span> <span class="fu">log</span>(mass_g),</span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>                <span class="at">T_start     =</span> lubridate<span class="sc">::</span><span class="fu">dmy_hm</span>(T_start),</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a>                <span class="at">T_end       =</span> lubridate<span class="sc">::</span><span class="fu">dmy_hm</span>(T_end),</span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a>                <span class="at">MR_kJ_h     =</span> (<span class="fl">0.06</span> <span class="sc">*</span> (<span class="fl">3.941</span> <span class="sc">*</span> VO2_ml_min <span class="sc">+</span> <span class="fl">1.106</span> <span class="sc">*</span> VCO2_ml_min)) <span class="sc">*</span> <span class="fl">4.184</span>, <span class="co"># Convert VO2 and VCO2 to MR kJ h</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>                <span class="at">MR_J_h      =</span> MR_kJ_h <span class="sc">*</span> <span class="dv">1000</span>,</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a>                <span class="at">MR_W        =</span> MR_kJ_h <span class="sc">*</span> <span class="fl">0.2777777778</span>, <span class="co"># convert to Watt</span></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a>                <span class="at">MR_mW       =</span> MR_W <span class="sc">*</span> <span class="dv">1000</span>,</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a>                <span class="at">lnVO2       =</span> <span class="fu">log</span>(VO2_ml_h),</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a>                <span class="at">lnVCO2      =</span> <span class="fu">log</span>(VCO2_ml_h),</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a>                <span class="at">lnMR        =</span> <span class="fu">log</span>(MR_J_h),</span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a>                <span class="at">lnEWL       =</span> <span class="fu">log</span>(EWL_mg_h),</span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a>                <span class="at">t_diff      =</span> Tsk_C <span class="sc">-</span> temp_act)  </span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="co"># filter torpor data only</span></span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>torpor_dat <span class="ot">&lt;-</span> resp_dat <span class="sc">%&gt;%</span></span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(Tsk_C <span class="sc">&lt;</span> <span class="dv">28</span>)</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a>resp_sum <span class="ot">&lt;-</span> torpor_dat <span class="sc">%&gt;%</span></span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">group_by</span>(pop, temp_exp) <span class="sc">%&gt;%</span></span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">summarise</span>(<span class="at">n         =</span> <span class="fu">length</span>(VO2_ml_h),</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a>                   <span class="at">EWL_mean  =</span> <span class="fu">mean</span>(EWL_mg_h, <span class="at">na.rm =</span> T),</span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a>                   <span class="at">EWL_sd    =</span> <span class="fu">sd</span>(EWL_mg_h, <span class="at">na.rm =</span> T),</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a>                   <span class="at">Tsk_mean  =</span> <span class="fu">mean</span>(Tsk_C, <span class="at">na.rm =</span> T),</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a>                   <span class="at">Tsk_sd    =</span> <span class="fu">sd</span>(Tsk_C, <span class="at">na.rm =</span> T),</span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a>                   <span class="at">MR_mean   =</span> <span class="fu">mean</span>(MR_kJ_h, <span class="at">na.rm =</span> T),</span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a>                   <span class="at">MR_sd     =</span> <span class="fu">sd</span>(MR_kJ_h, <span class="at">na.rm =</span> T)</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a>                   )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p><strong>Table S1</strong> | Summary data from the respirometry experiment. Sample size (<em>n</em>), mean body mass (g), mean ± standard deviation rate of oxygen consumption (<span class="math inline">\(\dot{V}\textrm{O}_2\)</span>, ml h<sup>-1</sup>), rate of carbon dioxide production (<span class="math inline">\(\dot{V}\textrm{CO}_2\)</span>, ml h<sup>-1</sup>), torpor metabolic rate (TMR, J h<sup>-1</sup>), torpor evaporative water loss (TEWL, mg h<sup>-1</sup>), the skin surface temperature (<em>T</em><sub>sk</sub>, °C), and <em>Pd</em> detection via qPCR by sex, air temperature exposure (<em>T</em><sub>a</sub>, °C), and sites.</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<colgroup>
<col style="width: 5%">
<col style="width: 7%">
<col style="width: 2%">
<col style="width: 4%">
<col style="width: 5%">
<col style="width: 21%">
<col style="width: 21%">
<col style="width: 8%">
<col style="width: 10%">
<col style="width: 8%">
<col style="width: 3%">
</colgroup>
<thead>
<tr class="header">
<th style="text-align: left;">Site</th>
<th style="text-align: right;"><em>T</em><sub>a</sub> (°C)</th>
<th style="text-align: right;"><em>n</em></th>
<th style="text-align: left;">Sex</th>
<th style="text-align: left;">Mass (g)</th>
<th style="text-align: left;"><span class="math inline">\(\dot{V}\textrm{O}_2\)</span> (ml h<sup>-1</sup>)</th>
<th style="text-align: left;"><span class="math inline">\(\dot{V}\textrm{CO}_2\)</span> (ml h<sup>-1</sup>)</th>
<th style="text-align: left;">TMR (J h<sup>-1</sup>)</th>
<th style="text-align: left;">TEWL (mg h<sup>-1</sup>)</th>
<th style="text-align: left;"><em>T</em><sub>sk</sub> (°C)</th>
<th style="text-align: left;"><em>Pd</em></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">4</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">13.68</td>
<td style="text-align: left;">2.37±1.43</td>
<td style="text-align: left;">2.05±1.20</td>
<td style="text-align: left;">48.61±29.10</td>
<td style="text-align: left;">4.31±1.45</td>
<td style="text-align: left;">15.48±1.49</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">3</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">14.27</td>
<td style="text-align: left;">2.66±1.94</td>
<td style="text-align: left;">2.26±1.76</td>
<td style="text-align: left;">54.39±40.20</td>
<td style="text-align: left;">4.35±1.31</td>
<td style="text-align: left;">16.67±0.51</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">4</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">13.68</td>
<td style="text-align: left;">3.27±2.02</td>
<td style="text-align: left;">2.93±1.98</td>
<td style="text-align: left;">67.55±42.53</td>
<td style="text-align: left;">4.80±0.74</td>
<td style="text-align: left;">16.93±2.60</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">3</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">14.27</td>
<td style="text-align: left;">4.13±2.50</td>
<td style="text-align: left;">3.61±2.20</td>
<td style="text-align: left;">84.86±51.28</td>
<td style="text-align: left;">4.92±1.08</td>
<td style="text-align: left;">17.67±0.19</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">6</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">14.03</td>
<td style="text-align: left;">6.42± 6.36</td>
<td style="text-align: left;">6.08±6.13</td>
<td style="text-align: left;">133.98±133.17</td>
<td style="text-align: left;">5.01±1.04</td>
<td style="text-align: left;">18.65±2.26</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">5</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">14.40</td>
<td style="text-align: left;">13.35±10.07</td>
<td style="text-align: left;">11.89±9.86</td>
<td style="text-align: left;">275.17±211.41</td>
<td style="text-align: left;">11.46±5.31</td>
<td style="text-align: left;">20.23±3.09</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">6</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">13.62</td>
<td style="text-align: left;">19.95±18.51</td>
<td style="text-align: left;">16.52±14.75</td>
<td style="text-align: left;">405.40±373.30</td>
<td style="text-align: left;">13.75±6.54</td>
<td style="text-align: left;">21.24±2.58</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Jenolan</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">5</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">14.40</td>
<td style="text-align: left;">24.33±14.73</td>
<td style="text-align: left;">21.05±12.32</td>
<td style="text-align: left;">498.53±299.35</td>
<td style="text-align: left;">21.67±9.04</td>
<td style="text-align: left;">23.35±2.28</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">7</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">13.26</td>
<td style="text-align: left;">0.72±0.33</td>
<td style="text-align: left;">0.69±0.30</td>
<td style="text-align: left;">15.01± 6.82</td>
<td style="text-align: left;">3.37±0.67</td>
<td style="text-align: left;">14.97±0.46</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">5</td>
<td style="text-align: right;">8</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">15.19</td>
<td style="text-align: left;">1.65±0.68</td>
<td style="text-align: left;">1.54±0.63</td>
<td style="text-align: left;">34.27±14.15</td>
<td style="text-align: left;">3.79±0.31</td>
<td style="text-align: left;">15.19±0.39</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">7</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">13.26</td>
<td style="text-align: left;">1.19±0.48</td>
<td style="text-align: left;">1.15±0.49</td>
<td style="text-align: left;">24.98±10.19</td>
<td style="text-align: left;">3.83±0.49</td>
<td style="text-align: left;">16.10±0.44</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">8</td>
<td style="text-align: right;">8</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">15.19</td>
<td style="text-align: left;">2.22±1.17</td>
<td style="text-align: left;">2.07±1.13</td>
<td style="text-align: left;">46.23±24.47</td>
<td style="text-align: left;">6.31±3.68</td>
<td style="text-align: left;">16.05±0.53</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">7</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">13.26</td>
<td style="text-align: left;">1.77±0.90</td>
<td style="text-align: left;">1.67±0.84</td>
<td style="text-align: left;">36.99±18.73</td>
<td style="text-align: left;">5.31±1.84</td>
<td style="text-align: left;">17.13±0.31</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">11</td>
<td style="text-align: right;">8</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">15.19</td>
<td style="text-align: left;">2.61±1.37</td>
<td style="text-align: left;">2.43±1.35</td>
<td style="text-align: left;">54.32±28.73</td>
<td style="text-align: left;">9.42±5.70</td>
<td style="text-align: left;">17.27±0.46</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">7</td>
<td style="text-align: left;">female</td>
<td style="text-align: left;">13.26</td>
<td style="text-align: left;">3.75±2.05</td>
<td style="text-align: left;">3.40±1.74</td>
<td style="text-align: left;">77.56±41.80</td>
<td style="text-align: left;">21.54±6.12</td>
<td style="text-align: left;">20.42±1.91</td>
<td style="text-align: left;">-</td>
</tr>
<tr class="even">
<td style="text-align: left;">Yessabah</td>
<td style="text-align: right;">15</td>
<td style="text-align: right;">8</td>
<td style="text-align: left;">male</td>
<td style="text-align: left;">15.19</td>
<td style="text-align: left;">3.43±1.68</td>
<td style="text-align: left;">3.18±1.58</td>
<td style="text-align: left;">71.37±34.82</td>
<td style="text-align: left;">23.46±6.89</td>
<td style="text-align: left;">19.65±0.52</td>
<td style="text-align: left;">-</td>
</tr>
</tbody>
</table>
</div>
</div>
</section>
<section id="snps-and-genetic-relatedness" class="level2">
<h2 class="anchored" data-anchor-id="snps-and-genetic-relatedness">SNPs and genetic relatedness</h2>
<p>The following code was written with Ian Brennan to process, filter and trim the raw SNP data (containing 46,224 loci, 4,075 nucleotide sites) using the <a href="https://green-striped-gecko.github.io/dartR/"><code>dartR</code></a> package <span class="citation" data-cites="dartR2022">(<a href="#ref-dartR2022" role="doc-biblioref">Mijangos et al., 2022</a>)</span>. Note: the raw SNP data are being used as part of a larger study examining the genetic connectivity and population dynamics of Australian bent-winged bats. The SNP data will be available upon it’s publication (Planckaert <em>et al.</em>, In preparation).</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load DArT file Visually check SNP data</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>dartR.base<span class="sc">::</span><span class="fu">gl.smearplot</span>(bat_gl_wu)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="fu">nLoc</span>(bat_gl_wu)  <span class="co"># 46224 loci </span></span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a><span class="fu">nInd</span>(bat_gl_wu)  <span class="co"># 76 individuals</span></span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a><span class="do">## FILTERING ## ------------------------------------------ Check raw data</span></span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>dartR.base<span class="sc">::</span><span class="fu">gl.report.callrate</span>(bat_gl_wu)  <span class="co"># loci callrate </span></span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>dartR.base<span class="sc">::</span><span class="fu">gl.report.callrate</span>(bat_gl_wu, <span class="at">method =</span> <span class="st">"ind"</span>)  <span class="co"># individual callrate</span></span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>dartR.base<span class="sc">::</span><span class="fu">gl.report.reproducibility</span>(bat_gl_wu)  <span class="co"># reproducibility</span></span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a><span class="co"># strict threshold on repeatability</span></span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a>bat_gl2 <span class="ot">&lt;-</span> dartR.base<span class="sc">::</span><span class="fu">gl.filter.reproducibility</span>(bat_gl_wu, <span class="at">threshold =</span> <span class="fl">0.999</span>)</span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a><span class="co"># remove invariant (monomorphic) sites, we will have to do this again later,</span></span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a><span class="co"># once we've filtered down the dataset and potentially caused new monomorphs</span></span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>bat_gl3 <span class="ot">&lt;-</span> dartR.base<span class="sc">::</span><span class="fu">gl.filter.monomorphs</span>(bat_gl2)</span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a><span class="co"># adjust threshold on callrate as necessary</span></span>
<span id="cb2-20"><a href="#cb2-20" aria-hidden="true" tabindex="-1"></a>dartR.base<span class="sc">::</span><span class="fu">gl.report.callrate</span>(bat_gl3)  <span class="co"># loci callrate </span></span>
<span id="cb2-21"><a href="#cb2-21" aria-hidden="true" tabindex="-1"></a>bat_gl4 <span class="ot">&lt;-</span> dartR.base<span class="sc">::</span><span class="fu">gl.filter.callrate</span>(bat_gl3, <span class="at">method =</span> <span class="st">"loc"</span>, <span class="at">threshold =</span> <span class="fl">0.9</span>)</span>
<span id="cb2-22"><a href="#cb2-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-23"><a href="#cb2-23" aria-hidden="true" tabindex="-1"></a><span class="co"># filter out multiple SNPs from the same (linked) locus</span></span>
<span id="cb2-24"><a href="#cb2-24" aria-hidden="true" tabindex="-1"></a>bat_gl5 <span class="ot">&lt;-</span> dartR.base<span class="sc">::</span><span class="fu">gl.filter.secondaries</span>(bat_gl4, <span class="at">method =</span> <span class="st">"best"</span>, <span class="at">v =</span> <span class="dv">2</span>)</span>
<span id="cb2-25"><a href="#cb2-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-26"><a href="#cb2-26" aria-hidden="true" tabindex="-1"></a><span class="co"># filter by the minor allele frequency, I wouldn't exceed 0.05 in small</span></span>
<span id="cb2-27"><a href="#cb2-27" aria-hidden="true" tabindex="-1"></a><span class="co"># datasets you worry about losing loci, so can be ~0.04 in larger datasets</span></span>
<span id="cb2-28"><a href="#cb2-28" aria-hidden="true" tabindex="-1"></a><span class="co"># should probably be closer to ~0.01</span></span>
<span id="cb2-29"><a href="#cb2-29" aria-hidden="true" tabindex="-1"></a>bat_gl6 <span class="ot">&lt;-</span> dartR.base<span class="sc">::</span><span class="fu">gl.filter.maf</span>(bat_gl5, <span class="at">threshold =</span> (<span class="dv">2</span><span class="sc">/</span><span class="fu">nInd</span>(bat_gl5)), <span class="at">v =</span> <span class="dv">5</span>)</span>
<span id="cb2-30"><a href="#cb2-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-31"><a href="#cb2-31" aria-hidden="true" tabindex="-1"></a><span class="co"># one more pass to clean up callrate and remove monomorphs introduced</span></span>
<span id="cb2-32"><a href="#cb2-32" aria-hidden="true" tabindex="-1"></a>dartR.base<span class="sc">::</span><span class="fu">gl.report.callrate</span>(bat_gl6, <span class="at">method =</span> <span class="st">"ind"</span>)  <span class="co"># individual callrate</span></span>
<span id="cb2-33"><a href="#cb2-33" aria-hidden="true" tabindex="-1"></a>bat_gl7 <span class="ot">&lt;-</span> dartR.base<span class="sc">::</span><span class="fu">gl.filter.callrate</span>(bat_gl6, <span class="at">method =</span> <span class="st">"ind"</span>, <span class="at">threshold =</span> <span class="fl">0.95</span>,</span>
<span id="cb2-34"><a href="#cb2-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">mono.rm =</span> T, <span class="at">recalc =</span> T, <span class="at">recursive =</span> T, <span class="at">v =</span> <span class="dv">5</span>)</span>
<span id="cb2-35"><a href="#cb2-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-36"><a href="#cb2-36" aria-hidden="true" tabindex="-1"></a><span class="co"># Export a Concatenated SNP Alignment Export with amiguities (A/G = R; C/T=Y)</span></span>
<span id="cb2-37"><a href="#cb2-37" aria-hidden="true" tabindex="-1"></a><span class="co"># for heterozygous sites</span></span>
<span id="cb2-38"><a href="#cb2-38" aria-hidden="true" tabindex="-1"></a>dartR.base<span class="sc">::</span><span class="fu">gl2fasta</span>(bat_gl7, <span class="at">method =</span> <span class="dv">4</span>, <span class="at">outfile =</span> <span class="st">"bat_gl7_mth4.fas"</span>, <span class="at">outpath =</span> <span class="st">"local_file"</span>,</span>
<span id="cb2-39"><a href="#cb2-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">verbose =</span> <span class="dv">5</span>)</span>
<span id="cb2-40"><a href="#cb2-40" aria-hidden="true" tabindex="-1"></a><span class="co"># Methods 2 and 4 randomly resolve heterozygous sites</span></span>
<span id="cb2-41"><a href="#cb2-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-42"><a href="#cb2-42" aria-hidden="true" tabindex="-1"></a><span class="co"># read in the alignment as a DNAbin object via ape</span></span>
<span id="cb2-43"><a href="#cb2-43" aria-hidden="true" tabindex="-1"></a>seq.tmp <span class="ot">&lt;-</span> seqinr<span class="sc">::</span><span class="fu">read.fasta</span>(<span class="st">"local_file"</span>, <span class="at">seqtype =</span> <span class="st">"DNA"</span>, <span class="at">forceDNAtolower =</span> F)</span>
<span id="cb2-44"><a href="#cb2-44" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-45"><a href="#cb2-45" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(seq.tmp)) {</span>
<span id="cb2-46"><a href="#cb2-46" aria-hidden="true" tabindex="-1"></a>    <span class="co"># remove any empty entries in the alignment</span></span>
<span id="cb2-47"><a href="#cb2-47" aria-hidden="true" tabindex="-1"></a>    seq.tmp[[i]] <span class="ot">&lt;-</span> seq.tmp[[i]][<span class="sc">-</span><span class="fu">which</span>(seq.tmp[[i]] <span class="sc">==</span> <span class="st">" "</span>)]</span>
<span id="cb2-48"><a href="#cb2-48" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb2-49"><a href="#cb2-49" aria-hidden="true" tabindex="-1"></a><span class="co"># make a dictionary for ambiguity codes</span></span>
<span id="cb2-50"><a href="#cb2-50" aria-hidden="true" tabindex="-1"></a>amb.codes <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="at">M =</span> <span class="fu">c</span>(<span class="st">"A"</span>, <span class="st">"C"</span>), <span class="at">R =</span> <span class="fu">c</span>(<span class="st">"A"</span>, <span class="st">"G"</span>), <span class="at">W =</span> <span class="fu">c</span>(<span class="st">"A"</span>, <span class="st">"T"</span>), <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"C"</span>, <span class="st">"G"</span>),</span>
<span id="cb2-51"><a href="#cb2-51" aria-hidden="true" tabindex="-1"></a>    <span class="at">Y =</span> <span class="fu">c</span>(<span class="st">"C"</span>, <span class="st">"T"</span>), <span class="at">K =</span> <span class="fu">c</span>(<span class="st">"G"</span>, <span class="st">"T"</span>))</span>
<span id="cb2-52"><a href="#cb2-52" aria-hidden="true" tabindex="-1"></a><span class="co"># loop through and replace any ambiguous bases by randomly choosing one or the</span></span>
<span id="cb2-53"><a href="#cb2-53" aria-hidden="true" tabindex="-1"></a><span class="co"># other appropriate base</span></span>
<span id="cb2-54"><a href="#cb2-54" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(seq.tmp)) {</span>
<span id="cb2-55"><a href="#cb2-55" aria-hidden="true" tabindex="-1"></a>    seq.trash <span class="ot">&lt;-</span> seq.tmp[[i]]</span>
<span id="cb2-56"><a href="#cb2-56" aria-hidden="true" tabindex="-1"></a>    rep.trash <span class="ot">&lt;-</span> <span class="fu">which</span>(seq.trash <span class="sc">%in%</span> <span class="fu">names</span>(amb.codes))</span>
<span id="cb2-57"><a href="#cb2-57" aria-hidden="true" tabindex="-1"></a>    <span class="cf">for</span> (j <span class="cf">in</span> rep.trash) {</span>
<span id="cb2-58"><a href="#cb2-58" aria-hidden="true" tabindex="-1"></a>        seq.trash[j] <span class="ot">&lt;-</span> <span class="fu">sample</span>(amb.codes[[<span class="fu">which</span>(<span class="fu">names</span>(amb.codes) <span class="sc">==</span> seq.trash[j])]],</span>
<span id="cb2-59"><a href="#cb2-59" aria-hidden="true" tabindex="-1"></a>            <span class="dv">1</span>)</span>
<span id="cb2-60"><a href="#cb2-60" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb2-61"><a href="#cb2-61" aria-hidden="true" tabindex="-1"></a>    seq.tmp[[i]] <span class="ot">&lt;-</span> seq.trash</span>
<span id="cb2-62"><a href="#cb2-62" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb2-63"><a href="#cb2-63" aria-hidden="true" tabindex="-1"></a><span class="co"># By randomly resolving the heterozygous positions, some sites in the alignment</span></span>
<span id="cb2-64"><a href="#cb2-64" aria-hidden="true" tabindex="-1"></a><span class="co"># are rendered invariable. Let's get rid of them. We'll do this manually (not</span></span>
<span id="cb2-65"><a href="#cb2-65" aria-hidden="true" tabindex="-1"></a><span class="co"># using gl.filter.monomorphs) because our sequence data are in DNAbin format,</span></span>
<span id="cb2-66"><a href="#cb2-66" aria-hidden="true" tabindex="-1"></a><span class="co"># not genlight (GL).</span></span>
<span id="cb2-67"><a href="#cb2-67" aria-hidden="true" tabindex="-1"></a>del.pos <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb2-68"><a href="#cb2-68" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(seq.tmp[[<span class="dv">1</span>]])) {</span>
<span id="cb2-69"><a href="#cb2-69" aria-hidden="true" tabindex="-1"></a>    pos.trash <span class="ot">&lt;-</span> <span class="fu">unlist</span>(purrr<span class="sc">::</span><span class="fu">map</span>(seq.tmp, i))</span>
<span id="cb2-70"><a href="#cb2-70" aria-hidden="true" tabindex="-1"></a>    length.trash <span class="ot">&lt;-</span> <span class="fu">length</span>(<span class="fu">unique</span>(pos.trash))</span>
<span id="cb2-71"><a href="#cb2-71" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (length.trash <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb2-72"><a href="#cb2-72" aria-hidden="true" tabindex="-1"></a>        del.pos <span class="ot">&lt;-</span> <span class="fu">c</span>(del.pos, i)</span>
<span id="cb2-73"><a href="#cb2-73" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> {</span>
<span id="cb2-74"><a href="#cb2-74" aria-hidden="true" tabindex="-1"></a>        <span class="cf">if</span> (length.trash <span class="sc">==</span> <span class="dv">2</span> <span class="sc">&amp;</span> <span class="st">"N"</span> <span class="sc">%in%</span> pos.trash) {</span>
<span id="cb2-75"><a href="#cb2-75" aria-hidden="true" tabindex="-1"></a>            del.pos <span class="ot">&lt;-</span> <span class="fu">c</span>(del.pos, i)</span>
<span id="cb2-76"><a href="#cb2-76" aria-hidden="true" tabindex="-1"></a>        }</span>
<span id="cb2-77"><a href="#cb2-77" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb2-78"><a href="#cb2-78" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb2-79"><a href="#cb2-79" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(seq.tmp)) {</span>
<span id="cb2-80"><a href="#cb2-80" aria-hidden="true" tabindex="-1"></a>    seq.tmp[[i]] <span class="ot">&lt;-</span> seq.tmp[[i]][<span class="sc">-</span>del.pos]</span>
<span id="cb2-81"><a href="#cb2-81" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb2-82"><a href="#cb2-82" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-83"><a href="#cb2-83" aria-hidden="true" tabindex="-1"></a>seqinr<span class="sc">::</span><span class="fu">write.fasta</span>(seq.tmp, <span class="at">names =</span> <span class="fu">names</span>(seq.tmp), <span class="at">nbchar =</span> <span class="fu">length</span>(seq.tmp[[<span class="dv">1</span>]]),</span>
<span id="cb2-84"><a href="#cb2-84" aria-hidden="true" tabindex="-1"></a>    <span class="at">file.out =</span> <span class="st">"local_file"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>The resulting fasta file was processed through the <a href="http://www.iqtree.org/">IQ-TREE</a> software to construct the genetic relatedness tree with ascertainment bias correction <span class="citation" data-cites="Nguyen2015">(<a href="#ref-Nguyen2015" role="doc-biblioref">Nguyen et al., 2015</a>)</span>.</p>
</section>
<section id="prepare-tree-for-analysis" class="level2">
<h2 class="anchored" data-anchor-id="prepare-tree-for-analysis">Prepare tree for analysis</h2>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>phylo_tree <span class="ot">&lt;-</span> ape<span class="sc">::</span><span class="fu">read.tree</span>(<span class="st">"/Users/nicholaswu/Library/CloudStorage/OneDrive-MurdochUniversity/Projects - Old/LP200100331 - WNS - WSU/M - Population genetics/bat_gl7_mth3.treefile"</span>)</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>tree_tip_label <span class="ot">&lt;-</span> phylo_tree<span class="sc">$</span>tip.label  <span class="co"># extract tree tip names</span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>ID_list <span class="ot">&lt;-</span> <span class="fu">unique</span>(torpor_dat<span class="sc">$</span>ID_gene)  <span class="co"># extract individual name from respirometry dataset</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a><span class="co"># as.data.frame(setdiff(tree_tip_label, ID_list)) # check what names from the</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a><span class="co"># dataset not found in phylo_tree Three outgroups in tree</span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>pruned_tree <span class="ot">&lt;-</span> ape<span class="sc">::</span><span class="fu">drop.tip</span>(phylo_tree, <span class="fu">setdiff</span>(phylo_tree<span class="sc">$</span>tip.label, ID_list))  <span class="co"># prune phylo_tree to keep species from the main dataset</span></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a><span class="co"># check if lengths match for both data and tree</span></span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a><span class="co"># length(unique(pruned_tree$tip.label)) # 27 length(unique(torpor_dat$ID_gene))</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a><span class="co"># # 27</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Check for polytomies ape::is.binary(pruned_tree) # TRUE</span></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Compute branch lengths</span></span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>pruned_tree_cal <span class="ot">&lt;-</span> ape<span class="sc">::</span><span class="fu">compute.brlen</span>(pruned_tree, <span class="at">method =</span> <span class="st">"Grafen"</span>, <span class="at">power =</span> <span class="dv">1</span>)</span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a><span class="co"># ape::is.ultrametric(pruned_tree_cal) # TRUE</span></span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Create covariance matrix for analysis</span></span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>phylo_cor <span class="ot">&lt;-</span> ape<span class="sc">::</span><span class="fu">vcv</span>(pruned_tree_cal, <span class="at">cor =</span> F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
</section>
<section id="analysis" class="level1">
<h1>Analysis</h1>
<p>Run phylogenetic multi-level model via <a href="https://www.rdocumentation.org/packages/brms/versions/0.6.0/topics/brm"><code>brm</code></a> function. We constructed four chains with 5,000 steps per chain, including 2,500-step warm-up periods, hence a total of 10,000 steps were retained to estimate posterior distributions [i.e., (5,000 − 2,500) × 4 = 10,000]. An alpha delta was .99 to decrease the number of divergent transitions. Fixed effects were assigned weakly informative priors following a Gaussian distribution to speed up model convergence, and Student’s t prior with three degrees of freedom was used for group-level, hierarchical effects. The degree of convergence of model was deemed to be achieved when the Gelman–Rubin statistic <span class="citation" data-cites="Gelman1992">(<a href="#ref-Gelman1992" role="doc-biblioref">Gelman and Rubin, 1992</a>)</span>, was 1.</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>Tsk_mod <span class="ot">&lt;-</span> brms<span class="sc">::</span><span class="fu">brm</span>(Tsk_C <span class="sc">~</span> temp_exp <span class="sc">*</span> pop <span class="sc">+</span> sex <span class="sc">+</span> lnMass <span class="sc">+</span> tor_start <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> ID) <span class="sc">+</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>    (<span class="dv">1</span> <span class="sc">|</span> chamber_n) <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> <span class="fu">gr</span>(ID_gene, <span class="at">cov =</span> phylo)), <span class="at">data =</span> torpor_dat, <span class="at">data2 =</span> <span class="fu">list</span>(<span class="at">phylo =</span> phylo_cor),</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">chains =</span> <span class="dv">4</span>, <span class="at">cores =</span> <span class="dv">4</span>, <span class="at">iter =</span> <span class="dv">10000</span>, <span class="at">warmup =</span> <span class="dv">5000</span>, <span class="at">control =</span> <span class="fu">list</span>(<span class="at">adapt_delta =</span> <span class="fl">0.99</span>))</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>MR_mod <span class="ot">&lt;-</span> brms<span class="sc">::</span><span class="fu">brm</span>(lnMR <span class="sc">~</span> temp_exp <span class="sc">*</span> pop <span class="sc">+</span> sex <span class="sc">+</span> lnMass <span class="sc">+</span> tor_start <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> ID) <span class="sc">+</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>    (<span class="dv">1</span> <span class="sc">|</span> chamber_n) <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> <span class="fu">gr</span>(ID_gene, <span class="at">cov =</span> phylo)), <span class="at">data =</span> torpor_dat, <span class="at">data2 =</span> <span class="fu">list</span>(<span class="at">phylo =</span> phylo_cor),</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">chains =</span> <span class="dv">4</span>, <span class="at">cores =</span> <span class="dv">4</span>, <span class="at">iter =</span> <span class="dv">10000</span>, <span class="at">warmup =</span> <span class="dv">5000</span>, <span class="at">control =</span> <span class="fu">list</span>(<span class="at">adapt_delta =</span> <span class="fl">0.99</span>))</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>MR_mod2 <span class="ot">&lt;-</span> brms<span class="sc">::</span><span class="fu">brm</span>(lnMR <span class="sc">~</span> temp_exp <span class="sc">*</span> pop <span class="sc">+</span> sex <span class="sc">+</span> lnMass <span class="sc">+</span> Tsk_C <span class="sc">+</span> tor_start <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span></span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>    ID) <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> chamber_n) <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> <span class="fu">gr</span>(ID_gene, <span class="at">cov =</span> phylo)), <span class="at">data =</span> torpor_dat, <span class="at">data2 =</span> <span class="fu">list</span>(<span class="at">phylo =</span> phylo_cor),</span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">chains =</span> <span class="dv">4</span>, <span class="at">cores =</span> <span class="dv">4</span>, <span class="at">iter =</span> <span class="dv">10000</span>, <span class="at">warmup =</span> <span class="dv">5000</span>, <span class="at">control =</span> <span class="fu">list</span>(<span class="at">adapt_delta =</span> <span class="fl">0.99</span>))</span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>EWL_mod <span class="ot">&lt;-</span> brms<span class="sc">::</span><span class="fu">brm</span>(lnEWL <span class="sc">~</span> temp_exp <span class="sc">*</span> pop <span class="sc">+</span> sex <span class="sc">+</span> lnMass <span class="sc">+</span> tor_start <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> ID) <span class="sc">+</span></span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>    (<span class="dv">1</span> <span class="sc">|</span> chamber_n) <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> <span class="fu">gr</span>(ID_gene, <span class="at">cov =</span> phylo)), <span class="at">data =</span> torpor_dat, <span class="at">data2 =</span> <span class="fu">list</span>(<span class="at">phylo =</span> phylo_cor),</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">chains =</span> <span class="dv">4</span>, <span class="at">cores =</span> <span class="dv">4</span>, <span class="at">iter =</span> <span class="dv">10000</span>, <span class="at">warmup =</span> <span class="dv">5000</span>, <span class="at">control =</span> <span class="fu">list</span>(<span class="at">adapt_delta =</span> <span class="fl">0.99</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell" data-layout-align="center">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="supplementary_information_files/figure-html/Fig%20S3-1.png" class="img-fluid quarto-figure quarto-figure-center figure-img" width="864"></p>
</figure>
</div>
</div>
</div>
<p><strong>Fig. S3</strong> | Scatterplots of the observed data (y) vs the average simulated data (y<sub>rep</sub>) from the posterior predictive distribution for the (<strong>a</strong>) <em>T</em><sub>sk</sub> model, (<strong>b</strong>) the TMR model, and (<strong>c</strong>) the TEWL model. Dashed line represents a slope of 1.</p>
<section id="model-output" class="level2">
<h2 class="anchored" data-anchor-id="model-output">Model output</h2>
<div class="tabset-margin-container"></div><div class="panel-tabset nav-pills">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" href="">Table S2 - <em>T</em><sub>sk</sub> model</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false" href="">Table S3 - TMR model</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false" href="">Table S4 - TEWL model</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false" href="">Table S5 - TMR + <em>T</em><sub>sk</sub> model</a></li></ul>
<div class="tab-content nav-pills">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<p><strong>Table S2</strong> | Summary statistics testing the effect of air temperature on the bat skin surface temperature (<em>T</em><sub>sk</sub>). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%), which includes the intercept (<span class="math inline">\(\beta_0\)</span>), exposure temperature (<em>T</em><sub>a</sub>), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass, the time since onset of torpor, and the interaction between <em>T</em><sub>a</sub> and population. Group-level effects include the standard deviations (<span class="math inline">\(\sigma\)</span>) for chamber-level effects (<span class="math inline">\(\sigma_{chamber}^2\)</span>), individual-level effects (<span class="math inline">\(\sigma_{ID}^2\)</span>), and phylogenetic relatedness (<span class="math inline">\(\sigma_{phylogeny}^2\)</span>). Heritability was calculated as <span class="math inline">\(h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)\)</span>. The <span class="math inline">\(R_{marginal}^2\)</span> representing variance explained by fixed effects only is 68.32 ± 2.88%, while the <span class="math inline">\(R_{conditional}^2\)</span> representing variance explained by both fixed and group-level effects is 87.01 ± 1.56%.</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;">Parameter</th>
<th style="text-align: right;">Estimate</th>
<th style="text-align: right;">Est.Error</th>
<th style="text-align: right;">Q2.5</th>
<th style="text-align: right;">Q97.5</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><strong>Fixed effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\beta_0\)</span></td>
<td style="text-align: right;">9.27</td>
<td style="text-align: right;">15.52</td>
<td style="text-align: right;">-21.18</td>
<td style="text-align: right;">40.32</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>T</em><sub>a</sub></td>
<td style="text-align: right;">0.52</td>
<td style="text-align: right;">0.05</td>
<td style="text-align: right;">0.43</td>
<td style="text-align: right;">0.63</td>
</tr>
<tr class="even">
<td style="text-align: left;">Population - Yessabah</td>
<td style="text-align: right;">-1.70</td>
<td style="text-align: right;">1.02</td>
<td style="text-align: right;">-3.71</td>
<td style="text-align: right;">0.28</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Sex - Male</td>
<td style="text-align: right;">0.20</td>
<td style="text-align: right;">0.91</td>
<td style="text-align: right;">-1.62</td>
<td style="text-align: right;">1.99</td>
</tr>
<tr class="even">
<td style="text-align: left;">lnMass</td>
<td style="text-align: right;">1.99</td>
<td style="text-align: right;">5.92</td>
<td style="text-align: right;">-9.81</td>
<td style="text-align: right;">13.68</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Torpor onset</td>
<td style="text-align: right;">-0.13</td>
<td style="text-align: right;">0.15</td>
<td style="text-align: right;">-0.44</td>
<td style="text-align: right;">0.17</td>
</tr>
<tr class="even">
<td style="text-align: left;"><em>T</em><sub>a</sub>: Population - Yessabah</td>
<td style="text-align: right;">-0.04</td>
<td style="text-align: right;">0.06</td>
<td style="text-align: right;">-0.16</td>
<td style="text-align: right;">0.08</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>Group-level effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\sigma_{chamber}^2\)</span></td>
<td style="text-align: right;">0.44</td>
<td style="text-align: right;">0.40</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">1.46</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\sigma_{ID}^2\)</span></td>
<td style="text-align: right;">1.57</td>
<td style="text-align: right;">0.34</td>
<td style="text-align: right;">0.99</td>
<td style="text-align: right;">2.32</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\sigma_{phylogeny}^2\)</span></td>
<td style="text-align: right;">0.61</td>
<td style="text-align: right;">0.55</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">2.06</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>Heritability</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(h^2\)</span></td>
<td style="text-align: right;">0.25</td>
<td style="text-align: right;">0.24</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.81</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<p><strong>Table S3</strong> | Summary statistics testing the effect of air temperature on torpor metabolic rate (TMR). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%), which includes the intercept (<span class="math inline">\(\beta_0\)</span>), exposure temperature (<em>T</em><sub>a</sub>), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass, the time since onset of torpor, and the interaction between <em>T</em><sub>a</sub> and population. Group-level effects include the standard deviations (<span class="math inline">\(\sigma\)</span>) for chamber-level effects (<span class="math inline">\(\sigma_{chamber}^2\)</span>), individual-level effects (<span class="math inline">\(\sigma_{ID}^2\)</span>), and phylogenetic relatedness (<span class="math inline">\(\sigma_{phylogeny}^2\)</span>). Heritability was calculated as <span class="math inline">\(h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)\)</span>. The <span class="math inline">\(R_{marginal}^2\)</span> representing variance explained by fixed effects only is 66.31 ± 4.18%, while the <span class="math inline">\(R_{conditional}^2\)</span> representing variance explained by both fixed and group-level effects is 86.77 ± 1.56%.</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;">Parameter</th>
<th style="text-align: right;">Estimate</th>
<th style="text-align: right;">Est.Error</th>
<th style="text-align: right;">Q2.5</th>
<th style="text-align: right;">Q97.5</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><strong>Fixed effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\beta_0\)</span></td>
<td style="text-align: right;">-8.32</td>
<td style="text-align: right;">6.07</td>
<td style="text-align: right;">-20.08</td>
<td style="text-align: right;">3.71</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>T</em><sub>a</sub></td>
<td style="text-align: right;">0.14</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">0.10</td>
<td style="text-align: right;">0.18</td>
</tr>
<tr class="even">
<td style="text-align: left;">Population - Yessabah</td>
<td style="text-align: right;">-0.93</td>
<td style="text-align: right;">0.41</td>
<td style="text-align: right;">-1.74</td>
<td style="text-align: right;">-0.11</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Sex - Male</td>
<td style="text-align: right;">-0.04</td>
<td style="text-align: right;">0.35</td>
<td style="text-align: right;">-0.74</td>
<td style="text-align: right;">0.62</td>
</tr>
<tr class="even">
<td style="text-align: left;">lnMass</td>
<td style="text-align: right;">4.56</td>
<td style="text-align: right;">2.32</td>
<td style="text-align: right;">-0.02</td>
<td style="text-align: right;">9.08</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Torpor onset</td>
<td style="text-align: right;">-0.07</td>
<td style="text-align: right;">0.06</td>
<td style="text-align: right;">-0.19</td>
<td style="text-align: right;">0.05</td>
</tr>
<tr class="even">
<td style="text-align: left;"><em>T</em><sub>a</sub>: Population - Yessabah</td>
<td style="text-align: right;">-0.03</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">-0.08</td>
<td style="text-align: right;">0.02</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>Group-level effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\sigma_{chamber}^2\)</span></td>
<td style="text-align: right;">0.23</td>
<td style="text-align: right;">0.21</td>
<td style="text-align: right;">0.01</td>
<td style="text-align: right;">0.80</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\sigma_{ID}^2\)</span></td>
<td style="text-align: right;">0.56</td>
<td style="text-align: right;">0.17</td>
<td style="text-align: right;">0.18</td>
<td style="text-align: right;">0.89</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\sigma_{phylogeny}^2\)</span></td>
<td style="text-align: right;">0.42</td>
<td style="text-align: right;">0.35</td>
<td style="text-align: right;">0.01</td>
<td style="text-align: right;">1.32</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>Heritability</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(h^2\)</span></td>
<td style="text-align: right;">0.41</td>
<td style="text-align: right;">0.30</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.92</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<p><strong>Table S4</strong> | Summary statistics testing the effect of air temperature on torpor evaporative water loss (TEWL). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%), which includes the intercept (<span class="math inline">\(\beta_0\)</span>), exposure temperature (<em>T</em><sub>a</sub>), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass,the time since onset of torpor, and the interaction between <em>T</em><sub>a</sub> and population. Group-level effects include the standard deviations (<span class="math inline">\(\sigma\)</span>) for chamber-level effects (<span class="math inline">\(\sigma_{chamber}^2\)</span>), individual-level effects (<span class="math inline">\(\sigma_{ID}^2\)</span>), and phylogenetic relatedness (<span class="math inline">\(\sigma_{phylogeny}^2\)</span>). Heritability was calculated as <span class="math inline">\(h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)\)</span>. The <span class="math inline">\(R_{marginal}^2\)</span> representing variance explained by fixed effects only is 73.62 ± 2.62%, while the <span class="math inline">\(R_{conditional}^2\)</span> representing variance explained by both fixed and group-level effects is 77.36 ± 2.25%.</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;">Parameter</th>
<th style="text-align: right;">Estimate</th>
<th style="text-align: right;">Est.Error</th>
<th style="text-align: right;">Q2.5</th>
<th style="text-align: right;">Q97.5</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><strong>Fixed effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\beta_0\)</span></td>
<td style="text-align: right;">-1.77</td>
<td style="text-align: right;">2.17</td>
<td style="text-align: right;">-6.03</td>
<td style="text-align: right;">2.49</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>T</em><sub>a</sub></td>
<td style="text-align: right;">0.13</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">0.09</td>
<td style="text-align: right;">0.16</td>
</tr>
<tr class="even">
<td style="text-align: left;">Population - Yessabah</td>
<td style="text-align: right;">-0.47</td>
<td style="text-align: right;">0.25</td>
<td style="text-align: right;">-0.96</td>
<td style="text-align: right;">0.02</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Sex - Male</td>
<td style="text-align: right;">0.20</td>
<td style="text-align: right;">0.13</td>
<td style="text-align: right;">-0.05</td>
<td style="text-align: right;">0.45</td>
</tr>
<tr class="even">
<td style="text-align: left;">lnMass</td>
<td style="text-align: right;">0.96</td>
<td style="text-align: right;">0.83</td>
<td style="text-align: right;">-0.67</td>
<td style="text-align: right;">2.60</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Torpor onset</td>
<td style="text-align: right;">-0.04</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">-0.08</td>
<td style="text-align: right;">0.01</td>
</tr>
<tr class="even">
<td style="text-align: left;"><em>T</em><sub>a</sub>: Population - Yessabah</td>
<td style="text-align: right;">0.05</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">0.01</td>
<td style="text-align: right;">0.09</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>Group-level effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\sigma_{chamber}^2\)</span></td>
<td style="text-align: right;">0.10</td>
<td style="text-align: right;">0.08</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.31</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\sigma_{ID}^2\)</span></td>
<td style="text-align: right;">0.08</td>
<td style="text-align: right;">0.06</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.21</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\sigma_{phylogeny}^2\)</span></td>
<td style="text-align: right;">0.15</td>
<td style="text-align: right;">0.09</td>
<td style="text-align: right;">0.01</td>
<td style="text-align: right;">0.35</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><strong>Heritability</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(h^2\)</span></td>
<td style="text-align: right;">0.15</td>
<td style="text-align: right;">0.13</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.49</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<p><strong>Table S5</strong> | Summary statistics testing the effect of air temperature and skin surface temperature on torpor metabolic rate (TMR). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%). The model includes the intercept (<span class="math inline">\(\beta_0\)</span>), exposure temperature (<em>T</em><sub>a</sub>), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass, <em>T</em><sub>sk</sub>, the time since onset of torpor, and the interaction between <em>T</em><sub>a</sub> and population. Group-level effects include the standard deviations (<span class="math inline">\(\sigma\)</span>) for chamber-level effects (<span class="math inline">\(\sigma_{chamber}^2\)</span>), individual-level effects (<span class="math inline">\(\sigma_{ID}^2\)</span>), and phylogenetic relatedness (<span class="math inline">\(\sigma_{phylogeny}^2\)</span>). Heritability was calculated as <span class="math inline">\(h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)\)</span>. The <span class="math inline">\(R_{marginal}^2\)</span> representing variance explained by fixed effects only is 66.31 ± 4.18%, while the <span class="math inline">\(R_{conditional}^2\)</span> representing variance explained by both fixed and group-level effects is 86.77 ± 1.56%.</p>
<div class="cell">
<div class="cell-output-display">
<table class="caption-top table table-sm table-striped small">
<thead>
<tr class="header">
<th style="text-align: left;">Parameter</th>
<th style="text-align: right;">Estimate</th>
<th style="text-align: right;">Est.Error</th>
<th style="text-align: right;">Q2.5</th>
<th style="text-align: right;">Q97.5</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;"><strong>Fixed effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\beta_0\)</span></td>
<td style="text-align: right;">-10.10</td>
<td style="text-align: right;">4.44</td>
<td style="text-align: right;">-18.87</td>
<td style="text-align: right;">-1.32</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>T</em><sub>a</sub></td>
<td style="text-align: right;">0.03</td>
<td style="text-align: right;">0.03</td>
<td style="text-align: right;">-0.02</td>
<td style="text-align: right;">0.08</td>
</tr>
<tr class="even">
<td style="text-align: left;">Population - Yessabah</td>
<td style="text-align: right;">-0.53</td>
<td style="text-align: right;">0.33</td>
<td style="text-align: right;">-1.18</td>
<td style="text-align: right;">0.11</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Sex - Male</td>
<td style="text-align: right;">-0.08</td>
<td style="text-align: right;">0.25</td>
<td style="text-align: right;">-0.58</td>
<td style="text-align: right;">0.41</td>
</tr>
<tr class="even">
<td style="text-align: left;">lnMass</td>
<td style="text-align: right;">4.05</td>
<td style="text-align: right;">1.70</td>
<td style="text-align: right;">0.70</td>
<td style="text-align: right;">7.43</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>T</em><sub>sk</sub></td>
<td style="text-align: right;">0.21</td>
<td style="text-align: right;">0.04</td>
<td style="text-align: right;">0.14</td>
<td style="text-align: right;">0.29</td>
</tr>
<tr class="even">
<td style="text-align: left;">Torpor onset</td>
<td style="text-align: right;">-0.04</td>
<td style="text-align: right;">0.05</td>
<td style="text-align: right;">-0.14</td>
<td style="text-align: right;">0.05</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><em>T</em><sub>a</sub>: Population - Yessabah</td>
<td style="text-align: right;">-0.02</td>
<td style="text-align: right;">0.02</td>
<td style="text-align: right;">-0.06</td>
<td style="text-align: right;">0.02</td>
</tr>
<tr class="even">
<td style="text-align: left;"><strong>Group-level effects</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\sigma_{chamber}^2\)</span></td>
<td style="text-align: right;">0.25</td>
<td style="text-align: right;">0.20</td>
<td style="text-align: right;">0.01</td>
<td style="text-align: right;">0.76</td>
</tr>
<tr class="even">
<td style="text-align: left;"><span class="math inline">\(\sigma_{ID}^2\)</span></td>
<td style="text-align: right;">0.38</td>
<td style="text-align: right;">0.12</td>
<td style="text-align: right;">0.14</td>
<td style="text-align: right;">0.62</td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(\sigma_{phylogeny}^2\)</span></td>
<td style="text-align: right;">0.30</td>
<td style="text-align: right;">0.23</td>
<td style="text-align: right;">0.01</td>
<td style="text-align: right;">0.88</td>
</tr>
<tr class="even">
<td style="text-align: left;"><strong>Heritability</strong></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
<td style="text-align: right;"></td>
</tr>
<tr class="odd">
<td style="text-align: left;"><span class="math inline">\(h^2\)</span></td>
<td style="text-align: right;">0.36</td>
<td style="text-align: right;">0.27</td>
<td style="text-align: right;">0.00</td>
<td style="text-align: right;">0.86</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<hr>
</section>
</section>
<section id="figures" class="level1">
<h1>Figures</h1>
<p>Code to produce the main document figures is provided below. Figures produced were further modified in <a href="https://www.adobe.com/au/products/illustrator.html">Adobe Illustrator</a> for publication.</p>
<section id="figure-1---study-sites" class="level2">
<h2 class="anchored" data-anchor-id="figure-1---study-sites">Figure 1 - Study sites</h2>
<p>Download elevation map from <code>rnaturalearth</code> and convert to dataframe.</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Australia map</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>aus_map <span class="ot">&lt;-</span> rnaturalearth<span class="sc">::</span><span class="fu">ne_states</span>(<span class="at">country =</span> <span class="st">"Australia"</span>, <span class="at">returnclass =</span> <span class="st">"sf"</span>)</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Elevation map</span></span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>aus_elev_sp <span class="ot">&lt;-</span> elevatr<span class="sc">::</span><span class="fu">get_elev_raster</span>(<span class="at">locations =</span> aus_map, <span class="at">z =</span> <span class="dv">7</span>, <span class="at">clip =</span> <span class="st">"locations"</span>)  <span class="co"># 1km resolution or 30 arc sec</span></span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a><span class="co"># aus_elev_sp &lt;- readRDS(file.path(data_path, '/aus_elev_sp.rds'))</span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>aus_elev_df <span class="ot">&lt;-</span> raster<span class="sc">::</span><span class="fu">as.data.frame</span>(aus_elev_sp, <span class="at">xy =</span> <span class="cn">TRUE</span>)</span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(aus_elev_df)[<span class="dv">3</span>] <span class="ot">&lt;-</span> <span class="st">"elevation"</span></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>aus_elev_df <span class="ot">&lt;-</span> aus_elev_df[<span class="fu">complete.cases</span>(aus_elev_df), ]  <span class="co"># Remove NAs</span></span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a>aus_elev_df_2 <span class="ot">&lt;-</span> aus_elev_df <span class="sc">%&gt;%</span></span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">filter</span>(x <span class="sc">&gt;=</span> <span class="dv">143</span> <span class="sc">&amp;</span> y <span class="sc">&lt;=</span> <span class="sc">-</span><span class="dv">24</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Produce eastcoast map with study sites, and temperature profiles.</p>
<div class="cell" data-layout-align="center">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Miniopterus distribution</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>range_map <span class="ot">&lt;-</span> raster<span class="sc">::</span><span class="fu">shapefile</span>(<span class="fu">file.path</span>(data_path, <span class="st">"data/Miniopterus_orianae_oceanensis/Miniopterus_orianae_oceanensis.shp"</span>))</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>range_map <span class="ot">&lt;-</span> sp<span class="sc">::</span><span class="fu">spTransform</span>(range_map, raster<span class="sc">::</span><span class="fu">crs</span>(<span class="st">"+proj=longlat +datum=WGS84 +no_defs"</span>))</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Sites</span></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>resp_sites_df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">x =</span> <span class="fu">c</span>(<span class="fl">150.018564</span>, <span class="fl">152.688690</span>), </span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>                            <span class="at">y =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="fl">33.803127</span>, <span class="sc">-</span><span class="fl">31.093943</span>), </span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>                            <span class="at">site =</span> <span class="fu">c</span>(<span class="st">"Jenolan"</span>,<span class="st">"Yessabah"</span>))</span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>map_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_polygon</span>(<span class="at">data =</span> range_map, <span class="fu">aes</span>(<span class="at">x =</span> long, <span class="at">y =</span> lat, <span class="at">group =</span> group), <span class="at">colour =</span> <span class="cn">NA</span>, <span class="at">fill =</span> <span class="st">"#f595a6"</span>) <span class="sc">+</span></span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_raster</span>(<span class="at">data =</span> aus_elev_df_2, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> y, <span class="at">fill =</span> elevation), <span class="at">alpha =</span> <span class="fl">0.5</span>, <span class="at">show.legend =</span> F) <span class="sc">+</span></span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_gradient2</span>(<span class="at">low =</span> <span class="st">"white"</span>, <span class="at">mid =</span> <span class="st">"white"</span>, <span class="at">high =</span> <span class="st">"black"</span>, <span class="at">midpoint =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_sf</span>(<span class="at">data =</span> aus_map, <span class="at">color =</span> <span class="st">"#8C8C8C"</span>, <span class="at">fill =</span> <span class="cn">NA</span>) <span class="sc">+</span></span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">data =</span> resp_sites_df, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> y), <span class="at">shape =</span> <span class="dv">21</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">fill =</span> <span class="st">"white"</span>, <span class="at">size =</span> <span class="dv">3</span>) <span class="sc">+</span></span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlim</span>(<span class="dv">143</span>, <span class="dv">155</span>) <span class="sc">+</span> <span class="fu">ylim</span>(<span class="sc">-</span><span class="fl">39.5</span>, <span class="sc">-</span><span class="dv">24</span>) <span class="sc">+</span></span>
<span id="cb6-17"><a href="#cb6-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_sf</span>() <span class="sc">+</span></span>
<span id="cb6-18"><a href="#cb6-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_void</span>()</span>
<span id="cb6-19"><a href="#cb6-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-20"><a href="#cb6-20" aria-hidden="true" tabindex="-1"></a><span class="co"># Load all files </span></span>
<span id="cb6-21"><a href="#cb6-21" aria-hidden="true" tabindex="-1"></a>HOBO_list  <span class="ot">&lt;-</span> <span class="fu">list.files</span>(<span class="at">path =</span> <span class="fu">file.path</span>(data_path, <span class="st">"data/"</span>), <span class="at">pattern =</span> <span class="st">"</span><span class="sc">\\</span><span class="st">HOBO.csv$"</span>, <span class="at">full.names =</span> T)</span>
<span id="cb6-22"><a href="#cb6-22" aria-hidden="true" tabindex="-1"></a>HOBO_list2 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">file_name =</span> HOBO_list, <span class="at">id =</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(HOBO_list))) <span class="co"># id for joining</span></span>
<span id="cb6-23"><a href="#cb6-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-24"><a href="#cb6-24" aria-hidden="true" tabindex="-1"></a>HOBO_dat <span class="ot">&lt;-</span> HOBO_list <span class="sc">%&gt;%</span> </span>
<span id="cb6-25"><a href="#cb6-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">lapply</span>(read_csv) <span class="sc">%&gt;%</span> <span class="co"># read all files at once</span></span>
<span id="cb6-26"><a href="#cb6-26" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">bind_rows</span>(<span class="at">.id =</span> <span class="st">"id"</span>) <span class="sc">%&gt;%</span> <span class="co"># bind all tables into one object, and give id for each</span></span>
<span id="cb6-27"><a href="#cb6-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">data.frame</span>() <span class="sc">%&gt;%</span></span>
<span id="cb6-28"><a href="#cb6-28" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">date_time =</span> <span class="fu">as.POSIXct</span>(date_time, <span class="at">format =</span> <span class="st">"%m/%d/%Y %H:%M"</span>),</span>
<span id="cb6-29"><a href="#cb6-29" aria-hidden="true" tabindex="-1"></a>                <span class="at">date      =</span> <span class="fu">as.POSIXct</span>(date_time, <span class="at">format =</span> <span class="st">"%m/%d/%Y"</span>),</span>
<span id="cb6-30"><a href="#cb6-30" aria-hidden="true" tabindex="-1"></a>                <span class="at">time      =</span> <span class="fu">strftime</span>(date_time, <span class="at">format =</span> <span class="st">"%H:%M:%S"</span>),</span>
<span id="cb6-31"><a href="#cb6-31" aria-hidden="true" tabindex="-1"></a>                <span class="at">jul_day   =</span> <span class="fu">as.POSIXlt</span>(date_time)<span class="sc">$</span>yday,</span>
<span id="cb6-32"><a href="#cb6-32" aria-hidden="true" tabindex="-1"></a>                <span class="at">month     =</span> <span class="fu">format</span>(<span class="fu">as.Date</span>(date_time, <span class="at">format =</span> <span class="st">"%m/%d/%Y"</span>),<span class="st">"%m"</span>),</span>
<span id="cb6-33"><a href="#cb6-33" aria-hidden="true" tabindex="-1"></a>                <span class="at">month_cat =</span> <span class="fu">format</span>(<span class="fu">as.Date</span>(date_time, <span class="at">format =</span> <span class="st">"%m/%d/%Y"</span>),<span class="st">"%B"</span>),</span>
<span id="cb6-34"><a href="#cb6-34" aria-hidden="true" tabindex="-1"></a>                <span class="at">location =</span> <span class="fu">factor</span>(location, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"EXT"</span>, <span class="st">"deep"</span>))) </span>
<span id="cb6-35"><a href="#cb6-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-36"><a href="#cb6-36" aria-hidden="true" tabindex="-1"></a>temp_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb6-37"><a href="#cb6-37" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">data =</span> HOBO_dat <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">filter</span>(location <span class="sc">==</span> <span class="st">"EXT"</span>), </span>
<span id="cb6-38"><a href="#cb6-38" aria-hidden="true" tabindex="-1"></a>            <span class="fu">aes</span>(<span class="at">x =</span> jul_day, <span class="at">y =</span> air_temp), <span class="at">size =</span> <span class="fl">0.5</span>, <span class="at">colour =</span> <span class="st">"#f595a6"</span>) <span class="sc">+</span></span>
<span id="cb6-39"><a href="#cb6-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="at">data =</span> HOBO_dat <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">filter</span>(location <span class="sc">==</span> <span class="st">"deep"</span>), </span>
<span id="cb6-40"><a href="#cb6-40" aria-hidden="true" tabindex="-1"></a>            <span class="fu">aes</span>(<span class="at">x =</span> jul_day, <span class="at">y =</span> air_temp), <span class="at">size =</span> <span class="fl">1.2</span>, <span class="at">colour =</span> <span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb6-41"><a href="#cb6-41" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">"Air temperature (°C)"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="cn">NULL</span>) <span class="sc">+</span></span>
<span id="cb6-42"><a href="#cb6-42" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mytheme</span>() <span class="sc">+</span></span>
<span id="cb6-43"><a href="#cb6-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"bottom"</span>) <span class="sc">+</span></span>
<span id="cb6-44"><a href="#cb6-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span> site, <span class="at">ncol =</span> <span class="dv">1</span>)</span>
<span id="cb6-45"><a href="#cb6-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-46"><a href="#cb6-46" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(map_plot, temp_plot, <span class="at">ncol =</span> <span class="dv">2</span>, <span class="at">rel_widths =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="fl">0.5</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="figure-2---tsk-tmr-and-tewl" class="level2">
<h2 class="anchored" data-anchor-id="figure-2---tsk-tmr-and-tewl">Figure 2 - <em>T</em><sub>sk</sub>, TMR, and TEWL</h2>
<div class="cell" data-layout-align="center">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>Tsk_me <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(ggeffects<span class="sc">::</span><span class="fu">ggpredict</span>(Tsk_mod, <span class="at">terms =</span> <span class="fu">c</span>(<span class="st">"temp_exp"</span>, <span class="st">"pop"</span>,</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>    <span class="st">"tor_start [5]"</span>, <span class="st">"lnMass [2.7]"</span>)))</span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>Tsk_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>() <span class="sc">+</span> <span class="co"># raw data Tsk_plot &lt;- ggplot() + # raw data</span></span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_line</span>(<span class="at">data =</span> torpor_dat, <span class="fu">aes</span>(<span class="at">x =</span> temp_exp, <span class="at">y =</span> Tsk_C, <span class="at">group =</span> ID), <span class="at">colour =</span> <span class="st">"lightgrey"</span>,</span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">data =</span> torpor_dat,</span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">x =</span> temp_exp, <span class="at">y =</span> Tsk_C, <span class="at">shape =</span> pop), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>),</span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="co"># summary data alpha = 0.5) + # summary data</span></span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_errorbar</span>(<span class="at">data =</span> Tsk_me, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">ymin =</span> conf.low, <span class="at">ymax =</span> conf.high, <span class="at">group =</span> group),</span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">width =</span> <span class="fl">0.1</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_line</span>(<span class="at">data =</span> Tsk_me, <span class="fu">aes</span>(<span class="at">x =</span> x,</span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> predicted, <span class="at">group =</span> group, <span class="at">linetype =</span> group), <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span></span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_point</span>(<span class="at">data =</span> Tsk_me, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> predicted, <span class="at">shape =</span> group, <span class="at">fill =</span> group),</span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>        <span class="at">size =</span> <span class="dv">3</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>,</span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a>    <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"white"</span>, <span class="st">"black"</span>)) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="fu">expression</span>(<span class="fu">italic</span>(<span class="st">"T"</span>)[<span class="st">"a"</span>] <span class="sc">~</span></span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a>    <span class="st">"(°C)"</span>)) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="fu">expression</span>(<span class="fu">italic</span>(<span class="st">"T"</span>)[<span class="st">"sk"</span>] <span class="sc">~</span> <span class="st">"(°C)"</span>)) <span class="sc">+</span> <span class="fu">mytheme</span>()</span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a>MR_me <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(ggeffects<span class="sc">::</span><span class="fu">ggpredict</span>(MR_mod, <span class="at">terms =</span> <span class="fu">c</span>(<span class="st">"temp_exp"</span>, <span class="st">"pop"</span>,</span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a>    <span class="st">"tor_start [5]"</span>, <span class="st">"lnMass [2.7]"</span>))) <span class="sc">%&gt;%</span></span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">est =</span> <span class="fu">exp</span>(predicted), <span class="at">low =</span> <span class="fu">exp</span>(conf.low), <span class="at">high =</span> <span class="fu">exp</span>(conf.high))</span>
<span id="cb7-20"><a href="#cb7-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-21"><a href="#cb7-21" aria-hidden="true" tabindex="-1"></a>MR_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>() <span class="sc">+</span> <span class="co"># raw data MR_plot &lt;- ggplot() + # raw data</span></span>
<span id="cb7-22"><a href="#cb7-22" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_line</span>(<span class="at">data =</span> torpor_dat, <span class="fu">aes</span>(<span class="at">x =</span> temp_exp, <span class="at">y =</span> MR_J_h, <span class="at">group =</span> ID), <span class="at">colour =</span> <span class="st">"lightgrey"</span>,</span>
<span id="cb7-23"><a href="#cb7-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">data =</span> torpor_dat,</span>
<span id="cb7-24"><a href="#cb7-24" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">x =</span> temp_exp, <span class="at">y =</span> MR_J_h, <span class="at">shape =</span> pop), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>),</span>
<span id="cb7-25"><a href="#cb7-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="co"># summary data alpha = 0.5) + # summary data</span></span>
<span id="cb7-26"><a href="#cb7-26" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_errorbar</span>(<span class="at">data =</span> MR_me, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">ymin =</span> low, <span class="at">ymax =</span> high, <span class="at">group =</span> group), <span class="at">width =</span> <span class="fl">0.1</span>,</span>
<span id="cb7-27"><a href="#cb7-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_line</span>(<span class="at">data =</span> MR_me, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> est,</span>
<span id="cb7-28"><a href="#cb7-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">group =</span> group, <span class="at">linetype =</span> group), <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">data =</span> MR_me,</span>
<span id="cb7-29"><a href="#cb7-29" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> est, <span class="at">shape =</span> group, <span class="at">fill =</span> group), <span class="at">size =</span> <span class="dv">3</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span></span>
<span id="cb7-30"><a href="#cb7-30" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"white"</span>,</span>
<span id="cb7-31"><a href="#cb7-31" aria-hidden="true" tabindex="-1"></a>    <span class="st">"black"</span>)) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="fu">expression</span>(<span class="fu">italic</span>(<span class="st">"T"</span>)[<span class="st">"a"</span>] <span class="sc">~</span> <span class="st">"(°C)"</span>)) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="fu">expression</span>(<span class="st">"TMR (J h"</span><span class="sc">^</span><span class="st">"-1"</span> <span class="sc">*</span></span>
<span id="cb7-32"><a href="#cb7-32" aria-hidden="true" tabindex="-1"></a>    <span class="st">")"</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">10</span>, <span class="dv">25</span>, <span class="dv">50</span>, <span class="dv">100</span>, <span class="dv">200</span>, <span class="dv">400</span>, <span class="dv">800</span>), <span class="at">trans =</span> <span class="st">"sqrt"</span>) <span class="sc">+</span></span>
<span id="cb7-33"><a href="#cb7-33" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mytheme</span>()</span>
<span id="cb7-34"><a href="#cb7-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-35"><a href="#cb7-35" aria-hidden="true" tabindex="-1"></a>EWL_me <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(ggeffects<span class="sc">::</span><span class="fu">ggpredict</span>(EWL_mod, <span class="at">terms =</span> <span class="fu">c</span>(<span class="st">"temp_exp"</span>, <span class="st">"pop"</span>,</span>
<span id="cb7-36"><a href="#cb7-36" aria-hidden="true" tabindex="-1"></a>    <span class="st">"tor_start [5]"</span>, <span class="st">"lnMass [2.7]"</span>))) <span class="sc">%&gt;%</span></span>
<span id="cb7-37"><a href="#cb7-37" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">est =</span> <span class="fu">exp</span>(predicted), <span class="at">low =</span> <span class="fu">exp</span>(conf.low), <span class="at">high =</span> <span class="fu">exp</span>(conf.high))</span>
<span id="cb7-38"><a href="#cb7-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-39"><a href="#cb7-39" aria-hidden="true" tabindex="-1"></a>EWL_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>() <span class="sc">+</span> <span class="co"># raw data EWL_plot &lt;- ggplot() + # raw data</span></span>
<span id="cb7-40"><a href="#cb7-40" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_line</span>(<span class="at">data =</span> torpor_dat, <span class="fu">aes</span>(<span class="at">x =</span> temp_exp, <span class="at">y =</span> EWL_mg_h, <span class="at">group =</span> ID), <span class="at">colour =</span> <span class="st">"lightgrey"</span>,</span>
<span id="cb7-41"><a href="#cb7-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">data =</span> torpor_dat,</span>
<span id="cb7-42"><a href="#cb7-42" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">x =</span> temp_exp, <span class="at">y =</span> EWL_mg_h, <span class="at">shape =</span> pop), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>),</span>
<span id="cb7-43"><a href="#cb7-43" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="co"># summary data alpha = 0.5) + # summary data</span></span>
<span id="cb7-44"><a href="#cb7-44" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_errorbar</span>(<span class="at">data =</span> EWL_me, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">ymin =</span> low, <span class="at">ymax =</span> high, <span class="at">group =</span> group),</span>
<span id="cb7-45"><a href="#cb7-45" aria-hidden="true" tabindex="-1"></a>    <span class="at">width =</span> <span class="fl">0.1</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_line</span>(<span class="at">data =</span> EWL_me, <span class="fu">aes</span>(<span class="at">x =</span> x,</span>
<span id="cb7-46"><a href="#cb7-46" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> est, <span class="at">group =</span> group, <span class="at">linetype =</span> group), <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span></span>
<span id="cb7-47"><a href="#cb7-47" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_point</span>(<span class="at">data =</span> EWL_me, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> est, <span class="at">shape =</span> group, <span class="at">fill =</span> group), <span class="at">size =</span> <span class="dv">3</span>,</span>
<span id="cb7-48"><a href="#cb7-48" aria-hidden="true" tabindex="-1"></a>        <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">24</span>)) <span class="sc">+</span></span>
<span id="cb7-49"><a href="#cb7-49" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"white"</span>, <span class="st">"black"</span>)) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="fu">expression</span>(<span class="fu">italic</span>(<span class="st">"T"</span>)[<span class="st">"a"</span>] <span class="sc">~</span></span>
<span id="cb7-50"><a href="#cb7-50" aria-hidden="true" tabindex="-1"></a>    <span class="st">"(°C)"</span>)) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="fu">expression</span>(<span class="st">"TEWL (mg H"</span>[<span class="st">"2"</span>] <span class="sc">*</span> <span class="st">"O"</span> <span class="sc">~</span> <span class="st">"h"</span><span class="sc">^</span><span class="st">"-1"</span> <span class="sc">*</span> <span class="st">")"</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">1</span>,</span>
<span id="cb7-51"><a href="#cb7-51" aria-hidden="true" tabindex="-1"></a>    <span class="fl">2.5</span>, <span class="dv">5</span>, <span class="dv">10</span>, <span class="dv">20</span>, <span class="dv">30</span>), <span class="at">trans =</span> <span class="st">"sqrt"</span>) <span class="sc">+</span> <span class="fu">mytheme</span>()</span>
<span id="cb7-52"><a href="#cb7-52" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-53"><a href="#cb7-53" aria-hidden="true" tabindex="-1"></a>MR_me2 <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(ggeffects<span class="sc">::</span><span class="fu">ggpredict</span>(MR_mod2, <span class="at">terms =</span> <span class="fu">c</span>(<span class="st">"Tsk_C"</span>, <span class="st">"pop"</span>, <span class="st">"tor_start [5]"</span>,</span>
<span id="cb7-54"><a href="#cb7-54" aria-hidden="true" tabindex="-1"></a>    <span class="st">"lnMass [2.7]"</span>))) <span class="sc">%&gt;%</span></span>
<span id="cb7-55"><a href="#cb7-55" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">est =</span> <span class="fu">exp</span>(predicted), <span class="at">low =</span> <span class="fu">exp</span>(conf.low), <span class="at">high =</span> <span class="fu">exp</span>(conf.high))</span>
<span id="cb7-56"><a href="#cb7-56" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-57"><a href="#cb7-57" aria-hidden="true" tabindex="-1"></a>MR_plot2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>() <span class="sc">+</span> <span class="co"># raw data MR_plot2 &lt;- ggplot() + # raw data</span></span>
<span id="cb7-58"><a href="#cb7-58" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_line</span>(<span class="at">data =</span> torpor_dat, <span class="fu">aes</span>(<span class="at">x =</span> Tsk_C, <span class="at">y =</span> MR_J_h, <span class="at">group =</span> ID), <span class="at">colour =</span> <span class="st">"lightgrey"</span>,</span>
<span id="cb7-59"><a href="#cb7-59" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">data =</span> torpor_dat,</span>
<span id="cb7-60"><a href="#cb7-60" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">x =</span> Tsk_C, <span class="at">y =</span> MR_J_h, <span class="at">shape =</span> pop), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="fl">0.2</span>),</span>
<span id="cb7-61"><a href="#cb7-61" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="co"># summary data alpha = 0.5) + # summary data</span></span>
<span id="cb7-62"><a href="#cb7-62" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_ribbon</span>(<span class="at">data =</span> MR_me2, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">ymin =</span> low, <span class="at">ymax =</span> high, <span class="at">group =</span> group), <span class="at">alpha =</span> <span class="fl">0.1</span>) <span class="sc">+</span></span>
<span id="cb7-63"><a href="#cb7-63" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_line</span>(<span class="at">data =</span> MR_me2, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> est, <span class="at">group =</span> group, <span class="at">linetype =</span> group)) <span class="sc">+</span></span>
<span id="cb7-64"><a href="#cb7-64" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"white"</span>,</span>
<span id="cb7-65"><a href="#cb7-65" aria-hidden="true" tabindex="-1"></a>    <span class="st">"black"</span>)) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="fu">expression</span>(<span class="fu">italic</span>(<span class="st">"T"</span>)[<span class="st">"sk"</span>] <span class="sc">~</span> <span class="st">"(°C)"</span>)) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="fu">expression</span>(<span class="st">"MR (J h"</span><span class="sc">^</span><span class="st">"-1"</span> <span class="sc">*</span></span>
<span id="cb7-66"><a href="#cb7-66" aria-hidden="true" tabindex="-1"></a>    <span class="st">")"</span>)) <span class="sc">+</span> <span class="fu">mytheme</span>()</span>
<span id="cb7-67"><a href="#cb7-67" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-68"><a href="#cb7-68" aria-hidden="true" tabindex="-1"></a>cowplot<span class="sc">::</span><span class="fu">plot_grid</span>(MR_plot <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>), EWL_plot <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>),</span>
<span id="cb7-69"><a href="#cb7-69" aria-hidden="true" tabindex="-1"></a>    Tsk_plot <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>), MR_plot2 <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>),</span>
<span id="cb7-70"><a href="#cb7-70" aria-hidden="true" tabindex="-1"></a>    <span class="at">nrow =</span> <span class="dv">2</span>, <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"a"</span>, <span class="st">"b"</span>, <span class="st">"c"</span>, <span class="st">"d"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="figure-3---phylogeny" class="level2">
<h2 class="anchored" data-anchor-id="figure-3---phylogeny">Figure 3 - Phylogeny</h2>
<div class="cell" data-layout-align="center">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>MR_mod_post <span class="ot">&lt;-</span> <span class="fu">posterior_samples</span>(MR_mod)  <span class="co"># extract posterior</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>EWL_mod_post <span class="ot">&lt;-</span> <span class="fu">posterior_samples</span>(EWL_mod)  <span class="co"># extract posterior</span></span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Loop to extract within ID-level posterior of intercept</span></span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>ID_unique <span class="ot">&lt;-</span> <span class="fu">sort</span>(<span class="fu">unique</span>(torpor_dat<span class="sc">$</span>ID))  <span class="co"># pop ID names</span></span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>out_list <span class="ot">&lt;-</span> <span class="fu">vector</span>(<span class="at">mode =</span> <span class="st">"list"</span>, <span class="at">length =</span> <span class="fu">length</span>(ID_unique))</span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (j <span class="cf">in</span> <span class="fu">seq_along</span>(ID_unique)) {</span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a>    out_list[[j]] <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">matrix</span>(<span class="cn">NA</span>, <span class="fu">nrow</span>(MR_mod_post), <span class="dv">1</span>))</span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a>    dat_names <span class="ot">&lt;-</span> <span class="fu">paste0</span>(ID_unique[j])</span>
<span id="cb8-10"><a href="#cb8-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(out_list[[j]]) <span class="ot">&lt;-</span> dat_names</span>
<span id="cb8-11"><a href="#cb8-11" aria-hidden="true" tabindex="-1"></a>    out_list[[j]][, dat_names[<span class="dv">1</span>]] <span class="ot">&lt;-</span> MR_mod_post[, <span class="fu">paste0</span>(<span class="st">"r_ID["</span>, ID_unique[j],</span>
<span id="cb8-12"><a href="#cb8-12" aria-hidden="true" tabindex="-1"></a>        <span class="st">",Intercept]"</span>)]</span>
<span id="cb8-13"><a href="#cb8-13" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb8-14"><a href="#cb8-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-15"><a href="#cb8-15" aria-hidden="true" tabindex="-1"></a>MR_Int_out <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">"cbind.data.frame"</span>, out_list)</span>
<span id="cb8-16"><a href="#cb8-16" aria-hidden="true" tabindex="-1"></a>MR_Int_post <span class="ot">&lt;-</span> <span class="fu">sample_n</span>(MR_Int_out, <span class="dv">4000</span>)  <span class="co"># Extract 4000 random rows</span></span>
<span id="cb8-17"><a href="#cb8-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-18"><a href="#cb8-18" aria-hidden="true" tabindex="-1"></a>out_list <span class="ot">&lt;-</span> <span class="fu">vector</span>(<span class="at">mode =</span> <span class="st">"list"</span>, <span class="at">length =</span> <span class="fu">length</span>(ID_unique))</span>
<span id="cb8-19"><a href="#cb8-19" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (j <span class="cf">in</span> <span class="fu">seq_along</span>(ID_unique)) {</span>
<span id="cb8-20"><a href="#cb8-20" aria-hidden="true" tabindex="-1"></a>    out_list[[j]] <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">matrix</span>(<span class="cn">NA</span>, <span class="fu">nrow</span>(EWL_mod_post), <span class="dv">1</span>))</span>
<span id="cb8-21"><a href="#cb8-21" aria-hidden="true" tabindex="-1"></a>    dat_names <span class="ot">&lt;-</span> <span class="fu">paste0</span>(ID_unique[j])</span>
<span id="cb8-22"><a href="#cb8-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">names</span>(out_list[[j]]) <span class="ot">&lt;-</span> dat_names</span>
<span id="cb8-23"><a href="#cb8-23" aria-hidden="true" tabindex="-1"></a>    out_list[[j]][, dat_names[<span class="dv">1</span>]] <span class="ot">&lt;-</span> EWL_mod_post[, <span class="fu">paste0</span>(<span class="st">"r_ID["</span>, ID_unique[j],</span>
<span id="cb8-24"><a href="#cb8-24" aria-hidden="true" tabindex="-1"></a>        <span class="st">",Intercept]"</span>)]</span>
<span id="cb8-25"><a href="#cb8-25" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb8-26"><a href="#cb8-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-27"><a href="#cb8-27" aria-hidden="true" tabindex="-1"></a>EWL_Int_out <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">"cbind.data.frame"</span>, out_list)</span>
<span id="cb8-28"><a href="#cb8-28" aria-hidden="true" tabindex="-1"></a>EWL_Int_post <span class="ot">&lt;-</span> <span class="fu">sample_n</span>(EWL_Int_out, <span class="dv">4000</span>)  <span class="co"># Extract 4000 random rows</span></span>
<span id="cb8-29"><a href="#cb8-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-30"><a href="#cb8-30" aria-hidden="true" tabindex="-1"></a><span class="co"># Tidy data</span></span>
<span id="cb8-31"><a href="#cb8-31" aria-hidden="true" tabindex="-1"></a>MR_Int_post_long <span class="ot">&lt;-</span> MR_Int_post <span class="sc">%&gt;%</span></span>
<span id="cb8-32"><a href="#cb8-32" aria-hidden="true" tabindex="-1"></a>    tidyr<span class="sc">::</span><span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">everything</span>(), <span class="at">names_to =</span> <span class="st">"ID"</span>, <span class="at">values_to =</span> <span class="st">"Intercept"</span>)</span>
<span id="cb8-33"><a href="#cb8-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-34"><a href="#cb8-34" aria-hidden="true" tabindex="-1"></a>EWL_Int_post_long <span class="ot">&lt;-</span> EWL_Int_post <span class="sc">%&gt;%</span></span>
<span id="cb8-35"><a href="#cb8-35" aria-hidden="true" tabindex="-1"></a>    tidyr<span class="sc">::</span><span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">everything</span>(), <span class="at">names_to =</span> <span class="st">"ID"</span>, <span class="at">values_to =</span> <span class="st">"Intercept"</span>)</span>
<span id="cb8-36"><a href="#cb8-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-37"><a href="#cb8-37" aria-hidden="true" tabindex="-1"></a>MR_Int_post_sum <span class="ot">&lt;-</span> MR_Int_post_long <span class="sc">%&gt;%</span></span>
<span id="cb8-38"><a href="#cb8-38" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">group_by</span>(ID) <span class="sc">%&gt;%</span></span>
<span id="cb8-39"><a href="#cb8-39" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">summarise</span>(<span class="at">mean_MR =</span> <span class="fu">mean</span>(Intercept))</span>
<span id="cb8-40"><a href="#cb8-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-41"><a href="#cb8-41" aria-hidden="true" tabindex="-1"></a>EWL_Int_post_sum <span class="ot">&lt;-</span> EWL_Int_post_long <span class="sc">%&gt;%</span></span>
<span id="cb8-42"><a href="#cb8-42" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">group_by</span>(ID) <span class="sc">%&gt;%</span></span>
<span id="cb8-43"><a href="#cb8-43" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">summarise</span>(<span class="at">mean_EWL =</span> <span class="fu">mean</span>(Intercept))</span>
<span id="cb8-44"><a href="#cb8-44" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-45"><a href="#cb8-45" aria-hidden="true" tabindex="-1"></a>phylo_dat <span class="ot">&lt;-</span> torpor_dat <span class="sc">%&gt;%</span></span>
<span id="cb8-46"><a href="#cb8-46" aria-hidden="true" tabindex="-1"></a>    dplyr<span class="sc">::</span><span class="fu">group_by</span>(ID_gene, ID, sex, pop) <span class="sc">%&gt;%</span></span>
<span id="cb8-47"><a href="#cb8-47" aria-hidden="true" tabindex="-1"></a>    <span class="fu">summarise</span>(<span class="at">mean_mass =</span> <span class="fu">mean</span>(mass_g))</span>
<span id="cb8-48"><a href="#cb8-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-49"><a href="#cb8-49" aria-hidden="true" tabindex="-1"></a>x <span class="ot">&lt;-</span> dplyr<span class="sc">::</span><span class="fu">as_tibble</span>(pruned_tree_cal)</span>
<span id="cb8-50"><a href="#cb8-50" aria-hidden="true" tabindex="-1"></a>y <span class="ot">&lt;-</span> dplyr<span class="sc">::</span><span class="fu">full_join</span>(phylo_dat, MR_Int_post_sum, <span class="at">by =</span> <span class="st">"ID"</span>)</span>
<span id="cb8-51"><a href="#cb8-51" aria-hidden="true" tabindex="-1"></a>y <span class="ot">&lt;-</span> dplyr<span class="sc">::</span><span class="fu">full_join</span>(y, EWL_Int_post_sum, <span class="at">by =</span> <span class="st">"ID"</span>)</span>
<span id="cb8-52"><a href="#cb8-52" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-53"><a href="#cb8-53" aria-hidden="true" tabindex="-1"></a>tree_dat <span class="ot">&lt;-</span> tidytree<span class="sc">::</span><span class="fu">as.treedata</span>(dplyr<span class="sc">::</span><span class="fu">full_join</span>(x, y <span class="sc">%&gt;%</span></span>
<span id="cb8-54"><a href="#cb8-54" aria-hidden="true" tabindex="-1"></a>    <span class="fu">rename</span>(<span class="at">label =</span> ID_gene), <span class="at">by =</span> <span class="st">"label"</span>))</span>
<span id="cb8-55"><a href="#cb8-55" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-56"><a href="#cb8-56" aria-hidden="true" tabindex="-1"></a>tree_plot <span class="ot">&lt;-</span> ggtree<span class="sc">::</span><span class="fu">ggtree</span>(tree_dat) <span class="sc">+</span> ggtree<span class="sc">::</span><span class="fu">geom_tippoint</span>(<span class="fu">aes</span>(<span class="at">shape =</span> pop, <span class="at">fill =</span> pop),</span>
<span id="cb8-57"><a href="#cb8-57" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="dv">2</span>, <span class="at">show.legend =</span> F) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"white"</span>,</span>
<span id="cb8-58"><a href="#cb8-58" aria-hidden="true" tabindex="-1"></a>    <span class="st">"black"</span>))</span>
<span id="cb8-59"><a href="#cb8-59" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-60"><a href="#cb8-60" aria-hidden="true" tabindex="-1"></a>tree_plot2 <span class="ot">&lt;-</span> tree_plot <span class="sc">+</span> ggtreeExtra<span class="sc">::</span><span class="fu">geom_fruit</span>(<span class="at">geom =</span> geom_bar, <span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> mean_MR,</span>
<span id="cb8-61"><a href="#cb8-61" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> pop), <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="fl">0.1</span>, <span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">orientation =</span> <span class="st">"y"</span>,</span>
<span id="cb8-62"><a href="#cb8-62" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.params =</span> <span class="fu">list</span>(<span class="at">axis =</span> <span class="st">"x"</span>, <span class="at">text.size =</span> <span class="dv">3</span>), <span class="at">offset =</span> <span class="fl">0.3</span>, <span class="at">pwidth =</span> <span class="fl">0.4</span>, <span class="at">show.legend =</span> F)</span>
<span id="cb8-63"><a href="#cb8-63" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-64"><a href="#cb8-64" aria-hidden="true" tabindex="-1"></a>tree_plot2 <span class="sc">+</span> ggtreeExtra<span class="sc">::</span><span class="fu">geom_fruit</span>(<span class="at">geom =</span> geom_bar, <span class="at">mapping =</span> <span class="fu">aes</span>(<span class="at">x =</span> mean_EWL,</span>
<span id="cb8-65"><a href="#cb8-65" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> pop), <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="fl">0.1</span>, <span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">orientation =</span> <span class="st">"y"</span>,</span>
<span id="cb8-66"><a href="#cb8-66" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.params =</span> <span class="fu">list</span>(<span class="at">axis =</span> <span class="st">"x"</span>, <span class="at">text.size =</span> <span class="dv">3</span>), <span class="at">offset =</span> <span class="fl">0.3</span>, <span class="at">pwidth =</span> <span class="fl">0.4</span>, <span class="at">show.legend =</span> F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<hr>
</section>
</section>
<section id="references" class="level1 unnumbered">
<h1 class="unnumbered">References</h1>
<div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-Gelman1992" class="csl-entry" role="listitem">
<strong>Gelman, A. and Rubin, D. B.</strong> (1992). <a href="https://doi.org/10.1214/ss/1177011136">Inference from iterative simulation using multiple sequences</a>. <em>Statistical Science</em> <strong>7</strong>, 457–472.
</div>
<div id="ref-Lighton2019" class="csl-entry" role="listitem">
<strong>Lighton, J. R. B.</strong> (2019). <em>Measuring metabolic rates: A manual for scientists</em>. ed. 2. Oxford University Press.
</div>
<div id="ref-dartR2022" class="csl-entry" role="listitem">
<strong>Mijangos, J. L., Berry, O. F., Pacioni, C. and Georges, A.</strong> (2022). <a href="https://doi.org/10.1111/2041-210X.13918"><span class="nocase">dartR v2</span>: An accessible genetic analysis platform for conservation, ecology and agriculture</a>. <em>Methods in Ecology and Evolution</em> <strong>18</strong>, 691–699.
</div>
<div id="ref-Nguyen2015" class="csl-entry" role="listitem">
<strong>Nguyen, L.-T., Schmidt, H. A., Von Haeseler, A. and Minh, B. Q.</strong> (2015). <a href="https://doi.org/10.1093/molbev/msu300">IQ-TREE: A fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</a>. <em>Molecular Biology and Evolution</em> <strong>32</strong>, 268–274.
</div>
<div id="ref-Stull2000" class="csl-entry" role="listitem">
<strong>Stull, R. B.</strong> (2000). Meteorology for scientists and engineers: A technical companion book with ahrens’ meteorology today.
</div>
</div>
<p><br></p>
<hr>
<section id="session-information" class="level2">
<h2 class="anchored" data-anchor-id="session-information">Session Information</h2>
<div class="cell">
<div class="cell-output-display">
<p><strong>R version 4.4.2 (2024-10-31)</strong></p>
<p><strong>Platform:</strong> aarch64-apple-darwin20</p>
<p><strong>attached base packages:</strong> <em>stats</em>, <em>graphics</em>, <em>grDevices</em>, <em>utils</em>, <em>datasets</em>, <em>methods</em> and <em>base</em></p>
<p><strong>other attached packages:</strong> <em>MCMCglmm(v.2.36)</em>, <em>coda(v.0.19-4.1)</em>, <em>Matrix(v.1.7-3)</em>, <em>rstan(v.2.32.7)</em>, <em>StanHeaders(v.2.32.10)</em>, <em>brms(v.2.22.0)</em>, <em>Rcpp(v.1.0.14)</em>, <em>raster(v.3.6-32)</em>, <em>elevatr(v.0.99.0)</em>, <em>sp(v.2.2-0)</em>, <em>sf(v.1.0-20)</em>, <em>rnaturalearth(v.1.0.1)</em>, <em>ggtreeExtra(v.1.14.0)</em>, <em>ggtree(v.3.12.0)</em>, <em>ape(v.5.8-1)</em>, <em>dartR.base(v.1.0.5)</em>, <em>dartR.data(v.1.0.8)</em>, <em>adegenet(v.2.1.11)</em>, <em>ade4(v.1.7-23)</em>, <em>cowplot(v.1.1.3)</em>, <em>lubridate(v.1.9.4)</em>, <em>forcats(v.1.0.0)</em>, <em>stringr(v.1.5.1)</em>, <em>dplyr(v.1.1.4)</em>, <em>purrr(v.1.0.4)</em>, <em>readr(v.2.1.5)</em>, <em>tidyr(v.1.3.1)</em>, <em>tibble(v.3.2.1)</em>, <em>ggplot2(v.3.5.2)</em>, <em>tidyverse(v.2.0.0)</em> and <em>knitr(v.1.50)</em></p>
<p><strong>loaded via a namespace (and not attached):</strong> <em>cubature(v.2.1.2)</em>, <em>splines(v.4.4.2)</em>, <em>later(v.1.4.2)</em>, <em>polspline(v.1.1.25)</em>, <em>ggplotify(v.0.1.2)</em>, <em>datawizard(v.1.0.2)</em>, <em>rpart(v.4.1.24)</em>, <em>lifecycle(v.1.0.4)</em>, <em>Rdpack(v.2.6.4)</em>, <em>doParallel(v.1.0.17)</em>, <em>vroom(v.1.6.5)</em>, <em>haplo.stats(v.1.9.7)</em>, <em>lattice(v.0.22-7)</em>, <em>MASS(v.7.3-65)</em>, <em>insight(v.1.2.0)</em>, <em>backports(v.1.5.0)</em>, <em>magrittr(v.2.0.3)</em>, <em>Hmisc(v.5.2-3)</em>, <em>rmarkdown(v.2.29)</em>, <em>yaml(v.2.3.10)</em>, <em>httpuv(v.1.6.16)</em>, <em>pkgbuild(v.1.4.7)</em>, <em>minqa(v.1.2.8)</em>, <em>DBI(v.1.2.3)</em>, <em>SNPassoc(v.2.1-2)</em>, <em>RColorBrewer(v.1.1-3)</em>, <em>multcomp(v.1.4-28)</em>, <em>abind(v.1.4-8)</em>, <em>zlibbioc(v.1.50.0)</em>, <em>glmmTMB(v.1.1.11)</em>, <em>BiocGenerics(v.0.50.0)</em>, <em>gdsfmt(v.1.40.2)</em>, <em>yulab.utils(v.0.2.0)</em>, <em>nnet(v.7.3-20)</em>, <em>TH.data(v.1.1-3)</em>, <em>tensorA(v.0.36.2.1)</em>, <em>sandwich(v.3.1-1)</em>, <em>inline(v.0.3.21)</em>, <em>poisbinom(v.1.0.1)</em>, <em>tidytree(v.0.4.6)</em>, <em>terra(v.1.8-42)</em>, <em>vegan(v.2.6-10)</em>, <em>units(v.0.8-7)</em>, <em>performance(v.0.13.0)</em>, <em>MatrixModels(v.0.5-4)</em>, <em>bridgesampling(v.1.1-2)</em>, <em>StAMPP(v.1.6.3)</em>, <em>permute(v.0.9-7)</em>, <em>codetools(v.0.2-20)</em>, <em>tidyselect(v.1.2.1)</em>, <em>aplot(v.0.2.5)</em>, <em>ggeffects(v.2.2.1)</em>, <em>bayesplot(v.1.12.0)</em>, <em>farver(v.2.1.2)</em>, <em>lme4(v.1.1-37)</em>, <em>matrixStats(v.1.5.0)</em>, <em>stats4(v.4.4.2)</em>, <em>base64enc(v.0.1-3)</em>, <em>jsonlite(v.2.0.0)</em>, <em>e1071(v.1.7-16)</em>, <em>progressr(v.0.15.1)</em>, <em>Formula(v.1.2-5)</em>, <em>survival(v.3.8-3)</em>, <em>iterators(v.1.0.14)</em>, <em>emmeans(v.1.11.1)</em>, <em>foreach(v.1.5.2)</em>, <em>tools(v.4.4.2)</em>, <em>ggnewscale(v.0.5.1)</em>, <em>treeio(v.1.28.0)</em>, <em>SNPRelate(v.1.38.1)</em>, <em>glue(v.1.8.0)</em>, <em>gridExtra(v.2.3)</em>, <em>xfun(v.0.52)</em>, <em>mgcv(v.1.9-3)</em>, <em>distributional(v.0.5.0)</em>, <em>numDeriv(v.2016.8-1.1)</em>, <em>loo(v.2.8.0)</em>, <em>withr(v.3.0.2)</em>, <em>formatR(v.1.14)</em>, <em>fastmap(v.1.2.0)</em>, <em>boot(v.1.3-31)</em>, <em>SparseM(v.1.84-2)</em>, <em>digest(v.0.6.37)</em>, <em>timechange(v.0.3.0)</em>, <em>R6(v.2.6.1)</em>, <em>mime(v.0.13)</em>, <em>gridGraphics(v.0.5-1)</em>, <em>estimability(v.1.5.1)</em>, <em>colorspace(v.2.1-1)</em>, <em>dichromat(v.2.0-0.1)</em>, <em>generics(v.0.1.3)</em>, <em>corpcor(v.1.6.10)</em>, <em>data.table(v.1.17.0)</em>, <em>class(v.7.3-23)</em>, <em>httr(v.1.4.7)</em>, <em>htmlwidgets(v.1.6.4)</em>, <em>pkgconfig(v.2.0.3)</em>, <em>gtable(v.0.3.6)</em>, <em>htmltools(v.0.5.8.1)</em>, <em>TMB(v.1.9.17)</em>, <em>scales(v.1.4.0)</em>, <em>posterior(v.1.6.1)</em>, <em>reformulas(v.0.4.1)</em>, <em>ggfun(v.0.1.8)</em>, <em>rstudioapi(v.0.17.1)</em>, <em>tzdb(v.0.5.0)</em>, <em>reshape2(v.1.4.4)</em>, <em>nloptr(v.2.2.1)</em>, <em>checkmate(v.2.3.2)</em>, <em>nlme(v.3.1-168)</em>, <em>proxy(v.0.4-27)</em>, <em>zoo(v.1.8-14)</em>, <em>KernSmooth(v.2.23-26)</em>, <em>parallel(v.4.4.2)</em>, <em>foreign(v.0.8-90)</em>, <em>pillar(v.1.10.2)</em>, <em>grid(v.4.4.2)</em>, <em>vctrs(v.0.6.5)</em>, <em>promises(v.1.3.2)</em>, <em>xtable(v.1.8-4)</em>, <em>cluster(v.2.1.8.1)</em>, <em>htmlTable(v.2.4.3)</em>, <em>evaluate(v.1.0.3)</em>, <em>mvtnorm(v.1.3-3)</em>, <em>cli(v.3.6.5)</em>, <em>compiler(v.4.4.2)</em>, <em>rlang(v.1.1.6)</em>, <em>crayon(v.1.5.3)</em>, <em>rms(v.8.0-0)</em>, <em>rstantools(v.2.4.0)</em>, <em>labeling(v.0.4.3)</em>, <em>classInt(v.0.4-11)</em>, <em>plyr(v.1.8.9)</em>, <em>fs(v.1.6.6)</em>, <em>pander(v.0.6.6)</em>, <em>stringi(v.1.8.7)</em>, <em>QuickJSR(v.1.7.0)</em>, <em>arsenal(v.3.6.3)</em>, <em>lazyeval(v.0.2.2)</em>, <em>bayestestR(v.0.15.3)</em>, <em>Brobdingnag(v.1.2-9)</em>, <em>pacman(v.0.5.1)</em>, <em>quantreg(v.6.1)</em>, <em>hms(v.1.1.3)</em>, <em>patchwork(v.1.3.0)</em>, <em>bit64(v.4.6.0-1)</em>, <em>pegas(v.1.3)</em>, <em>seqinr(v.4.2-36)</em>, <em>shiny(v.1.10.0)</em>, <em>haven(v.2.5.4)</em>, <em>rbibutils(v.2.3)</em>, <em>igraph(v.2.1.4)</em>, <em>RcppParallel(v.5.1.10)</em>, <em>snpStats(v.1.54.0)</em> and <em>bit(v.4.6.0)</em></p>
</div>
</div>
<!-- -->

</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb9" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Supplementary Information"</span></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="an">date:</span><span class="co"> today</span></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a><span class="an">date-format:</span><span class="co"> "Do MMM YYYY"</span></span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a><span class="an">author:</span><span class="co"> </span></span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a><span class="co">- name: Nicholas C Wu</span></span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a><span class="co">  email: nicholas.wu.nz@gmail.com</span></span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a><span class="co">  orcid: 0000-0002-7130-1279</span></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a><span class="co">- name: Tomás Villada-Cadavid</span></span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a><span class="co">  email: t.cadavid@westernsydney.edu.au</span></span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a><span class="co">  orcid: 0000-0002-1743-0694</span></span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a><span class="co">- name: Christopher Turbill</span></span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a><span class="co">  email: c.turbill@westernsydney.edu.au</span></span>
<span id="cb9-15"><a href="#cb9-15" aria-hidden="true" tabindex="-1"></a><span class="co">  orcid: 0000-0001-9810-7102</span></span>
<span id="cb9-16"><a href="#cb9-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-17"><a href="#cb9-17" aria-hidden="true" tabindex="-1"></a><span class="an">csl:</span><span class="co"> ./bib/the-journal-of-experimental-biology.csl</span></span>
<span id="cb9-18"><a href="#cb9-18" aria-hidden="true" tabindex="-1"></a><span class="an">bibliography:</span><span class="co"> ./bib/batresp_ref.bib</span></span>
<span id="cb9-19"><a href="#cb9-19" aria-hidden="true" tabindex="-1"></a><span class="an">link-citations:</span><span class="co"> true</span></span>
<span id="cb9-20"><a href="#cb9-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-21"><a href="#cb9-21" aria-hidden="true" tabindex="-1"></a><span class="co">#- Nicholas C. Wu^[Hawkesbury Institute for the Environment, Western Sydney University, NSW 2753, Australia, nicholas.wu.nz@gmail.com]</span></span>
<span id="cb9-22"><a href="#cb9-22" aria-hidden="true" tabindex="-1"></a><span class="co">#- Tomás Villada Cadavid^[Hawkesbury Institute for the Environment, Western Sydney University, NSW 2753, Australia, t.cadavid@westernsydney.edu.au]</span></span>
<span id="cb9-23"><a href="#cb9-23" aria-hidden="true" tabindex="-1"></a><span class="co">#- Christopher Turbill^[Hawkesbury Institute for the Environment, Western Sydney University, NSW 2753, Australia, c.turbill@westernsydney.edu.au], ^[School of Science, Western Sydney University, NSW 2753, Australia]</span></span>
<span id="cb9-24"><a href="#cb9-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-25"><a href="#cb9-25" aria-hidden="true" tabindex="-1"></a><span class="an">format:</span><span class="co"> html</span></span>
<span id="cb9-26"><a href="#cb9-26" aria-hidden="true" tabindex="-1"></a><span class="an">code_download:</span><span class="co"> yes</span></span>
<span id="cb9-27"><a href="#cb9-27" aria-hidden="true" tabindex="-1"></a><span class="an">reference-location:</span><span class="co"> margin</span></span>
<span id="cb9-28"><a href="#cb9-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-29"><a href="#cb9-29" aria-hidden="true" tabindex="-1"></a><span class="an">toc:</span><span class="co"> true</span></span>
<span id="cb9-30"><a href="#cb9-30" aria-hidden="true" tabindex="-1"></a><span class="an">code-fold:</span><span class="co"> true</span></span>
<span id="cb9-31"><a href="#cb9-31" aria-hidden="true" tabindex="-1"></a><span class="an">code-tools:</span><span class="co"> true</span></span>
<span id="cb9-32"><a href="#cb9-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-33"><a href="#cb9-33" aria-hidden="true" tabindex="-1"></a><span class="an">revealjs:</span></span>
<span id="cb9-34"><a href="#cb9-34" aria-hidden="true" tabindex="-1"></a><span class="an">theme:</span><span class="co"> </span></span>
<span id="cb9-35"><a href="#cb9-35" aria-hidden="true" tabindex="-1"></a><span class="co">- journal</span></span>
<span id="cb9-36"><a href="#cb9-36" aria-hidden="true" tabindex="-1"></a><span class="co">- journal_mod.scss</span></span>
<span id="cb9-37"><a href="#cb9-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-38"><a href="#cb9-38" aria-hidden="true" tabindex="-1"></a><span class="an">linkcolor:</span><span class="co"> "#e12744" </span></span>
<span id="cb9-39"><a href="#cb9-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-40"><a href="#cb9-40" aria-hidden="true" tabindex="-1"></a><span class="an">fontsize:</span><span class="co"> 95%;</span></span>
<span id="cb9-41"><a href="#cb9-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-42"><a href="#cb9-42" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb9-43"><a href="#cb9-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-44"><a href="#cb9-44" aria-hidden="true" tabindex="-1"></a><span class="co">[</span><span class="ot">GitHub Repository</span><span class="co">](https://github.com/nicholaswunz/bat-resp)</span>:</span>
<span id="cb9-45"><a href="#cb9-45" aria-hidden="true" tabindex="-1"></a>The Quarto code to produce the HTML file can be downloaded from the *Code* drop-down menu (top right).</span>
<span id="cb9-46"><a href="#cb9-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-47"><a href="#cb9-47" aria-hidden="true" tabindex="-1"></a>This supplementary file contains the *R* workflow for processing and analysing the raw data, and creating figures for the manuscript titled "**Local climate and genetic influence on intraspecific patterns in torpor physiology**".</span>
<span id="cb9-48"><a href="#cb9-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-49"><a href="#cb9-49" aria-hidden="true" tabindex="-1"></a><span class="in">```{r setup, include=FALSE}</span></span>
<span id="cb9-50"><a href="#cb9-50" aria-hidden="true" tabindex="-1"></a><span class="in">library(knitr)</span></span>
<span id="cb9-51"><a href="#cb9-51" aria-hidden="true" tabindex="-1"></a><span class="in">knitr::opts_chunk$set(echo = TRUE, cache = FALSE, tidy = TRUE, message = FALSE, warning = FALSE)</span></span>
<span id="cb9-52"><a href="#cb9-52" aria-hidden="true" tabindex="-1"></a><span class="in">options(dplyr.width = Inf, knitr.kable.NA = "", digits = 2)</span></span>
<span id="cb9-53"><a href="#cb9-53" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-54"><a href="#cb9-54" aria-hidden="true" tabindex="-1"></a><span class="in"># Load library</span></span>
<span id="cb9-55"><a href="#cb9-55" aria-hidden="true" tabindex="-1"></a><span class="in">pacman::p_load(tidyverse, cowplot, </span></span>
<span id="cb9-56"><a href="#cb9-56" aria-hidden="true" tabindex="-1"></a><span class="in">               dartR.base, ape, ggtree, ggtreeExtra,</span></span>
<span id="cb9-57"><a href="#cb9-57" aria-hidden="true" tabindex="-1"></a><span class="in">               rnaturalearth, sf, sp, elevatr, raster,</span></span>
<span id="cb9-58"><a href="#cb9-58" aria-hidden="true" tabindex="-1"></a><span class="in">               brms, rstan, MCMCglmm)</span></span>
<span id="cb9-59"><a href="#cb9-59" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-60"><a href="#cb9-60" aria-hidden="true" tabindex="-1"></a><span class="in"># Functions</span></span>
<span id="cb9-61"><a href="#cb9-61" aria-hidden="true" tabindex="-1"></a><span class="in">mytheme &lt;- function() {</span></span>
<span id="cb9-62"><a href="#cb9-62" aria-hidden="true" tabindex="-1"></a><span class="in">  theme_bw() +</span></span>
<span id="cb9-63"><a href="#cb9-63" aria-hidden="true" tabindex="-1"></a><span class="in">    theme(panel.border          = element_rect(fill = NA, colour = "black"),</span></span>
<span id="cb9-64"><a href="#cb9-64" aria-hidden="true" tabindex="-1"></a><span class="in">          panel.grid.major      = element_blank(),</span></span>
<span id="cb9-65"><a href="#cb9-65" aria-hidden="true" tabindex="-1"></a><span class="in">          panel.grid.minor      = element_blank(),</span></span>
<span id="cb9-66"><a href="#cb9-66" aria-hidden="true" tabindex="-1"></a><span class="in">          axis.line             = element_blank(),</span></span>
<span id="cb9-67"><a href="#cb9-67" aria-hidden="true" tabindex="-1"></a><span class="in">          axis.ticks            = element_line(colour = "black"),</span></span>
<span id="cb9-68"><a href="#cb9-68" aria-hidden="true" tabindex="-1"></a><span class="in">          axis.text             = element_text(size = 10, colour = "black"),</span></span>
<span id="cb9-69"><a href="#cb9-69" aria-hidden="true" tabindex="-1"></a><span class="in">          axis.title            = element_text(size = 10),</span></span>
<span id="cb9-70"><a href="#cb9-70" aria-hidden="true" tabindex="-1"></a><span class="in">          axis.title.y          = element_text(vjust = 3),</span></span>
<span id="cb9-71"><a href="#cb9-71" aria-hidden="true" tabindex="-1"></a><span class="in">          axis.title.x          = element_text(vjust = -1),</span></span>
<span id="cb9-72"><a href="#cb9-72" aria-hidden="true" tabindex="-1"></a><span class="in">          panel.background      = element_rect(fill = NA),</span></span>
<span id="cb9-73"><a href="#cb9-73" aria-hidden="true" tabindex="-1"></a><span class="in">          plot.background       = element_rect(fill = NA, color = NA),</span></span>
<span id="cb9-74"><a href="#cb9-74" aria-hidden="true" tabindex="-1"></a><span class="in">          plot.margin           = unit(c(0.2, 0.2, 0.2, 0.2), units = , "cm"),</span></span>
<span id="cb9-75"><a href="#cb9-75" aria-hidden="true" tabindex="-1"></a><span class="in">          legend.background     = element_rect(fill = NA, color = NA),</span></span>
<span id="cb9-76"><a href="#cb9-76" aria-hidden="true" tabindex="-1"></a><span class="in">          legend.box.background = element_rect(fill = NA, color = NA),</span></span>
<span id="cb9-77"><a href="#cb9-77" aria-hidden="true" tabindex="-1"></a><span class="in">          strip.text.x          = element_text(size = 10, color = "black", face = "bold"),</span></span>
<span id="cb9-78"><a href="#cb9-78" aria-hidden="true" tabindex="-1"></a><span class="in">          strip.background      = element_rect(fill = NA, color = NA),</span></span>
<span id="cb9-79"><a href="#cb9-79" aria-hidden="true" tabindex="-1"></a><span class="in">          plot.title            = element_text(size = 12)</span></span>
<span id="cb9-80"><a href="#cb9-80" aria-hidden="true" tabindex="-1"></a><span class="in">    )</span></span>
<span id="cb9-81"><a href="#cb9-81" aria-hidden="true" tabindex="-1"></a><span class="in">} # set up plot theme</span></span>
<span id="cb9-82"><a href="#cb9-82" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-83"><a href="#cb9-83" aria-hidden="true" tabindex="-1"></a><span class="in">addbracket &lt;- function(x){paste("[", x, "]")} # add brackets between words</span></span>
<span id="cb9-84"><a href="#cb9-84" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-85"><a href="#cb9-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-86"><a href="#cb9-86" aria-hidden="true" tabindex="-1"></a><span class="in"># Set options in Rstan</span></span>
<span id="cb9-87"><a href="#cb9-87" aria-hidden="true" tabindex="-1"></a><span class="in">set.seed(10)</span></span>
<span id="cb9-88"><a href="#cb9-88" aria-hidden="true" tabindex="-1"></a><span class="in">rstan_options(auto_write = TRUE) # translate to STAN platform for running Bayesian model</span></span>
<span id="cb9-89"><a href="#cb9-89" aria-hidden="true" tabindex="-1"></a><span class="in">options(mc.cores = parallel::detectCores()) # detects how many cores available to use</span></span>
<span id="cb9-90"><a href="#cb9-90" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-91"><a href="#cb9-91" aria-hidden="true" tabindex="-1"></a><span class="in"># Set directory</span></span>
<span id="cb9-92"><a href="#cb9-92" aria-hidden="true" tabindex="-1"></a><span class="in">data_path &lt;- file.path("/Users/nicholaswu/Library/CloudStorage/OneDrive-MurdochUniversity/Projects - Old/LP200100331 - WNS - WSU/M - Bat respirometry/")</span></span>
<span id="cb9-93"><a href="#cb9-93" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-94"><a href="#cb9-94" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-95"><a href="#cb9-95" aria-hidden="true" tabindex="-1"></a>***</span>
<span id="cb9-96"><a href="#cb9-96" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-97"><a href="#cb9-97" aria-hidden="true" tabindex="-1"></a><span class="fu"># Supplementary Methods</span></span>
<span id="cb9-98"><a href="#cb9-98" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-99"><a href="#cb9-99" aria-hidden="true" tabindex="-1"></a><span class="fu">## Respirometry set-up</span></span>
<span id="cb9-100"><a href="#cb9-100" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-101"><a href="#cb9-101" aria-hidden="true" tabindex="-1"></a><span class="al">![](Fig S1 - Respirometry setup.png)</span></span>
<span id="cb9-102"><a href="#cb9-102" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-103"><a href="#cb9-103" aria-hidden="true" tabindex="-1"></a>**Fig. S1** | Schematic diagram of the flow-through respirometry set up to measure oxygen consumption, carbon dioxide production, and evaporative water loss of seven torpored *Miniopterus orianae oceanensis* and one control blank chamber. The mass flow controllers were placed outside the temperature-controlled incubator to change the flow rate during the experiment manually.</span>
<span id="cb9-104"><a href="#cb9-104" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-105"><a href="#cb9-105" aria-hidden="true" tabindex="-1"></a><span class="fu">## Calculations and conversation</span></span>
<span id="cb9-106"><a href="#cb9-106" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-107"><a href="#cb9-107" aria-hidden="true" tabindex="-1"></a>The rate of oxygen consumption ($\dot{V}\textrm{O}_2$; ml O~2~ min^-1^) was calculated following @Lighton2019 in @eq-vo2:</span>
<span id="cb9-108"><a href="#cb9-108" aria-hidden="true" tabindex="-1"></a> </span>
<span id="cb9-109"><a href="#cb9-109" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb9-110"><a href="#cb9-110" aria-hidden="true" tabindex="-1"></a>\dot{V}\textrm{O}_2 = \frac{\textrm{FR}^\textrm{'}_{\textrm{i}}[(F_{\textrm{i}}\textrm{O}_{2}-F_{\textrm{e}}\textrm{O}_{2})-F_{\textrm{e}}\textrm{O}_{2}(F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}-F^\textrm{'}_{\textrm{i}}\textrm{CO}_{2})]}{(1-F_{\textrm{e}}\textrm{O}_{2})}</span>
<span id="cb9-111"><a href="#cb9-111" aria-hidden="true" tabindex="-1"></a>$$ {#eq-vo2}</span>
<span id="cb9-112"><a href="#cb9-112" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-113"><a href="#cb9-113" aria-hidden="true" tabindex="-1"></a>where  $\textrm{FR}^\textrm{'}_{\textrm{i}}$  is the incurrent dry flow rate corrected for water vapour ($\textrm{FR}^\textrm{'}_{\textrm{i}} = \textrm{FR}_{\textrm{i}} \times \textrm{BP}/(\textrm{BP}-\textrm{WVP})$). $F_{\textrm{i}}\textrm{O}_{2}$ and $F_{\textrm{e}}\textrm{O}_{2}$ is the dry incurrent (baseline) and excurrent O~2~ gas concentration, respectively. $F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}$ and $F^\textrm{'}_{\textrm{i}}\textrm{CO}_{2}$ is the dry” equivalent of the excurrent and incurrent CO2 gas concentration corrected for water vapour, respectively. The rate of carbon dioxide production ($\dot{V}\textrm{CO}_2$; ml CO~2~ min^-1^) was calculated as @eq-vco2:</span>
<span id="cb9-114"><a href="#cb9-114" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-115"><a href="#cb9-115" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb9-116"><a href="#cb9-116" aria-hidden="true" tabindex="-1"></a>\dot{V}\textrm{CO}_2 = \frac{\textrm{FR}^\textrm{'}_{\textrm{i}}[(F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}-F^\textrm{'}_{\textrm{i}}\textrm{CO}_{2})-F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2}(F_{\textrm{i}}\textrm{O}_{2}-F_{\textrm{e}}\textrm{O}_{2})]}{(1-F^\textrm{'}_{\textrm{e}}\textrm{CO}_{2})}</span>
<span id="cb9-117"><a href="#cb9-117" aria-hidden="true" tabindex="-1"></a>$$ {#eq-vco2}</span>
<span id="cb9-118"><a href="#cb9-118" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-119"><a href="#cb9-119" aria-hidden="true" tabindex="-1"></a>Metabolic rate (MR; kJ h^-1^) was calculated from $\dot{V}\textrm{O}_2$ and $\dot{V}\textrm{CO}_2$ as @eq-mr:</span>
<span id="cb9-120"><a href="#cb9-120" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-121"><a href="#cb9-121" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb9-122"><a href="#cb9-122" aria-hidden="true" tabindex="-1"></a>\textrm{MR} = 0.06 \times (3.941 \times \dot{V}\textrm{O}_2 + 1.106 \times \dot{V}\textrm{CO}_2) \times 4.184</span>
<span id="cb9-123"><a href="#cb9-123" aria-hidden="true" tabindex="-1"></a>$$ {#eq-mr}</span>
<span id="cb9-124"><a href="#cb9-124" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-125"><a href="#cb9-125" aria-hidden="true" tabindex="-1"></a>The evaporative water loss (EWL) or $\dot{V}\textrm{H}_2\textrm{O}$ (µg H2O min^-1^) was calculated as @eq-ewl:</span>
<span id="cb9-126"><a href="#cb9-126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-127"><a href="#cb9-127" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb9-128"><a href="#cb9-128" aria-hidden="true" tabindex="-1"></a>\dot{V}\textrm{H}_2\textrm{O} = \textrm{FR}_{\textrm{i}} \left( \frac{\textrm{WVP}}{T \times R_{\textrm{w}}} \right)</span>
<span id="cb9-129"><a href="#cb9-129" aria-hidden="true" tabindex="-1"></a>$$ {#eq-ewl}</span>
<span id="cb9-130"><a href="#cb9-130" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-131"><a href="#cb9-131" aria-hidden="true" tabindex="-1"></a>where WVP is the water vapour pressure (kPa) minus the baseline WVP, $T$ is temperature in Kelvin (°C + 273.15), and $R_{\textrm{w}}$  is the gas constant for water vapour (461.5 J kg^−1^ K^−1^). $\dot{V}\textrm{H}_2\textrm{O}$ was converted to mg H~2~O h^-1^ by multiplying by 0.06. Prior to calculating $\dot{V}\textrm{O}_2$, $\dot{V}\textrm{CO}_2$, and $\dot{V}\textrm{H}_2\textrm{O}$, the baseline drift and time lag of the fractional gas concentrations were corrected for.</span>
<span id="cb9-132"><a href="#cb9-132" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-133"><a href="#cb9-133" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-134"><a href="#cb9-134" aria-hidden="true" tabindex="-1"></a><span class="fu">## Quantifying *Pd*</span></span>
<span id="cb9-135"><a href="#cb9-135" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-136"><a href="#cb9-136" aria-hidden="true" tabindex="-1"></a>To finalise with Anna</span>
<span id="cb9-137"><a href="#cb9-137" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-138"><a href="#cb9-138" aria-hidden="true" tabindex="-1"></a><span class="fu">## Cave microclimate</span></span>
<span id="cb9-139"><a href="#cb9-139" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-140"><a href="#cb9-140" aria-hidden="true" tabindex="-1"></a>The local air temperature and relative humidity outside and inside the cave was recorded to monitor differences in microclimate conditions available for cave-roosting bats. An external HOBO logger (MX2302A; Onset, MA, USA) mounted with a radiation shield (RS3-B; Onset) was placed outside ~20 m away at 1.5 m height from the cave entrance to log external microclimate at 30 second intervals. An internal HOBO logger (MX2301A; Onset) was placed inside the cave above ground in the same chamber where bats were observed roosting to log cave microclimate at 30 second intervals. Cave surface temperature and relative humidity was measured with an infrared thermometer (TG56 IR 30:1; FLIR, OR, USA) and HOBO logger on an 8 m extendable pole (MX2302A) every 10 m inside the cave or where bats were found. Cave surface temperature were measured in three replicates at three time points, May, July, and September. Vapour pressure deficit (kPa) was calculated as the difference between the saturation water vapour pressure ($e_s$) and the actual water vapour pressure ($e_a$). $e_s$ was calculated following the Clausius-Clapeyron equation <span class="co">[</span><span class="ot">@Stull2000</span><span class="co">]</span>.</span>
<span id="cb9-141"><a href="#cb9-141" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-142"><a href="#cb9-142" aria-hidden="true" tabindex="-1"></a><span class="in">```{r Fig S2, fig.align='center', fig.height=5, fig.width=9, echo = FALSE}</span></span>
<span id="cb9-143"><a href="#cb9-143" aria-hidden="true" tabindex="-1"></a><span class="in">temp_plot &lt;- read.csv(file.path(data_path, "data/cave_survey.csv")) %&gt;%</span></span>
<span id="cb9-144"><a href="#cb9-144" aria-hidden="true" tabindex="-1"></a><span class="in">  ggplot(aes(x = suf_temp, fill = bats)) +</span></span>
<span id="cb9-145"><a href="#cb9-145" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_histogram(colour = NA, position = "identity", binwidth = 0.3) +</span></span>
<span id="cb9-146"><a href="#cb9-146" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_vline(xintercept = c(5, 8, 11, 15), linetype = "dashed") +</span></span>
<span id="cb9-147"><a href="#cb9-147" aria-hidden="true" tabindex="-1"></a><span class="in">  xlab("Cave surface temperature (°C)") +</span></span>
<span id="cb9-148"><a href="#cb9-148" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab("Count") +</span></span>
<span id="cb9-149"><a href="#cb9-149" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values=  c("lightgrey", "black")) +</span></span>
<span id="cb9-150"><a href="#cb9-150" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_x_continuous(expand = c(0, 0), limits = c(0, 20)) + </span></span>
<span id="cb9-151"><a href="#cb9-151" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_y_continuous(expand = c(0, 0), limits = c(0, NA)) +</span></span>
<span id="cb9-152"><a href="#cb9-152" aria-hidden="true" tabindex="-1"></a><span class="in">  guides(fill = guide_legend(title = "Bats present?")) +</span></span>
<span id="cb9-153"><a href="#cb9-153" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme() +</span></span>
<span id="cb9-154"><a href="#cb9-154" aria-hidden="true" tabindex="-1"></a><span class="in">  facet_wrap(~ region, nrow = 2)</span></span>
<span id="cb9-155"><a href="#cb9-155" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-156"><a href="#cb9-156" aria-hidden="true" tabindex="-1"></a><span class="in">RH_plot &lt;- read.csv(file.path(data_path, "data/cave_survey.csv")) %&gt;%</span></span>
<span id="cb9-157"><a href="#cb9-157" aria-hidden="true" tabindex="-1"></a><span class="in">  ggplot(aes(x = RH, fill = bats)) +</span></span>
<span id="cb9-158"><a href="#cb9-158" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_histogram(colour = NA, position = "identity", binwidth = 1.4) +</span></span>
<span id="cb9-159"><a href="#cb9-159" aria-hidden="true" tabindex="-1"></a><span class="in">  xlab("Cave relative humidity (%)") +</span></span>
<span id="cb9-160"><a href="#cb9-160" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab("Count") +</span></span>
<span id="cb9-161"><a href="#cb9-161" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values=  c("lightgrey", "black")) +</span></span>
<span id="cb9-162"><a href="#cb9-162" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_x_continuous(expand = c(0, 0), limits = c(0, 100)) + </span></span>
<span id="cb9-163"><a href="#cb9-163" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_y_continuous(expand = c(0, 0), limits = c(0, NA)) +</span></span>
<span id="cb9-164"><a href="#cb9-164" aria-hidden="true" tabindex="-1"></a><span class="in">  guides(fill = guide_legend(title = "Bats present?")) +</span></span>
<span id="cb9-165"><a href="#cb9-165" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme() +</span></span>
<span id="cb9-166"><a href="#cb9-166" aria-hidden="true" tabindex="-1"></a><span class="in">  facet_wrap(~ region, nrow = 2)</span></span>
<span id="cb9-167"><a href="#cb9-167" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-168"><a href="#cb9-168" aria-hidden="true" tabindex="-1"></a><span class="in">VPD_plot &lt;- read.csv(file.path(data_path, "data/cave_survey.csv")) %&gt;%</span></span>
<span id="cb9-169"><a href="#cb9-169" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(es_kPa = 0.611 * exp(2500000 / 461.5 * (1 / 273 - 1 / (temp_C + 273.15))),</span></span>
<span id="cb9-170"><a href="#cb9-170" aria-hidden="true" tabindex="-1"></a><span class="in">                ea_kPa = RH * es_kPa / 100,</span></span>
<span id="cb9-171"><a href="#cb9-171" aria-hidden="true" tabindex="-1"></a><span class="in">                VPD_kPa = es_kPa - ea_kPa) %&gt;%</span></span>
<span id="cb9-172"><a href="#cb9-172" aria-hidden="true" tabindex="-1"></a><span class="in">  ggplot(aes(x = VPD_kPa, fill = bats)) +</span></span>
<span id="cb9-173"><a href="#cb9-173" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_histogram(colour = NA, position = "identity", binwidth = 0.03) +</span></span>
<span id="cb9-174"><a href="#cb9-174" aria-hidden="true" tabindex="-1"></a><span class="in">  xlab(" Cave VPD (kPa)") +</span></span>
<span id="cb9-175"><a href="#cb9-175" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab("Count") +</span></span>
<span id="cb9-176"><a href="#cb9-176" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values=  c("lightgrey", "black")) +</span></span>
<span id="cb9-177"><a href="#cb9-177" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_x_continuous(expand = c(0, 0), limits = c(0, 1.8)) + </span></span>
<span id="cb9-178"><a href="#cb9-178" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_y_continuous(expand = c(0, 0), limits = c(0, NA)) +</span></span>
<span id="cb9-179"><a href="#cb9-179" aria-hidden="true" tabindex="-1"></a><span class="in">  guides(fill = guide_legend(title = "Bats present?")) +</span></span>
<span id="cb9-180"><a href="#cb9-180" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme() +</span></span>
<span id="cb9-181"><a href="#cb9-181" aria-hidden="true" tabindex="-1"></a><span class="in">  facet_wrap(~ region, nrow = 2)</span></span>
<span id="cb9-182"><a href="#cb9-182" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-183"><a href="#cb9-183" aria-hidden="true" tabindex="-1"></a><span class="in">cowplot::plot_grid(temp_plot + theme(legend.position = "none"), </span></span>
<span id="cb9-184"><a href="#cb9-184" aria-hidden="true" tabindex="-1"></a><span class="in">                   RH_plot + theme(legend.position = "none"), </span></span>
<span id="cb9-185"><a href="#cb9-185" aria-hidden="true" tabindex="-1"></a><span class="in">                   VPD_plot + theme(legend.position = "none"),</span></span>
<span id="cb9-186"><a href="#cb9-186" aria-hidden="true" tabindex="-1"></a><span class="in">                   ncol = 3, labels = c('a', 'b', 'c'))</span></span>
<span id="cb9-187"><a href="#cb9-187" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-188"><a href="#cb9-188" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-189"><a href="#cb9-189" aria-hidden="true" tabindex="-1"></a>**Fig. S2** | Distribution of measured (**a**) cave surface temperatures (°C), (**b**) cave relative humidity (RH; %), and (**c**) calculated cave vapour pressure deficit (VPD; kPa) from Jenolan and Yessabah during the 2023 Austral winter period (May–September). Surface temperature, RH, and VPD with bats present were coloured &lt;span style="color:#000000;"&gt;**black**&lt;/span&gt;, while surface temperature, RH, and VPD without bats present were coloured &lt;span style="color:#999999;"&gt;**grey**&lt;/span&gt;. Dashed lines represented the experimental exposure temperatures which are within the expected temperature range in the caves.</span>
<span id="cb9-190"><a href="#cb9-190" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-191"><a href="#cb9-191" aria-hidden="true" tabindex="-1"></a><span class="fu"># Process Data</span></span>
<span id="cb9-192"><a href="#cb9-192" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-193"><a href="#cb9-193" aria-hidden="true" tabindex="-1"></a>Metadata of the imported respirometry data for analysis.</span>
<span id="cb9-194"><a href="#cb9-194" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-195"><a href="#cb9-195" aria-hidden="true" tabindex="-1"></a><span class="in">```{r metadata, echo=FALSE, message=FALSE}</span></span>
<span id="cb9-196"><a href="#cb9-196" aria-hidden="true" tabindex="-1"></a><span class="in">data.frame(Name = c("pop", "date_exp", "ID ", "ID_gene","sex", "FA_mm", "mass_g", "temp_exp", "temp_act", "RH_treatment", "chamber_n", "setFR_ml_min", "BP_kPa", "FR_adj_ml_min", "T_start", "T_end", "t_tor_h", "VO2_ml_min", "VCO2_ml_h", "EWL_mg_h", "Tsk_C","VO2_ml_h", "VCO2_ml_h", "VO2_ml_g_h", "VCO2_ml_g_h", "EWL_mg_g_h", "lnMass", "MR_kJ_h", "MR_J_h", "MR_W", "MR_mW", "t_diff"), </span></span>
<span id="cb9-197"><a href="#cb9-197" aria-hidden="true" tabindex="-1"></a><span class="in">           Description = c("Unique identifier of population source (Jenolan, Yessabah).",</span></span>
<span id="cb9-198"><a href="#cb9-198" aria-hidden="true" tabindex="-1"></a><span class="in">                           "The date when the experiment was conducted in day/month/year",</span></span>
<span id="cb9-199"><a href="#cb9-199" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Unique identifier of the individual.",</span></span>
<span id="cb9-200"><a href="#cb9-200" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Unique identifier of the biopsy sample for genetic analysis",</span></span>
<span id="cb9-201"><a href="#cb9-201" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Sex of the individual (female, male)",</span></span>
<span id="cb9-202"><a href="#cb9-202" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Forearm length in millimetre (mm).",</span></span>
<span id="cb9-203"><a href="#cb9-203" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Body mass in grams (g).",</span></span>
<span id="cb9-204"><a href="#cb9-204" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Set air temperature exposure in degrees Celcius (°C).",</span></span>
<span id="cb9-205"><a href="#cb9-205" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Recorded air temperature in degrees Celcius (°C).",</span></span>
<span id="cb9-206"><a href="#cb9-206" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Recorded relative humidity in percentage (%).",</span></span>
<span id="cb9-207"><a href="#cb9-207" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Unique identifier of the metabolic chamber",</span></span>
<span id="cb9-208"><a href="#cb9-208" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Set flow rate (ml/min).",</span></span>
<span id="cb9-209"><a href="#cb9-209" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Recorded barometric pressure (kPa).",</span></span>
<span id="cb9-210"><a href="#cb9-210" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Adjusted-flow rate (ml/min).",</span></span>
<span id="cb9-211"><a href="#cb9-211" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Start time of the respirometry measurments as year-month-day hour:min:sec.",</span></span>
<span id="cb9-212"><a href="#cb9-212" aria-hidden="true" tabindex="-1"></a><span class="in">                           "End time of the respirometry measurments as year-month-day hour:min:sec.",</span></span>
<span id="cb9-213"><a href="#cb9-213" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Time between onset of torpor and respirometry measurments.",</span></span>
<span id="cb9-214"><a href="#cb9-214" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated rate of oxygen consumption in millilitres per minute (ml/min).",</span></span>
<span id="cb9-215"><a href="#cb9-215" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated rate of carbon dioxide production in millilitres per minute (ml/min).",</span></span>
<span id="cb9-216"><a href="#cb9-216" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated rate of evaporative water loss in milligrams per minute (mg/h).",</span></span>
<span id="cb9-217"><a href="#cb9-217" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated skin surface temperature in degrees Celcius (°C).",</span></span>
<span id="cb9-218"><a href="#cb9-218" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated rate of oxygen consumption in millilitres per hour (ml/h).",</span></span>
<span id="cb9-219"><a href="#cb9-219" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated rate of carbon dioxide production in millilitres per hour (ml/h).",</span></span>
<span id="cb9-220"><a href="#cb9-220" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Mass-specific oxygen consumption in ml/g/h.",</span></span>
<span id="cb9-221"><a href="#cb9-221" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Mass-specific carbon dioxide production in ml/g/h.",</span></span>
<span id="cb9-222"><a href="#cb9-222" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Mass-specific evaporative water loss in mg/g/h.",</span></span>
<span id="cb9-223"><a href="#cb9-223" aria-hidden="true" tabindex="-1"></a><span class="in">                           "The natural logarithm of body mass.",</span></span>
<span id="cb9-224"><a href="#cb9-224" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated metabolic rate in kilojoules per hour (kJ/h).",</span></span>
<span id="cb9-225"><a href="#cb9-225" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated metabolic rate in joules per hour (J/h).",</span></span>
<span id="cb9-226"><a href="#cb9-226" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated metabolic rate in Watts (W).",</span></span>
<span id="cb9-227"><a href="#cb9-227" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Calculated metabolic rate in milliWatts (mW).",</span></span>
<span id="cb9-228"><a href="#cb9-228" aria-hidden="true" tabindex="-1"></a><span class="in">                           "Difference between skin temperature and actual air temperature in degrees Celcius (°C).")) %&gt;%</span></span>
<span id="cb9-229"><a href="#cb9-229" aria-hidden="true" tabindex="-1"></a><span class="in">  knitr::kable() </span></span>
<span id="cb9-230"><a href="#cb9-230" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-231"><a href="#cb9-231" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-232"><a href="#cb9-232" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-233"><a href="#cb9-233" aria-hidden="true" tabindex="-1"></a><span class="fu">## Load and clean data</span></span>
<span id="cb9-234"><a href="#cb9-234" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-235"><a href="#cb9-235" aria-hidden="true" tabindex="-1"></a>Load and clean the raw data for analysis. **Table S1** summarises data on torpor physiology traits by site, sex, and temperature exposure.</span>
<span id="cb9-236"><a href="#cb9-236" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-237"><a href="#cb9-237" aria-hidden="true" tabindex="-1"></a><span class="in">```{r clean, message=FALSE}</span></span>
<span id="cb9-238"><a href="#cb9-238" aria-hidden="true" tabindex="-1"></a><span class="in"># Respirometry data</span></span>
<span id="cb9-239"><a href="#cb9-239" aria-hidden="true" tabindex="-1"></a><span class="in">resp_dat &lt;- read.csv(file.path(data_path, "data/resp_raw_dat.csv")) %&gt;%</span></span>
<span id="cb9-240"><a href="#cb9-240" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(-c(file_name, FR_ml_min, transmitter_n, freq_hz)) %&gt;%</span></span>
<span id="cb9-241"><a href="#cb9-241" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::filter(cond == "dry" &amp; ID != c("MAM-18", "MAM-16")) %&gt;% # remove individuals not in torpor</span></span>
<span id="cb9-242"><a href="#cb9-242" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(pop         = factor(pop),</span></span>
<span id="cb9-243"><a href="#cb9-243" aria-hidden="true" tabindex="-1"></a><span class="in">                VO2_ml_h    = VO2_ml_min * 60,</span></span>
<span id="cb9-244"><a href="#cb9-244" aria-hidden="true" tabindex="-1"></a><span class="in">                VCO2_ml_h   = VCO2_ml_min * 60,</span></span>
<span id="cb9-245"><a href="#cb9-245" aria-hidden="true" tabindex="-1"></a><span class="in">                VO2_ml_g_h  = VO2_ml_h / mass_g,</span></span>
<span id="cb9-246"><a href="#cb9-246" aria-hidden="true" tabindex="-1"></a><span class="in">                VCO2_ml_g_h = VCO2_ml_h / mass_g,</span></span>
<span id="cb9-247"><a href="#cb9-247" aria-hidden="true" tabindex="-1"></a><span class="in">                EWL_mg_g_h  = EWL_mg_h / mass_g,</span></span>
<span id="cb9-248"><a href="#cb9-248" aria-hidden="true" tabindex="-1"></a><span class="in">                lnMass      = log(mass_g),</span></span>
<span id="cb9-249"><a href="#cb9-249" aria-hidden="true" tabindex="-1"></a><span class="in">                T_start     = lubridate::dmy_hm(T_start),</span></span>
<span id="cb9-250"><a href="#cb9-250" aria-hidden="true" tabindex="-1"></a><span class="in">                T_end       = lubridate::dmy_hm(T_end),</span></span>
<span id="cb9-251"><a href="#cb9-251" aria-hidden="true" tabindex="-1"></a><span class="in">                MR_kJ_h     = (0.06 * (3.941 * VO2_ml_min + 1.106 * VCO2_ml_min)) * 4.184, # Convert VO2 and VCO2 to MR kJ h</span></span>
<span id="cb9-252"><a href="#cb9-252" aria-hidden="true" tabindex="-1"></a><span class="in">                MR_J_h      = MR_kJ_h * 1000,</span></span>
<span id="cb9-253"><a href="#cb9-253" aria-hidden="true" tabindex="-1"></a><span class="in">                MR_W        = MR_kJ_h * 0.2777777778, # convert to Watt</span></span>
<span id="cb9-254"><a href="#cb9-254" aria-hidden="true" tabindex="-1"></a><span class="in">                MR_mW       = MR_W * 1000,</span></span>
<span id="cb9-255"><a href="#cb9-255" aria-hidden="true" tabindex="-1"></a><span class="in">                lnVO2       = log(VO2_ml_h),</span></span>
<span id="cb9-256"><a href="#cb9-256" aria-hidden="true" tabindex="-1"></a><span class="in">                lnVCO2      = log(VCO2_ml_h),</span></span>
<span id="cb9-257"><a href="#cb9-257" aria-hidden="true" tabindex="-1"></a><span class="in">                lnMR        = log(MR_J_h),</span></span>
<span id="cb9-258"><a href="#cb9-258" aria-hidden="true" tabindex="-1"></a><span class="in">                lnEWL       = log(EWL_mg_h),</span></span>
<span id="cb9-259"><a href="#cb9-259" aria-hidden="true" tabindex="-1"></a><span class="in">                t_diff      = Tsk_C - temp_act)  </span></span>
<span id="cb9-260"><a href="#cb9-260" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-261"><a href="#cb9-261" aria-hidden="true" tabindex="-1"></a><span class="in"># filter torpor data only</span></span>
<span id="cb9-262"><a href="#cb9-262" aria-hidden="true" tabindex="-1"></a><span class="in">torpor_dat &lt;- resp_dat %&gt;%</span></span>
<span id="cb9-263"><a href="#cb9-263" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::filter(Tsk_C &lt; 28)</span></span>
<span id="cb9-264"><a href="#cb9-264" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-265"><a href="#cb9-265" aria-hidden="true" tabindex="-1"></a><span class="in">resp_sum &lt;- torpor_dat %&gt;%</span></span>
<span id="cb9-266"><a href="#cb9-266" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::group_by(pop, temp_exp) %&gt;%</span></span>
<span id="cb9-267"><a href="#cb9-267" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::summarise(n         = length(VO2_ml_h),</span></span>
<span id="cb9-268"><a href="#cb9-268" aria-hidden="true" tabindex="-1"></a><span class="in">                   EWL_mean  = mean(EWL_mg_h, na.rm = T),</span></span>
<span id="cb9-269"><a href="#cb9-269" aria-hidden="true" tabindex="-1"></a><span class="in">                   EWL_sd    = sd(EWL_mg_h, na.rm = T),</span></span>
<span id="cb9-270"><a href="#cb9-270" aria-hidden="true" tabindex="-1"></a><span class="in">                   Tsk_mean  = mean(Tsk_C, na.rm = T),</span></span>
<span id="cb9-271"><a href="#cb9-271" aria-hidden="true" tabindex="-1"></a><span class="in">                   Tsk_sd    = sd(Tsk_C, na.rm = T),</span></span>
<span id="cb9-272"><a href="#cb9-272" aria-hidden="true" tabindex="-1"></a><span class="in">                   MR_mean   = mean(MR_kJ_h, na.rm = T),</span></span>
<span id="cb9-273"><a href="#cb9-273" aria-hidden="true" tabindex="-1"></a><span class="in">                   MR_sd     = sd(MR_kJ_h, na.rm = T)</span></span>
<span id="cb9-274"><a href="#cb9-274" aria-hidden="true" tabindex="-1"></a><span class="in">                   )</span></span>
<span id="cb9-275"><a href="#cb9-275" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-276"><a href="#cb9-276" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-277"><a href="#cb9-277" aria-hidden="true" tabindex="-1"></a>**Table S1** | Summary data from the respirometry experiment. Sample size (*n*), mean body mass (g), mean ± standard deviation rate of oxygen consumption ($\dot{V}\textrm{O}_2$, ml h^-1^), rate of carbon dioxide production ($\dot{V}\textrm{CO}_2$, ml h^-1^), torpor metabolic rate (TMR, J h^-1^), torpor evaporative water loss (TEWL, mg h^-1^), the skin surface temperature (*T*~sk~, °C), and *Pd* detection via qPCR by sex, air temperature exposure (*T*~a~, °C), and sites.</span>
<span id="cb9-278"><a href="#cb9-278" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-279"><a href="#cb9-279" aria-hidden="true" tabindex="-1"></a><span class="in">```{r tableS1, echo = FALSE}</span></span>
<span id="cb9-280"><a href="#cb9-280" aria-hidden="true" tabindex="-1"></a><span class="in">tableS1 &lt;- torpor_dat %&gt;%</span></span>
<span id="cb9-281"><a href="#cb9-281" aria-hidden="true" tabindex="-1"></a><span class="in">  group_by(pop, temp_exp, sex) %&gt;%</span></span>
<span id="cb9-282"><a href="#cb9-282" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::summarise(n        = length(VO2_ml_h),</span></span>
<span id="cb9-283"><a href="#cb9-283" aria-hidden="true" tabindex="-1"></a><span class="in">                   mass      = mean(mass_g, na.rm = T),</span></span>
<span id="cb9-284"><a href="#cb9-284" aria-hidden="true" tabindex="-1"></a><span class="in">                   O2_mean   = mean(VO2_ml_h, na.rm = T),</span></span>
<span id="cb9-285"><a href="#cb9-285" aria-hidden="true" tabindex="-1"></a><span class="in">                   O2_sd     = sd(VO2_ml_h, na.rm = T),</span></span>
<span id="cb9-286"><a href="#cb9-286" aria-hidden="true" tabindex="-1"></a><span class="in">                   CO2_mean  = mean(VCO2_ml_h, na.rm = T),</span></span>
<span id="cb9-287"><a href="#cb9-287" aria-hidden="true" tabindex="-1"></a><span class="in">                   CO2_sd    = sd(VCO2_ml_h, na.rm = T),</span></span>
<span id="cb9-288"><a href="#cb9-288" aria-hidden="true" tabindex="-1"></a><span class="in">                   MR_mean   = mean(MR_J_h, na.rm = T),</span></span>
<span id="cb9-289"><a href="#cb9-289" aria-hidden="true" tabindex="-1"></a><span class="in">                   MR_sd     = sd(MR_J_h, na.rm = T),</span></span>
<span id="cb9-290"><a href="#cb9-290" aria-hidden="true" tabindex="-1"></a><span class="in">                   EWL_mean  = mean(EWL_mg_h, na.rm = T),</span></span>
<span id="cb9-291"><a href="#cb9-291" aria-hidden="true" tabindex="-1"></a><span class="in">                   EWL_sd    = sd(EWL_mg_h, na.rm = T),</span></span>
<span id="cb9-292"><a href="#cb9-292" aria-hidden="true" tabindex="-1"></a><span class="in">                   Tsk_mean  = mean(Tsk_C, na.rm = T),</span></span>
<span id="cb9-293"><a href="#cb9-293" aria-hidden="true" tabindex="-1"></a><span class="in">                   Tsk_sd    = sd(Tsk_C, na.rm = T)</span></span>
<span id="cb9-294"><a href="#cb9-294" aria-hidden="true" tabindex="-1"></a><span class="in">                   ) %&gt;%</span></span>
<span id="cb9-295"><a href="#cb9-295" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(mass     = format(round(mass, 2), nsmall = 2),</span></span>
<span id="cb9-296"><a href="#cb9-296" aria-hidden="true" tabindex="-1"></a><span class="in">                O2_mean  = format(round(O2_mean, 2), nsmall = 2),</span></span>
<span id="cb9-297"><a href="#cb9-297" aria-hidden="true" tabindex="-1"></a><span class="in">                O2_sd    = format(round(O2_sd, 2), nsmall = 2),</span></span>
<span id="cb9-298"><a href="#cb9-298" aria-hidden="true" tabindex="-1"></a><span class="in">                CO2_mean = format(round(CO2_mean, 2), nsmall = 2),</span></span>
<span id="cb9-299"><a href="#cb9-299" aria-hidden="true" tabindex="-1"></a><span class="in">                CO2_sd   = format(round(CO2_sd, 2), nsmall = 2),</span></span>
<span id="cb9-300"><a href="#cb9-300" aria-hidden="true" tabindex="-1"></a><span class="in">                MR_mean  = format(round(MR_mean, 2), nsmall = 2),</span></span>
<span id="cb9-301"><a href="#cb9-301" aria-hidden="true" tabindex="-1"></a><span class="in">                MR_sd    = format(round(MR_sd, 2), nsmall = 2),</span></span>
<span id="cb9-302"><a href="#cb9-302" aria-hidden="true" tabindex="-1"></a><span class="in">                EWL_mean = format(round(EWL_mean, 2), nsmall = 2),</span></span>
<span id="cb9-303"><a href="#cb9-303" aria-hidden="true" tabindex="-1"></a><span class="in">                EWL_sd   = format(round(EWL_sd, 2), nsmall = 2),</span></span>
<span id="cb9-304"><a href="#cb9-304" aria-hidden="true" tabindex="-1"></a><span class="in">                Tsk_mean = format(round(Tsk_mean, 2), nsmall = 2),</span></span>
<span id="cb9-305"><a href="#cb9-305" aria-hidden="true" tabindex="-1"></a><span class="in">                Tsk_sd   = format(round(Tsk_sd, 2), nsmall = 2),</span></span>
<span id="cb9-306"><a href="#cb9-306" aria-hidden="true" tabindex="-1"></a><span class="in">                O2       = paste(O2_mean, O2_sd, sep = "±"),</span></span>
<span id="cb9-307"><a href="#cb9-307" aria-hidden="true" tabindex="-1"></a><span class="in">                CO2      = paste(CO2_mean, CO2_sd, sep = "±"),</span></span>
<span id="cb9-308"><a href="#cb9-308" aria-hidden="true" tabindex="-1"></a><span class="in">                MR       = paste(MR_mean, MR_sd, sep = "±"),</span></span>
<span id="cb9-309"><a href="#cb9-309" aria-hidden="true" tabindex="-1"></a><span class="in">                EWL      = paste(EWL_mean, EWL_sd, sep = "±"),</span></span>
<span id="cb9-310"><a href="#cb9-310" aria-hidden="true" tabindex="-1"></a><span class="in">                Tsk      = paste(Tsk_mean, Tsk_sd, sep = "±"),</span></span>
<span id="cb9-311"><a href="#cb9-311" aria-hidden="true" tabindex="-1"></a><span class="in">                Pd       = "-") %&gt;%</span></span>
<span id="cb9-312"><a href="#cb9-312" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(pop, temp_exp, n, sex, mass, O2, CO2, MR, EWL, Tsk, Pd)</span></span>
<span id="cb9-313"><a href="#cb9-313" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-314"><a href="#cb9-314" aria-hidden="true" tabindex="-1"></a><span class="in">tableS1 %&gt;% </span></span>
<span id="cb9-315"><a href="#cb9-315" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(</span></span>
<span id="cb9-316"><a href="#cb9-316" aria-hidden="true" tabindex="-1"></a><span class="in">      "Site"             = pop,</span></span>
<span id="cb9-317"><a href="#cb9-317" aria-hidden="true" tabindex="-1"></a><span class="in">      "*T*~a~ (°C)"      = temp_exp,</span></span>
<span id="cb9-318"><a href="#cb9-318" aria-hidden="true" tabindex="-1"></a><span class="in">      "*n*"              = n,</span></span>
<span id="cb9-319"><a href="#cb9-319" aria-hidden="true" tabindex="-1"></a><span class="in">      "Sex"              = sex,</span></span>
<span id="cb9-320"><a href="#cb9-320" aria-hidden="true" tabindex="-1"></a><span class="in">      "Mass (g)"         = mass,</span></span>
<span id="cb9-321"><a href="#cb9-321" aria-hidden="true" tabindex="-1"></a><span class="in">      "$\\dot{V}\\textrm{O}_2$ (ml h^-1^)" = O2,</span></span>
<span id="cb9-322"><a href="#cb9-322" aria-hidden="true" tabindex="-1"></a><span class="in">      "$\\dot{V}\\textrm{CO}_2$ (ml h^-1^)" = CO2,</span></span>
<span id="cb9-323"><a href="#cb9-323" aria-hidden="true" tabindex="-1"></a><span class="in">      "TMR (J h^-1^)"     = MR,</span></span>
<span id="cb9-324"><a href="#cb9-324" aria-hidden="true" tabindex="-1"></a><span class="in">      "TEWL (mg h^-1^)" = EWL,</span></span>
<span id="cb9-325"><a href="#cb9-325" aria-hidden="true" tabindex="-1"></a><span class="in">      "*T*~sk~ (°C)"     = Tsk,</span></span>
<span id="cb9-326"><a href="#cb9-326" aria-hidden="true" tabindex="-1"></a><span class="in">      "*Pd*"    = Pd) %&gt;%</span></span>
<span id="cb9-327"><a href="#cb9-327" aria-hidden="true" tabindex="-1"></a><span class="in">  knitr::kable(digits = 2)</span></span>
<span id="cb9-328"><a href="#cb9-328" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-329"><a href="#cb9-329" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-330"><a href="#cb9-330" aria-hidden="true" tabindex="-1"></a><span class="fu">## SNPs and genetic relatedness</span></span>
<span id="cb9-331"><a href="#cb9-331" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-332"><a href="#cb9-332" aria-hidden="true" tabindex="-1"></a>The following code was written with Ian Brennan to process, filter and trim the raw SNP data (containing 46,224 loci, 4,075 nucleotide sites) using the <span class="co">[</span><span class="ot">`dartR`</span><span class="co">](https://green-striped-gecko.github.io/dartR/)</span> package <span class="co">[</span><span class="ot">@dartR2022</span><span class="co">]</span>. Note: the raw SNP data are being used as part of a larger study examining the genetic connectivity and population dynamics of Australian bent-winged bats. The SNP data will be available upon it's publication (Planckaert *et al.*, In preparation).</span>
<span id="cb9-333"><a href="#cb9-333" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-334"><a href="#cb9-334" aria-hidden="true" tabindex="-1"></a><span class="in">```{r SNP, message=FALSE, eval=FALSE}</span></span>
<span id="cb9-335"><a href="#cb9-335" aria-hidden="true" tabindex="-1"></a><span class="in"># Load DArT file</span></span>
<span id="cb9-336"><a href="#cb9-336" aria-hidden="true" tabindex="-1"></a><span class="in"># Visually check SNP data</span></span>
<span id="cb9-337"><a href="#cb9-337" aria-hidden="true" tabindex="-1"></a><span class="in">dartR.base::gl.smearplot(bat_gl_wu)</span></span>
<span id="cb9-338"><a href="#cb9-338" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-339"><a href="#cb9-339" aria-hidden="true" tabindex="-1"></a><span class="in">nLoc(bat_gl_wu) # 46224 loci </span></span>
<span id="cb9-340"><a href="#cb9-340" aria-hidden="true" tabindex="-1"></a><span class="in">nInd(bat_gl_wu) # 76 individuals</span></span>
<span id="cb9-341"><a href="#cb9-341" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-342"><a href="#cb9-342" aria-hidden="true" tabindex="-1"></a><span class="in">## FILTERING ## ------------------------------------------</span></span>
<span id="cb9-343"><a href="#cb9-343" aria-hidden="true" tabindex="-1"></a><span class="in"># Check raw data</span></span>
<span id="cb9-344"><a href="#cb9-344" aria-hidden="true" tabindex="-1"></a><span class="in">dartR.base::gl.report.callrate(bat_gl_wu) # loci callrate </span></span>
<span id="cb9-345"><a href="#cb9-345" aria-hidden="true" tabindex="-1"></a><span class="in">dartR.base::gl.report.callrate(bat_gl_wu, method = 'ind') # individual callrate</span></span>
<span id="cb9-346"><a href="#cb9-346" aria-hidden="true" tabindex="-1"></a><span class="in">dartR.base::gl.report.reproducibility(bat_gl_wu) # reproducibility</span></span>
<span id="cb9-347"><a href="#cb9-347" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-348"><a href="#cb9-348" aria-hidden="true" tabindex="-1"></a><span class="in"># strict threshold on repeatability</span></span>
<span id="cb9-349"><a href="#cb9-349" aria-hidden="true" tabindex="-1"></a><span class="in">bat_gl2 &lt;- dartR.base::gl.filter.reproducibility(bat_gl_wu, threshold = 0.999)</span></span>
<span id="cb9-350"><a href="#cb9-350" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-351"><a href="#cb9-351" aria-hidden="true" tabindex="-1"></a><span class="in"># remove invariant (monomorphic) sites, we will have to do this</span></span>
<span id="cb9-352"><a href="#cb9-352" aria-hidden="true" tabindex="-1"></a><span class="in"># again later, once we've filtered down the dataset and potentially</span></span>
<span id="cb9-353"><a href="#cb9-353" aria-hidden="true" tabindex="-1"></a><span class="in"># caused new monomorphs</span></span>
<span id="cb9-354"><a href="#cb9-354" aria-hidden="true" tabindex="-1"></a><span class="in">bat_gl3 &lt;- dartR.base::gl.filter.monomorphs(bat_gl2)</span></span>
<span id="cb9-355"><a href="#cb9-355" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-356"><a href="#cb9-356" aria-hidden="true" tabindex="-1"></a><span class="in"># adjust threshold on callrate as necessary</span></span>
<span id="cb9-357"><a href="#cb9-357" aria-hidden="true" tabindex="-1"></a><span class="in">dartR.base::gl.report.callrate(bat_gl3) # loci callrate </span></span>
<span id="cb9-358"><a href="#cb9-358" aria-hidden="true" tabindex="-1"></a><span class="in">bat_gl4 &lt;- dartR.base::gl.filter.callrate(bat_gl3, method = "loc", threshold = 0.9)</span></span>
<span id="cb9-359"><a href="#cb9-359" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-360"><a href="#cb9-360" aria-hidden="true" tabindex="-1"></a><span class="in"># filter out multiple SNPs from the same (linked) locus</span></span>
<span id="cb9-361"><a href="#cb9-361" aria-hidden="true" tabindex="-1"></a><span class="in">bat_gl5 &lt;- dartR.base::gl.filter.secondaries(bat_gl4, method = "best", v = 2)</span></span>
<span id="cb9-362"><a href="#cb9-362" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-363"><a href="#cb9-363" aria-hidden="true" tabindex="-1"></a><span class="in"># filter by the minor allele frequency, I wouldn't exceed 0.05</span></span>
<span id="cb9-364"><a href="#cb9-364" aria-hidden="true" tabindex="-1"></a><span class="in"># in small datasets you worry about losing loci, so can be ~0.04</span></span>
<span id="cb9-365"><a href="#cb9-365" aria-hidden="true" tabindex="-1"></a><span class="in"># in larger datasets should probably be closer to ~0.01</span></span>
<span id="cb9-366"><a href="#cb9-366" aria-hidden="true" tabindex="-1"></a><span class="in">bat_gl6 &lt;- dartR.base::gl.filter.maf(bat_gl5, </span></span>
<span id="cb9-367"><a href="#cb9-367" aria-hidden="true" tabindex="-1"></a><span class="in">                                     threshold = (2 / nInd(bat_gl5)), </span></span>
<span id="cb9-368"><a href="#cb9-368" aria-hidden="true" tabindex="-1"></a><span class="in">                                     v = 5)</span></span>
<span id="cb9-369"><a href="#cb9-369" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-370"><a href="#cb9-370" aria-hidden="true" tabindex="-1"></a><span class="in"># one more pass to clean up callrate and remove monomorphs introduced</span></span>
<span id="cb9-371"><a href="#cb9-371" aria-hidden="true" tabindex="-1"></a><span class="in">dartR.base::gl.report.callrate(bat_gl6, method = 'ind') # individual callrate</span></span>
<span id="cb9-372"><a href="#cb9-372" aria-hidden="true" tabindex="-1"></a><span class="in">bat_gl7 &lt;- dartR.base::gl.filter.callrate(bat_gl6, </span></span>
<span id="cb9-373"><a href="#cb9-373" aria-hidden="true" tabindex="-1"></a><span class="in">                                          method = "ind", </span></span>
<span id="cb9-374"><a href="#cb9-374" aria-hidden="true" tabindex="-1"></a><span class="in">                                          threshold = 0.95, </span></span>
<span id="cb9-375"><a href="#cb9-375" aria-hidden="true" tabindex="-1"></a><span class="in">                                          mono.rm = T, </span></span>
<span id="cb9-376"><a href="#cb9-376" aria-hidden="true" tabindex="-1"></a><span class="in">                                          recalc = T, </span></span>
<span id="cb9-377"><a href="#cb9-377" aria-hidden="true" tabindex="-1"></a><span class="in">                                          recursive = T, </span></span>
<span id="cb9-378"><a href="#cb9-378" aria-hidden="true" tabindex="-1"></a><span class="in">                                          v = 5)</span></span>
<span id="cb9-379"><a href="#cb9-379" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-380"><a href="#cb9-380" aria-hidden="true" tabindex="-1"></a><span class="in"># Export a Concatenated SNP Alignment</span></span>
<span id="cb9-381"><a href="#cb9-381" aria-hidden="true" tabindex="-1"></a><span class="in"># Export with amiguities (A/G = R; C/T=Y) for heterozygous sites</span></span>
<span id="cb9-382"><a href="#cb9-382" aria-hidden="true" tabindex="-1"></a><span class="in">dartR.base::gl2fasta(bat_gl7, </span></span>
<span id="cb9-383"><a href="#cb9-383" aria-hidden="true" tabindex="-1"></a><span class="in">                     method = 4,</span></span>
<span id="cb9-384"><a href="#cb9-384" aria-hidden="true" tabindex="-1"></a><span class="in">                     outfile = "bat_gl7_mth4.fas",</span></span>
<span id="cb9-385"><a href="#cb9-385" aria-hidden="true" tabindex="-1"></a><span class="in">                     outpath = "local_file",</span></span>
<span id="cb9-386"><a href="#cb9-386" aria-hidden="true" tabindex="-1"></a><span class="in">                     verbose = 5)</span></span>
<span id="cb9-387"><a href="#cb9-387" aria-hidden="true" tabindex="-1"></a><span class="in"># Methods 2 and 4 randomly resolve heterozygous sites</span></span>
<span id="cb9-388"><a href="#cb9-388" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-389"><a href="#cb9-389" aria-hidden="true" tabindex="-1"></a><span class="in"># read in the alignment as a DNAbin object via ape</span></span>
<span id="cb9-390"><a href="#cb9-390" aria-hidden="true" tabindex="-1"></a><span class="in">seq.tmp &lt;- seqinr::read.fasta("local_file",</span></span>
<span id="cb9-391"><a href="#cb9-391" aria-hidden="true" tabindex="-1"></a><span class="in">                      seqtype = "DNA",</span></span>
<span id="cb9-392"><a href="#cb9-392" aria-hidden="true" tabindex="-1"></a><span class="in">                      forceDNAtolower = F)</span></span>
<span id="cb9-393"><a href="#cb9-393" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-394"><a href="#cb9-394" aria-hidden="true" tabindex="-1"></a><span class="in">for (i in 1:length(seq.tmp)){</span></span>
<span id="cb9-395"><a href="#cb9-395" aria-hidden="true" tabindex="-1"></a><span class="in">  # remove any empty entries in the alignment</span></span>
<span id="cb9-396"><a href="#cb9-396" aria-hidden="true" tabindex="-1"></a><span class="in">  seq.tmp[[i]] &lt;- seq.tmp[[i]][-which(seq.tmp[[i]] == " ")]</span></span>
<span id="cb9-397"><a href="#cb9-397" aria-hidden="true" tabindex="-1"></a><span class="in">}</span></span>
<span id="cb9-398"><a href="#cb9-398" aria-hidden="true" tabindex="-1"></a><span class="in"># make a dictionary for ambiguity codes</span></span>
<span id="cb9-399"><a href="#cb9-399" aria-hidden="true" tabindex="-1"></a><span class="in">amb.codes &lt;- list(M = c("A","C"),R = c("A","G"), W = c("A","T"),S = c("C","G"),Y = c("C","T"),K = c("G","T"))</span></span>
<span id="cb9-400"><a href="#cb9-400" aria-hidden="true" tabindex="-1"></a><span class="in"># loop through and replace any ambiguous bases by randomly choosing one or the other appropriate base</span></span>
<span id="cb9-401"><a href="#cb9-401" aria-hidden="true" tabindex="-1"></a><span class="in">for (i in 1:length(seq.tmp)){</span></span>
<span id="cb9-402"><a href="#cb9-402" aria-hidden="true" tabindex="-1"></a><span class="in">  seq.trash &lt;- seq.tmp[[i]]</span></span>
<span id="cb9-403"><a href="#cb9-403" aria-hidden="true" tabindex="-1"></a><span class="in">  rep.trash &lt;- which(seq.trash %in% names(amb.codes))</span></span>
<span id="cb9-404"><a href="#cb9-404" aria-hidden="true" tabindex="-1"></a><span class="in">  for (j in rep.trash){</span></span>
<span id="cb9-405"><a href="#cb9-405" aria-hidden="true" tabindex="-1"></a><span class="in">    seq.trash[j] &lt;- sample(amb.codes[[which(names(amb.codes) == seq.trash[j])]], 1)</span></span>
<span id="cb9-406"><a href="#cb9-406" aria-hidden="true" tabindex="-1"></a><span class="in">  }</span></span>
<span id="cb9-407"><a href="#cb9-407" aria-hidden="true" tabindex="-1"></a><span class="in">  seq.tmp[[i]] &lt;- seq.trash</span></span>
<span id="cb9-408"><a href="#cb9-408" aria-hidden="true" tabindex="-1"></a><span class="in">}</span></span>
<span id="cb9-409"><a href="#cb9-409" aria-hidden="true" tabindex="-1"></a><span class="in"># By randomly resolving the heterozygous positions, some sites in the alignment</span></span>
<span id="cb9-410"><a href="#cb9-410" aria-hidden="true" tabindex="-1"></a><span class="in"># are rendered invariable. Let's get rid of them. We'll do this manually (not using gl.filter.monomorphs)</span></span>
<span id="cb9-411"><a href="#cb9-411" aria-hidden="true" tabindex="-1"></a><span class="in"># because our sequence data are in DNAbin format, not genlight (GL).</span></span>
<span id="cb9-412"><a href="#cb9-412" aria-hidden="true" tabindex="-1"></a><span class="in">del.pos &lt;- c()</span></span>
<span id="cb9-413"><a href="#cb9-413" aria-hidden="true" tabindex="-1"></a><span class="in">for (i in 1:length(seq.tmp[[1]])){</span></span>
<span id="cb9-414"><a href="#cb9-414" aria-hidden="true" tabindex="-1"></a><span class="in">  pos.trash &lt;- unlist(purrr::map(seq.tmp,i))</span></span>
<span id="cb9-415"><a href="#cb9-415" aria-hidden="true" tabindex="-1"></a><span class="in">  length.trash &lt;- length(unique(pos.trash))</span></span>
<span id="cb9-416"><a href="#cb9-416" aria-hidden="true" tabindex="-1"></a><span class="in">  if(length.trash == 1){</span></span>
<span id="cb9-417"><a href="#cb9-417" aria-hidden="true" tabindex="-1"></a><span class="in">    del.pos &lt;- c(del.pos,i)</span></span>
<span id="cb9-418"><a href="#cb9-418" aria-hidden="true" tabindex="-1"></a><span class="in">  } else {</span></span>
<span id="cb9-419"><a href="#cb9-419" aria-hidden="true" tabindex="-1"></a><span class="in">    if(length.trash ==2 &amp; "N" %in% pos.trash){</span></span>
<span id="cb9-420"><a href="#cb9-420" aria-hidden="true" tabindex="-1"></a><span class="in">      del.pos &lt;- c(del.pos,i)</span></span>
<span id="cb9-421"><a href="#cb9-421" aria-hidden="true" tabindex="-1"></a><span class="in">    }</span></span>
<span id="cb9-422"><a href="#cb9-422" aria-hidden="true" tabindex="-1"></a><span class="in">  }</span></span>
<span id="cb9-423"><a href="#cb9-423" aria-hidden="true" tabindex="-1"></a><span class="in">}</span></span>
<span id="cb9-424"><a href="#cb9-424" aria-hidden="true" tabindex="-1"></a><span class="in">for(i in 1:length(seq.tmp)){</span></span>
<span id="cb9-425"><a href="#cb9-425" aria-hidden="true" tabindex="-1"></a><span class="in">  seq.tmp[[i]] &lt;- seq.tmp[[i]][-del.pos]</span></span>
<span id="cb9-426"><a href="#cb9-426" aria-hidden="true" tabindex="-1"></a><span class="in">}</span></span>
<span id="cb9-427"><a href="#cb9-427" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-428"><a href="#cb9-428" aria-hidden="true" tabindex="-1"></a><span class="in">seqinr::write.fasta(seq.tmp,</span></span>
<span id="cb9-429"><a href="#cb9-429" aria-hidden="true" tabindex="-1"></a><span class="in">                    names = names(seq.tmp),</span></span>
<span id="cb9-430"><a href="#cb9-430" aria-hidden="true" tabindex="-1"></a><span class="in">                    nbchar = length(seq.tmp[[1]]),</span></span>
<span id="cb9-431"><a href="#cb9-431" aria-hidden="true" tabindex="-1"></a><span class="in">                    file.out = "local_file")</span></span>
<span id="cb9-432"><a href="#cb9-432" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-433"><a href="#cb9-433" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-434"><a href="#cb9-434" aria-hidden="true" tabindex="-1"></a>The resulting fasta file was processed through the <span class="co">[</span><span class="ot">IQ-TREE</span><span class="co">](http://www.iqtree.org/)</span> software to construct the genetic relatedness tree with ascertainment bias correction <span class="co">[</span><span class="ot">@Nguyen2015</span><span class="co">]</span>.</span>
<span id="cb9-435"><a href="#cb9-435" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-436"><a href="#cb9-436" aria-hidden="true" tabindex="-1"></a><span class="fu">## Prepare tree for analysis</span></span>
<span id="cb9-437"><a href="#cb9-437" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-438"><a href="#cb9-438" aria-hidden="true" tabindex="-1"></a><span class="in">```{r tree}</span></span>
<span id="cb9-439"><a href="#cb9-439" aria-hidden="true" tabindex="-1"></a><span class="in">phylo_tree &lt;- ape::read.tree("/Users/nicholaswu/Library/CloudStorage/OneDrive-MurdochUniversity/Projects - Old/LP200100331 - WNS - WSU/M - Population genetics/bat_gl7_mth3.treefile")</span></span>
<span id="cb9-440"><a href="#cb9-440" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-441"><a href="#cb9-441" aria-hidden="true" tabindex="-1"></a><span class="in">tree_tip_label &lt;- phylo_tree$tip.label # extract tree tip names</span></span>
<span id="cb9-442"><a href="#cb9-442" aria-hidden="true" tabindex="-1"></a><span class="in">ID_list   &lt;- unique(torpor_dat$ID_gene) # extract individual name from respirometry dataset</span></span>
<span id="cb9-443"><a href="#cb9-443" aria-hidden="true" tabindex="-1"></a><span class="in">#as.data.frame(setdiff(tree_tip_label, ID_list)) # check what names from the dataset not found in phylo_tree</span></span>
<span id="cb9-444"><a href="#cb9-444" aria-hidden="true" tabindex="-1"></a><span class="in"># Three outgroups in tree</span></span>
<span id="cb9-445"><a href="#cb9-445" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-446"><a href="#cb9-446" aria-hidden="true" tabindex="-1"></a><span class="in">pruned_tree &lt;- ape::drop.tip(phylo_tree, setdiff(phylo_tree$tip.label, ID_list)) # prune phylo_tree to keep species from the main dataset</span></span>
<span id="cb9-447"><a href="#cb9-447" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-448"><a href="#cb9-448" aria-hidden="true" tabindex="-1"></a><span class="in"># check if lengths match for both data and tree</span></span>
<span id="cb9-449"><a href="#cb9-449" aria-hidden="true" tabindex="-1"></a><span class="in">#length(unique(pruned_tree$tip.label)) # 27</span></span>
<span id="cb9-450"><a href="#cb9-450" aria-hidden="true" tabindex="-1"></a><span class="in">#length(unique(torpor_dat$ID_gene)) # 27</span></span>
<span id="cb9-451"><a href="#cb9-451" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-452"><a href="#cb9-452" aria-hidden="true" tabindex="-1"></a><span class="in"># Check for polytomies</span></span>
<span id="cb9-453"><a href="#cb9-453" aria-hidden="true" tabindex="-1"></a><span class="in">#ape::is.binary(pruned_tree) # TRUE</span></span>
<span id="cb9-454"><a href="#cb9-454" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-455"><a href="#cb9-455" aria-hidden="true" tabindex="-1"></a><span class="in"># Compute branch lengths</span></span>
<span id="cb9-456"><a href="#cb9-456" aria-hidden="true" tabindex="-1"></a><span class="in">pruned_tree_cal &lt;- ape::compute.brlen(pruned_tree, method = "Grafen", power = 1)</span></span>
<span id="cb9-457"><a href="#cb9-457" aria-hidden="true" tabindex="-1"></a><span class="in">#ape::is.ultrametric(pruned_tree_cal) # TRUE</span></span>
<span id="cb9-458"><a href="#cb9-458" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-459"><a href="#cb9-459" aria-hidden="true" tabindex="-1"></a><span class="in"># Create covariance matrix for analysis</span></span>
<span id="cb9-460"><a href="#cb9-460" aria-hidden="true" tabindex="-1"></a><span class="in">phylo_cor &lt;- ape::vcv(pruned_tree_cal, cor = F)</span></span>
<span id="cb9-461"><a href="#cb9-461" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-462"><a href="#cb9-462" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-463"><a href="#cb9-463" aria-hidden="true" tabindex="-1"></a><span class="fu"># Analysis</span></span>
<span id="cb9-464"><a href="#cb9-464" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-465"><a href="#cb9-465" aria-hidden="true" tabindex="-1"></a>Run phylogenetic multi-level model via <span class="co">[</span><span class="ot">`brm`</span><span class="co">](https://www.rdocumentation.org/packages/brms/versions/0.6.0/topics/brm)</span> function. We constructed four chains with 5,000 steps per chain, including 2,500-step warm-up periods, hence a total of 10,000 steps were retained to estimate posterior distributions <span class="co">[</span><span class="ot">i.e., (5,000 − 2,500) × 4 = 10,000</span><span class="co">]</span>. An alpha delta was .99 to decrease the number of divergent transitions. Fixed effects were assigned weakly informative priors following a Gaussian distribution to speed up model convergence, and Student’s t prior with three degrees of freedom was used for group-level, hierarchical effects. The degree of convergence of model was deemed to be achieved when the Gelman–Rubin statistic <span class="co">[</span><span class="ot">@Gelman1992</span><span class="co">]</span>, was 1.</span>
<span id="cb9-466"><a href="#cb9-466" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-467"><a href="#cb9-467" aria-hidden="true" tabindex="-1"></a><span class="in">```{r mod, message=FALSE, cache=TRUE, results="hide"}</span></span>
<span id="cb9-468"><a href="#cb9-468" aria-hidden="true" tabindex="-1"></a><span class="in">Tsk_mod &lt;- brms::brm(Tsk_C ~ temp_exp * pop + sex + lnMass + tor_start + </span></span>
<span id="cb9-469"><a href="#cb9-469" aria-hidden="true" tabindex="-1"></a><span class="in">                     (1 | ID) + (1 | chamber_n) + (1 | gr(ID_gene, cov = phylo)), </span></span>
<span id="cb9-470"><a href="#cb9-470" aria-hidden="true" tabindex="-1"></a><span class="in">                    data    = torpor_dat,</span></span>
<span id="cb9-471"><a href="#cb9-471" aria-hidden="true" tabindex="-1"></a><span class="in">                    data2   = list(phylo = phylo_cor),</span></span>
<span id="cb9-472"><a href="#cb9-472" aria-hidden="true" tabindex="-1"></a><span class="in">                    chains  = 4, cores = 4,</span></span>
<span id="cb9-473"><a href="#cb9-473" aria-hidden="true" tabindex="-1"></a><span class="in">                    iter    = 1e4, warmup = 5e3,</span></span>
<span id="cb9-474"><a href="#cb9-474" aria-hidden="true" tabindex="-1"></a><span class="in">                    control = list(adapt_delta = 0.99))</span></span>
<span id="cb9-475"><a href="#cb9-475" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-476"><a href="#cb9-476" aria-hidden="true" tabindex="-1"></a><span class="in">MR_mod &lt;- brms::brm(lnMR ~ temp_exp * pop + sex + lnMass + tor_start +</span></span>
<span id="cb9-477"><a href="#cb9-477" aria-hidden="true" tabindex="-1"></a><span class="in">                     (1 | ID) + (1 | chamber_n) + (1 | gr(ID_gene, cov = phylo)), </span></span>
<span id="cb9-478"><a href="#cb9-478" aria-hidden="true" tabindex="-1"></a><span class="in">                    data    = torpor_dat,</span></span>
<span id="cb9-479"><a href="#cb9-479" aria-hidden="true" tabindex="-1"></a><span class="in">                    data2   = list(phylo = phylo_cor),</span></span>
<span id="cb9-480"><a href="#cb9-480" aria-hidden="true" tabindex="-1"></a><span class="in">                    chains  = 4, cores = 4,</span></span>
<span id="cb9-481"><a href="#cb9-481" aria-hidden="true" tabindex="-1"></a><span class="in">                    iter    = 1e4, warmup = 5e3,</span></span>
<span id="cb9-482"><a href="#cb9-482" aria-hidden="true" tabindex="-1"></a><span class="in">                    control = list(adapt_delta = 0.99))</span></span>
<span id="cb9-483"><a href="#cb9-483" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-484"><a href="#cb9-484" aria-hidden="true" tabindex="-1"></a><span class="in">MR_mod2 &lt;- brms::brm(lnMR ~ temp_exp * pop + sex + lnMass + Tsk_C + tor_start +</span></span>
<span id="cb9-485"><a href="#cb9-485" aria-hidden="true" tabindex="-1"></a><span class="in">                     (1 | ID) + (1 | chamber_n) + (1 | gr(ID_gene, cov = phylo)), </span></span>
<span id="cb9-486"><a href="#cb9-486" aria-hidden="true" tabindex="-1"></a><span class="in">                    data    = torpor_dat,</span></span>
<span id="cb9-487"><a href="#cb9-487" aria-hidden="true" tabindex="-1"></a><span class="in">                    data2   = list(phylo = phylo_cor),</span></span>
<span id="cb9-488"><a href="#cb9-488" aria-hidden="true" tabindex="-1"></a><span class="in">                    chains  = 4, cores = 4,</span></span>
<span id="cb9-489"><a href="#cb9-489" aria-hidden="true" tabindex="-1"></a><span class="in">                    iter    = 1e4, warmup = 5e3,</span></span>
<span id="cb9-490"><a href="#cb9-490" aria-hidden="true" tabindex="-1"></a><span class="in">                    control = list(adapt_delta = 0.99))</span></span>
<span id="cb9-491"><a href="#cb9-491" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-492"><a href="#cb9-492" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_mod &lt;- brms::brm(lnEWL ~ temp_exp * pop + sex + lnMass + tor_start +</span></span>
<span id="cb9-493"><a href="#cb9-493" aria-hidden="true" tabindex="-1"></a><span class="in">                      (1 | ID) + (1 | chamber_n) + (1 | gr(ID_gene, cov = phylo)), </span></span>
<span id="cb9-494"><a href="#cb9-494" aria-hidden="true" tabindex="-1"></a><span class="in">                    data    = torpor_dat,</span></span>
<span id="cb9-495"><a href="#cb9-495" aria-hidden="true" tabindex="-1"></a><span class="in">                    data2   = list(phylo = phylo_cor),</span></span>
<span id="cb9-496"><a href="#cb9-496" aria-hidden="true" tabindex="-1"></a><span class="in">                    chains  = 4, cores = 4,</span></span>
<span id="cb9-497"><a href="#cb9-497" aria-hidden="true" tabindex="-1"></a><span class="in">                    iter    = 1e4, warmup = 5e3,</span></span>
<span id="cb9-498"><a href="#cb9-498" aria-hidden="true" tabindex="-1"></a><span class="in">                    control = list(adapt_delta = 0.99))</span></span>
<span id="cb9-499"><a href="#cb9-499" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-500"><a href="#cb9-500" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-501"><a href="#cb9-501" aria-hidden="true" tabindex="-1"></a><span class="in">```{r Fig S3, echo = FALSE, fig.width = 9, fig.height = 3, fig.align = 'center'}</span></span>
<span id="cb9-502"><a href="#cb9-502" aria-hidden="true" tabindex="-1"></a><span class="in">bayesplot::color_scheme_set("darkgray")</span></span>
<span id="cb9-503"><a href="#cb9-503" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-504"><a href="#cb9-504" aria-hidden="true" tabindex="-1"></a><span class="in">Tsk_mod_pp &lt;- brms::pp_check(Tsk_mod, type = "scatter_avg") + mytheme()</span></span>
<span id="cb9-505"><a href="#cb9-505" aria-hidden="true" tabindex="-1"></a><span class="in">MR_mod_pp  &lt;- brms::pp_check(MR_mod, type = "scatter_avg") + mytheme()</span></span>
<span id="cb9-506"><a href="#cb9-506" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_mod_pp &lt;- brms::pp_check(EWL_mod, type = "scatter_avg") + mytheme()</span></span>
<span id="cb9-507"><a href="#cb9-507" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-508"><a href="#cb9-508" aria-hidden="true" tabindex="-1"></a><span class="in">cowplot::plot_grid(Tsk_mod_pp, MR_mod_pp, EWL_mod_pp, ncol = 3, labels = c('a', 'b', 'c', 'd'))</span></span>
<span id="cb9-509"><a href="#cb9-509" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-510"><a href="#cb9-510" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-511"><a href="#cb9-511" aria-hidden="true" tabindex="-1"></a>**Fig. S3** | Scatterplots of the observed data (y) vs the average simulated data (y~rep~) from the posterior predictive distribution for the (**a**) *T*~sk~ model, (**b**) the TMR model, and (**c**) the TEWL model. Dashed line represents a slope of 1. </span>
<span id="cb9-512"><a href="#cb9-512" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-513"><a href="#cb9-513" aria-hidden="true" tabindex="-1"></a><span class="fu">## Model output</span></span>
<span id="cb9-514"><a href="#cb9-514" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-515"><a href="#cb9-515" aria-hidden="true" tabindex="-1"></a>::: {.panel-tabset .nav-pills}</span>
<span id="cb9-516"><a href="#cb9-516" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-517"><a href="#cb9-517" aria-hidden="true" tabindex="-1"></a><span class="fu">### Table S2 - *T*~sk~ model</span></span>
<span id="cb9-518"><a href="#cb9-518" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-519"><a href="#cb9-519" aria-hidden="true" tabindex="-1"></a>**Table S2** | Summary statistics testing the effect of air temperature on the bat skin surface temperature (*T*~sk~). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%), which includes the intercept ($\beta_0$), exposure temperature (*T*~a~), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass, the time since onset of torpor, and the interaction between *T*~a~ and population. Group-level effects include the standard deviations ($\sigma$) for chamber-level effects ($\sigma_{chamber}^2$), individual-level effects ($\sigma_{ID}^2$), and phylogenetic relatedness ($\sigma_{phylogeny}^2$). Heritability was calculated as $h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)$. The $R_{marginal}^2$ representing variance explained by fixed effects only is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(Tsk_mod))[2,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(Tsk_mod))[2,2], 4) * 100)`</span>%, while the $R_{conditional}^2$ representing variance explained by both fixed and group-level effects is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(Tsk_mod))[1,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(Tsk_mod))[1,2], 4) * 100)`</span>%.</span>
<span id="cb9-520"><a href="#cb9-520" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-521"><a href="#cb9-521" aria-hidden="true" tabindex="-1"></a><span class="in">```{r tableS2, echo = FALSE}</span></span>
<span id="cb9-522"><a href="#cb9-522" aria-hidden="true" tabindex="-1"></a><span class="in"># Fixed effect</span></span>
<span id="cb9-523"><a href="#cb9-523" aria-hidden="true" tabindex="-1"></a><span class="in">fef &lt;- brms::fixef(Tsk_mod) %&gt;% </span></span>
<span id="cb9-524"><a href="#cb9-524" aria-hidden="true" tabindex="-1"></a><span class="in">  as.data.frame(.) %&gt;%</span></span>
<span id="cb9-525"><a href="#cb9-525" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-526"><a href="#cb9-526" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = str_replace(Parameter, "M", "-")) %&gt;% </span></span>
<span id="cb9-527"><a href="#cb9-527" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = case_when(</span></span>
<span id="cb9-528"><a href="#cb9-528" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "Intercept" ~ "$\\beta_0$",</span></span>
<span id="cb9-529"><a href="#cb9-529" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp" ~ "*T*~a~",</span></span>
<span id="cb9-530"><a href="#cb9-530" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "popYessabah" ~ "Population - Yessabah",</span></span>
<span id="cb9-531"><a href="#cb9-531" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "sexmale" ~ "Sex - Male",</span></span>
<span id="cb9-532"><a href="#cb9-532" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "ln-ass" ~ "lnMass",</span></span>
<span id="cb9-533"><a href="#cb9-533" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "tor_start" ~ "Torpor onset",</span></span>
<span id="cb9-534"><a href="#cb9-534" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp:popYessabah" ~ "*T*~a~: Population - Yessabah",</span></span>
<span id="cb9-535"><a href="#cb9-535" aria-hidden="true" tabindex="-1"></a><span class="in">    )) %&gt;% </span></span>
<span id="cb9-536"><a href="#cb9-536" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Fixed effects**", .before = 1)</span></span>
<span id="cb9-537"><a href="#cb9-537" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-538"><a href="#cb9-538" aria-hidden="true" tabindex="-1"></a><span class="in"># Random effect</span></span>
<span id="cb9-539"><a href="#cb9-539" aria-hidden="true" tabindex="-1"></a><span class="in">ref &lt;- summary(Tsk_mod)$random %&gt;% </span></span>
<span id="cb9-540"><a href="#cb9-540" aria-hidden="true" tabindex="-1"></a><span class="in">  bind_rows() %&gt;% # unlist</span></span>
<span id="cb9-541"><a href="#cb9-541" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-542"><a href="#cb9-542" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(1:5) %&gt;% </span></span>
<span id="cb9-543"><a href="#cb9-543" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = c("$\\sigma_{chamber}^2$", </span></span>
<span id="cb9-544"><a href="#cb9-544" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{ID}^2$",</span></span>
<span id="cb9-545"><a href="#cb9-545" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{phylogeny}^2$")) %&gt;%   </span></span>
<span id="cb9-546"><a href="#cb9-546" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = 4, </span></span>
<span id="cb9-547"><a href="#cb9-547" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = 5) %&gt;% </span></span>
<span id="cb9-548"><a href="#cb9-548" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Group-level effects**", .before = 1)</span></span>
<span id="cb9-549"><a href="#cb9-549" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-550"><a href="#cb9-550" aria-hidden="true" tabindex="-1"></a><span class="in"># Heritability</span></span>
<span id="cb9-551"><a href="#cb9-551" aria-hidden="true" tabindex="-1"></a><span class="in">Tsk_h2 &lt;- hypothesis(Tsk_mod, "sd_ID_gene__Intercept^2 / (sd_ID_gene__Intercept^2 + sigma^2) = 0", class = NULL)$hypothesis %&gt;% </span></span>
<span id="cb9-552"><a href="#cb9-552" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(-c(1,6:8)) %&gt;% </span></span>
<span id="cb9-553"><a href="#cb9-553" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = "$h^2$") %&gt;% </span></span>
<span id="cb9-554"><a href="#cb9-554" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = CI.Lower, </span></span>
<span id="cb9-555"><a href="#cb9-555" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = CI.Upper) %&gt;% </span></span>
<span id="cb9-556"><a href="#cb9-556" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Heritability**", .before = 1)</span></span>
<span id="cb9-557"><a href="#cb9-557" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-558"><a href="#cb9-558" aria-hidden="true" tabindex="-1"></a><span class="in"># Render table</span></span>
<span id="cb9-559"><a href="#cb9-559" aria-hidden="true" tabindex="-1"></a><span class="in">bind_rows(fef, ref, Tsk_h2) %&gt;% </span></span>
<span id="cb9-560"><a href="#cb9-560" aria-hidden="true" tabindex="-1"></a><span class="in">  remove_rownames() %&gt;% </span></span>
<span id="cb9-561"><a href="#cb9-561" aria-hidden="true" tabindex="-1"></a><span class="in">  knitr::kable(digits = 2)</span></span>
<span id="cb9-562"><a href="#cb9-562" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-563"><a href="#cb9-563" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-564"><a href="#cb9-564" aria-hidden="true" tabindex="-1"></a><span class="fu">### Table S3 - TMR model</span></span>
<span id="cb9-565"><a href="#cb9-565" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-566"><a href="#cb9-566" aria-hidden="true" tabindex="-1"></a>**Table S3** | Summary statistics testing the effect of air temperature on torpor metabolic rate (TMR). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%), which includes the intercept ($\beta_0$), exposure temperature (*T*~a~), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass, the time since onset of torpor, and the interaction between *T*~a~ and population. Group-level effects include the standard deviations ($\sigma$) for chamber-level effects ($\sigma_{chamber}^2$), individual-level effects ($\sigma_{ID}^2$), and phylogenetic relatedness ($\sigma_{phylogeny}^2$). Heritability was calculated as $h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)$. The $R_{marginal}^2$ representing variance explained by fixed effects only is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[2,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[2,2], 4) * 100)`</span>%, while the $R_{conditional}^2$ representing variance explained by both fixed and group-level effects is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[1,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[1,2], 4) * 100)`</span>%.</span>
<span id="cb9-567"><a href="#cb9-567" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-568"><a href="#cb9-568" aria-hidden="true" tabindex="-1"></a><span class="in">```{r tableS3, echo = FALSE}</span></span>
<span id="cb9-569"><a href="#cb9-569" aria-hidden="true" tabindex="-1"></a><span class="in"># Fixed effect</span></span>
<span id="cb9-570"><a href="#cb9-570" aria-hidden="true" tabindex="-1"></a><span class="in">fef &lt;- brms::fixef(MR_mod) %&gt;% </span></span>
<span id="cb9-571"><a href="#cb9-571" aria-hidden="true" tabindex="-1"></a><span class="in">  as.data.frame(.) %&gt;%</span></span>
<span id="cb9-572"><a href="#cb9-572" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-573"><a href="#cb9-573" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = str_replace(Parameter, "M", "-")) %&gt;% </span></span>
<span id="cb9-574"><a href="#cb9-574" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = case_when(</span></span>
<span id="cb9-575"><a href="#cb9-575" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "Intercept" ~ "$\\beta_0$",</span></span>
<span id="cb9-576"><a href="#cb9-576" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp" ~ "*T*~a~",</span></span>
<span id="cb9-577"><a href="#cb9-577" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "popYessabah" ~ "Population - Yessabah",</span></span>
<span id="cb9-578"><a href="#cb9-578" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "sexmale" ~ "Sex - Male",</span></span>
<span id="cb9-579"><a href="#cb9-579" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "ln-ass" ~ "lnMass",</span></span>
<span id="cb9-580"><a href="#cb9-580" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "tor_start" ~ "Torpor onset",</span></span>
<span id="cb9-581"><a href="#cb9-581" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp:popYessabah" ~ "*T*~a~: Population - Yessabah",</span></span>
<span id="cb9-582"><a href="#cb9-582" aria-hidden="true" tabindex="-1"></a><span class="in">    )) %&gt;% </span></span>
<span id="cb9-583"><a href="#cb9-583" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Fixed effects**", .before = 1)</span></span>
<span id="cb9-584"><a href="#cb9-584" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-585"><a href="#cb9-585" aria-hidden="true" tabindex="-1"></a><span class="in"># Random effect</span></span>
<span id="cb9-586"><a href="#cb9-586" aria-hidden="true" tabindex="-1"></a><span class="in">ref &lt;- summary(MR_mod)$random %&gt;% </span></span>
<span id="cb9-587"><a href="#cb9-587" aria-hidden="true" tabindex="-1"></a><span class="in">  bind_rows() %&gt;% # unlist</span></span>
<span id="cb9-588"><a href="#cb9-588" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-589"><a href="#cb9-589" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(1:5) %&gt;% </span></span>
<span id="cb9-590"><a href="#cb9-590" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = c("$\\sigma_{chamber}^2$", </span></span>
<span id="cb9-591"><a href="#cb9-591" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{ID}^2$",</span></span>
<span id="cb9-592"><a href="#cb9-592" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{phylogeny}^2$")) %&gt;%   </span></span>
<span id="cb9-593"><a href="#cb9-593" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = 4, </span></span>
<span id="cb9-594"><a href="#cb9-594" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = 5) %&gt;% </span></span>
<span id="cb9-595"><a href="#cb9-595" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Group-level effects**", .before = 1)</span></span>
<span id="cb9-596"><a href="#cb9-596" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-597"><a href="#cb9-597" aria-hidden="true" tabindex="-1"></a><span class="in"># Heritability</span></span>
<span id="cb9-598"><a href="#cb9-598" aria-hidden="true" tabindex="-1"></a><span class="in">MR_h2 &lt;- hypothesis(MR_mod, "sd_ID_gene__Intercept^2 / (sd_ID_gene__Intercept^2 + sigma^2) = 0", class = NULL)$hypothesis %&gt;% </span></span>
<span id="cb9-599"><a href="#cb9-599" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(-c(1,6:8)) %&gt;% </span></span>
<span id="cb9-600"><a href="#cb9-600" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = "$h^2$") %&gt;% </span></span>
<span id="cb9-601"><a href="#cb9-601" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = CI.Lower, </span></span>
<span id="cb9-602"><a href="#cb9-602" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = CI.Upper) %&gt;% </span></span>
<span id="cb9-603"><a href="#cb9-603" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Heritability**", .before = 1)</span></span>
<span id="cb9-604"><a href="#cb9-604" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-605"><a href="#cb9-605" aria-hidden="true" tabindex="-1"></a><span class="in"># Render table</span></span>
<span id="cb9-606"><a href="#cb9-606" aria-hidden="true" tabindex="-1"></a><span class="in">bind_rows(fef, ref, MR_h2) %&gt;% </span></span>
<span id="cb9-607"><a href="#cb9-607" aria-hidden="true" tabindex="-1"></a><span class="in">  remove_rownames() %&gt;% </span></span>
<span id="cb9-608"><a href="#cb9-608" aria-hidden="true" tabindex="-1"></a><span class="in">  knitr::kable(digits = 2)</span></span>
<span id="cb9-609"><a href="#cb9-609" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-610"><a href="#cb9-610" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-611"><a href="#cb9-611" aria-hidden="true" tabindex="-1"></a><span class="fu">### Table S4 - TEWL model</span></span>
<span id="cb9-612"><a href="#cb9-612" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-613"><a href="#cb9-613" aria-hidden="true" tabindex="-1"></a>**Table S4** | Summary statistics testing the effect of air temperature on torpor evaporative water loss (TEWL). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%), which includes the intercept ($\beta_0$), exposure temperature (*T*~a~), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass,the time since onset of torpor, and the interaction between *T*~a~ and population. Group-level effects include the standard deviations ($\sigma$) for chamber-level effects ($\sigma_{chamber}^2$), individual-level effects ($\sigma_{ID}^2$), and phylogenetic relatedness ($\sigma_{phylogeny}^2$). Heritability was calculated as $h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)$. The $R_{marginal}^2$ representing variance explained by fixed effects only is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(EWL_mod))[2,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(EWL_mod))[2,2], 4) * 100)`</span>%, while the $R_{conditional}^2$ representing variance explained by both fixed and group-level effects is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(EWL_mod))[1,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(EWL_mod))[1,2], 4) * 100)`</span>%.</span>
<span id="cb9-614"><a href="#cb9-614" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-615"><a href="#cb9-615" aria-hidden="true" tabindex="-1"></a><span class="in">```{r tableS4, echo = FALSE}</span></span>
<span id="cb9-616"><a href="#cb9-616" aria-hidden="true" tabindex="-1"></a><span class="in"># Fixed effect</span></span>
<span id="cb9-617"><a href="#cb9-617" aria-hidden="true" tabindex="-1"></a><span class="in">fef &lt;- brms::fixef(EWL_mod) %&gt;% </span></span>
<span id="cb9-618"><a href="#cb9-618" aria-hidden="true" tabindex="-1"></a><span class="in">  as.data.frame(.) %&gt;%</span></span>
<span id="cb9-619"><a href="#cb9-619" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-620"><a href="#cb9-620" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = str_replace(Parameter, "M", "-")) %&gt;% </span></span>
<span id="cb9-621"><a href="#cb9-621" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = case_when(</span></span>
<span id="cb9-622"><a href="#cb9-622" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "Intercept" ~ "$\\beta_0$",</span></span>
<span id="cb9-623"><a href="#cb9-623" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp" ~ "*T*~a~",</span></span>
<span id="cb9-624"><a href="#cb9-624" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "popYessabah" ~ "Population - Yessabah",</span></span>
<span id="cb9-625"><a href="#cb9-625" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "sexmale" ~ "Sex - Male",</span></span>
<span id="cb9-626"><a href="#cb9-626" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "ln-ass" ~ "lnMass",</span></span>
<span id="cb9-627"><a href="#cb9-627" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "tor_start" ~ "Torpor onset",</span></span>
<span id="cb9-628"><a href="#cb9-628" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp:popYessabah" ~ "*T*~a~: Population - Yessabah",</span></span>
<span id="cb9-629"><a href="#cb9-629" aria-hidden="true" tabindex="-1"></a><span class="in">    )) %&gt;% </span></span>
<span id="cb9-630"><a href="#cb9-630" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Fixed effects**", .before = 1)</span></span>
<span id="cb9-631"><a href="#cb9-631" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-632"><a href="#cb9-632" aria-hidden="true" tabindex="-1"></a><span class="in"># Random effect</span></span>
<span id="cb9-633"><a href="#cb9-633" aria-hidden="true" tabindex="-1"></a><span class="in">ref &lt;- summary(EWL_mod)$random %&gt;% </span></span>
<span id="cb9-634"><a href="#cb9-634" aria-hidden="true" tabindex="-1"></a><span class="in">  bind_rows() %&gt;% # unlist</span></span>
<span id="cb9-635"><a href="#cb9-635" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-636"><a href="#cb9-636" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(1:5) %&gt;% </span></span>
<span id="cb9-637"><a href="#cb9-637" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = c("$\\sigma_{chamber}^2$", </span></span>
<span id="cb9-638"><a href="#cb9-638" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{ID}^2$",</span></span>
<span id="cb9-639"><a href="#cb9-639" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{phylogeny}^2$")) %&gt;%   </span></span>
<span id="cb9-640"><a href="#cb9-640" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = 4, </span></span>
<span id="cb9-641"><a href="#cb9-641" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = 5) %&gt;% </span></span>
<span id="cb9-642"><a href="#cb9-642" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Group-level effects**", .before = 1)</span></span>
<span id="cb9-643"><a href="#cb9-643" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-644"><a href="#cb9-644" aria-hidden="true" tabindex="-1"></a><span class="in"># Heritability</span></span>
<span id="cb9-645"><a href="#cb9-645" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_h2 &lt;- hypothesis(EWL_mod, "sd_ID_gene__Intercept^2 / (sd_ID_gene__Intercept^2 + sigma^2) = 0", class = NULL)$hypothesis %&gt;% </span></span>
<span id="cb9-646"><a href="#cb9-646" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(-c(1,6:8)) %&gt;% </span></span>
<span id="cb9-647"><a href="#cb9-647" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = "$h^2$") %&gt;% </span></span>
<span id="cb9-648"><a href="#cb9-648" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = CI.Lower, </span></span>
<span id="cb9-649"><a href="#cb9-649" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = CI.Upper) %&gt;% </span></span>
<span id="cb9-650"><a href="#cb9-650" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Heritability**", .before = 1)</span></span>
<span id="cb9-651"><a href="#cb9-651" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-652"><a href="#cb9-652" aria-hidden="true" tabindex="-1"></a><span class="in"># Render table</span></span>
<span id="cb9-653"><a href="#cb9-653" aria-hidden="true" tabindex="-1"></a><span class="in">bind_rows(fef, ref, EWL_h2) %&gt;% </span></span>
<span id="cb9-654"><a href="#cb9-654" aria-hidden="true" tabindex="-1"></a><span class="in">  remove_rownames() %&gt;% </span></span>
<span id="cb9-655"><a href="#cb9-655" aria-hidden="true" tabindex="-1"></a><span class="in">  knitr::kable(digits = 2)</span></span>
<span id="cb9-656"><a href="#cb9-656" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-657"><a href="#cb9-657" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-658"><a href="#cb9-658" aria-hidden="true" tabindex="-1"></a><span class="fu">### Table S5 - TMR + *T*~sk~ model</span></span>
<span id="cb9-659"><a href="#cb9-659" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-660"><a href="#cb9-660" aria-hidden="true" tabindex="-1"></a>**Table S5** | Summary statistics testing the effect of air temperature and skin surface temperature on torpor metabolic rate (TMR). Mean parameter estimates, estimate error, and 95% Bayesian credible intervals (lower 2.5% and upper 97.5%). The model includes the intercept ($\beta_0$), exposure temperature (*T*~a~), population (Jenolan, Yessabah), sex (female, male), the logarithm-transformed body mass, *T*~sk~, the time since onset of torpor, and the interaction between *T*~a~ and population. Group-level effects include the standard deviations ($\sigma$) for chamber-level effects ($\sigma_{chamber}^2$), individual-level effects ($\sigma_{ID}^2$), and phylogenetic relatedness ($\sigma_{phylogeny}^2$). Heritability was calculated as $h^2 = \sigma_{phylogeny}^2 / (\sigma_{phylogeny}^2 + \sigma^2)$. The $R_{marginal}^2$ representing variance explained by fixed effects only is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[2,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[2,2], 4) * 100)`</span>%, while the $R_{conditional}^2$ representing variance explained by both fixed and group-level effects is <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[1,1], 4) * 100)`</span> ± <span class="in">`r as.character(round(data.frame(performance::r2_bayes(MR_mod))[1,2], 4) * 100)`</span>%.</span>
<span id="cb9-661"><a href="#cb9-661" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-662"><a href="#cb9-662" aria-hidden="true" tabindex="-1"></a><span class="in">```{r tableS5, echo = FALSE}</span></span>
<span id="cb9-663"><a href="#cb9-663" aria-hidden="true" tabindex="-1"></a><span class="in"># Fixed effect</span></span>
<span id="cb9-664"><a href="#cb9-664" aria-hidden="true" tabindex="-1"></a><span class="in">fef &lt;- brms::fixef(MR_mod2) %&gt;% </span></span>
<span id="cb9-665"><a href="#cb9-665" aria-hidden="true" tabindex="-1"></a><span class="in">  as.data.frame(.) %&gt;%</span></span>
<span id="cb9-666"><a href="#cb9-666" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-667"><a href="#cb9-667" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = str_replace(Parameter, "M", "-")) %&gt;% </span></span>
<span id="cb9-668"><a href="#cb9-668" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = case_when(</span></span>
<span id="cb9-669"><a href="#cb9-669" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "Intercept" ~ "$\\beta_0$",</span></span>
<span id="cb9-670"><a href="#cb9-670" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp" ~ "*T*~a~",</span></span>
<span id="cb9-671"><a href="#cb9-671" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "popYessabah" ~ "Population - Yessabah",</span></span>
<span id="cb9-672"><a href="#cb9-672" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "sexmale" ~ "Sex - Male",</span></span>
<span id="cb9-673"><a href="#cb9-673" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "ln-ass" ~ "lnMass",</span></span>
<span id="cb9-674"><a href="#cb9-674" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "Tsk_C" ~ "*T*~sk~",</span></span>
<span id="cb9-675"><a href="#cb9-675" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "tor_start" ~ "Torpor onset",</span></span>
<span id="cb9-676"><a href="#cb9-676" aria-hidden="true" tabindex="-1"></a><span class="in">    Parameter == "temp_exp:popYessabah" ~ "*T*~a~: Population - Yessabah",</span></span>
<span id="cb9-677"><a href="#cb9-677" aria-hidden="true" tabindex="-1"></a><span class="in">    )) %&gt;% </span></span>
<span id="cb9-678"><a href="#cb9-678" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Fixed effects**", .before = 1)</span></span>
<span id="cb9-679"><a href="#cb9-679" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-680"><a href="#cb9-680" aria-hidden="true" tabindex="-1"></a><span class="in"># Random effect</span></span>
<span id="cb9-681"><a href="#cb9-681" aria-hidden="true" tabindex="-1"></a><span class="in">ref &lt;- summary(MR_mod2)$random %&gt;% </span></span>
<span id="cb9-682"><a href="#cb9-682" aria-hidden="true" tabindex="-1"></a><span class="in">  bind_rows() %&gt;% # unlist</span></span>
<span id="cb9-683"><a href="#cb9-683" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::rownames_to_column("Parameter") %&gt;% </span></span>
<span id="cb9-684"><a href="#cb9-684" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(1:5) %&gt;% </span></span>
<span id="cb9-685"><a href="#cb9-685" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = c("$\\sigma_{chamber}^2$", </span></span>
<span id="cb9-686"><a href="#cb9-686" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{ID}^2$",</span></span>
<span id="cb9-687"><a href="#cb9-687" aria-hidden="true" tabindex="-1"></a><span class="in">                              "$\\sigma_{phylogeny}^2$")) %&gt;%   </span></span>
<span id="cb9-688"><a href="#cb9-688" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = 4, </span></span>
<span id="cb9-689"><a href="#cb9-689" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = 5) %&gt;% </span></span>
<span id="cb9-690"><a href="#cb9-690" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Group-level effects**", .before = 1)</span></span>
<span id="cb9-691"><a href="#cb9-691" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-692"><a href="#cb9-692" aria-hidden="true" tabindex="-1"></a><span class="in">#Heritability</span></span>
<span id="cb9-693"><a href="#cb9-693" aria-hidden="true" tabindex="-1"></a><span class="in">MR2_h2 &lt;- hypothesis(MR_mod2, "sd_ID_gene__Intercept^2 / (sd_ID_gene__Intercept^2 + sigma^2) = 0", class = NULL)$hypothesis %&gt;% </span></span>
<span id="cb9-694"><a href="#cb9-694" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::select(-c(1,6:8)) %&gt;% </span></span>
<span id="cb9-695"><a href="#cb9-695" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(Parameter = "$h^2$") %&gt;% </span></span>
<span id="cb9-696"><a href="#cb9-696" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::rename(Q2.5  = CI.Lower, </span></span>
<span id="cb9-697"><a href="#cb9-697" aria-hidden="true" tabindex="-1"></a><span class="in">                Q97.5 = CI.Upper) %&gt;% </span></span>
<span id="cb9-698"><a href="#cb9-698" aria-hidden="true" tabindex="-1"></a><span class="in">  tibble::add_row(Parameter = "**Heritability**", .before = 1)</span></span>
<span id="cb9-699"><a href="#cb9-699" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-700"><a href="#cb9-700" aria-hidden="true" tabindex="-1"></a><span class="in"># Render table</span></span>
<span id="cb9-701"><a href="#cb9-701" aria-hidden="true" tabindex="-1"></a><span class="in">bind_rows(fef, ref, MR2_h2) %&gt;% </span></span>
<span id="cb9-702"><a href="#cb9-702" aria-hidden="true" tabindex="-1"></a><span class="in">  remove_rownames() %&gt;% </span></span>
<span id="cb9-703"><a href="#cb9-703" aria-hidden="true" tabindex="-1"></a><span class="in">  knitr::kable(digits = 2)</span></span>
<span id="cb9-704"><a href="#cb9-704" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-705"><a href="#cb9-705" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-706"><a href="#cb9-706" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb9-707"><a href="#cb9-707" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-708"><a href="#cb9-708" aria-hidden="true" tabindex="-1"></a>***</span>
<span id="cb9-709"><a href="#cb9-709" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-710"><a href="#cb9-710" aria-hidden="true" tabindex="-1"></a><span class="fu"># Figures</span></span>
<span id="cb9-711"><a href="#cb9-711" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-712"><a href="#cb9-712" aria-hidden="true" tabindex="-1"></a>Code to produce the main document figures is provided below. Figures produced were further modified in <span class="co">[</span><span class="ot">Adobe Illustrator</span><span class="co">](https://www.adobe.com/au/products/illustrator.html)</span> for publication.</span>
<span id="cb9-713"><a href="#cb9-713" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-714"><a href="#cb9-714" aria-hidden="true" tabindex="-1"></a><span class="fu">## Figure 1 - Study sites</span></span>
<span id="cb9-715"><a href="#cb9-715" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-716"><a href="#cb9-716" aria-hidden="true" tabindex="-1"></a>Download elevation map from <span class="in">`rnaturalearth`</span> and convert to dataframe.</span>
<span id="cb9-717"><a href="#cb9-717" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-718"><a href="#cb9-718" aria-hidden="true" tabindex="-1"></a><span class="in">```{r map, cache=TRUE, results="hide"}</span></span>
<span id="cb9-719"><a href="#cb9-719" aria-hidden="true" tabindex="-1"></a><span class="in"># Australia map</span></span>
<span id="cb9-720"><a href="#cb9-720" aria-hidden="true" tabindex="-1"></a><span class="in">aus_map &lt;- rnaturalearth::ne_states(country = 'Australia', returnclass = 'sf')</span></span>
<span id="cb9-721"><a href="#cb9-721" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-722"><a href="#cb9-722" aria-hidden="true" tabindex="-1"></a><span class="in"># Elevation map</span></span>
<span id="cb9-723"><a href="#cb9-723" aria-hidden="true" tabindex="-1"></a><span class="in">aus_elev_sp &lt;- elevatr::get_elev_raster(locations = aus_map, z = 7, clip = "locations") # 1km resolution or 30 arc sec</span></span>
<span id="cb9-724"><a href="#cb9-724" aria-hidden="true" tabindex="-1"></a><span class="in">#aus_elev_sp &lt;- readRDS(file.path(data_path, "/aus_elev_sp.rds"))</span></span>
<span id="cb9-725"><a href="#cb9-725" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-726"><a href="#cb9-726" aria-hidden="true" tabindex="-1"></a><span class="in">aus_elev_df &lt;- raster::as.data.frame(aus_elev_sp, xy = TRUE)</span></span>
<span id="cb9-727"><a href="#cb9-727" aria-hidden="true" tabindex="-1"></a><span class="in">colnames(aus_elev_df)[3] &lt;- "elevation"</span></span>
<span id="cb9-728"><a href="#cb9-728" aria-hidden="true" tabindex="-1"></a><span class="in">aus_elev_df &lt;- aus_elev_df[complete.cases(aus_elev_df),] # Remove NAs</span></span>
<span id="cb9-729"><a href="#cb9-729" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-730"><a href="#cb9-730" aria-hidden="true" tabindex="-1"></a><span class="in">aus_elev_df_2 &lt;- aus_elev_df %&gt;%</span></span>
<span id="cb9-731"><a href="#cb9-731" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::filter(x &gt;= 143 &amp; y &lt;= -24)</span></span>
<span id="cb9-732"><a href="#cb9-732" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-733"><a href="#cb9-733" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-734"><a href="#cb9-734" aria-hidden="true" tabindex="-1"></a>Produce eastcoast map with study sites, and temperature profiles.</span>
<span id="cb9-735"><a href="#cb9-735" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-736"><a href="#cb9-736" aria-hidden="true" tabindex="-1"></a><span class="in">```{r Fig 1, fig.align='center', fig.height=4, fig.width=8, fig.show='hide'}</span></span>
<span id="cb9-737"><a href="#cb9-737" aria-hidden="true" tabindex="-1"></a><span class="in"># Miniopterus distribution</span></span>
<span id="cb9-738"><a href="#cb9-738" aria-hidden="true" tabindex="-1"></a><span class="in">range_map &lt;- raster::shapefile(file.path(data_path, "data/Miniopterus_orianae_oceanensis/Miniopterus_orianae_oceanensis.shp"))</span></span>
<span id="cb9-739"><a href="#cb9-739" aria-hidden="true" tabindex="-1"></a><span class="in">range_map &lt;- sp::spTransform(range_map, raster::crs("+proj=longlat +datum=WGS84 +no_defs"))</span></span>
<span id="cb9-740"><a href="#cb9-740" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-741"><a href="#cb9-741" aria-hidden="true" tabindex="-1"></a><span class="in"># Sites</span></span>
<span id="cb9-742"><a href="#cb9-742" aria-hidden="true" tabindex="-1"></a><span class="in">resp_sites_df &lt;- data.frame(x = c(150.018564, 152.688690), </span></span>
<span id="cb9-743"><a href="#cb9-743" aria-hidden="true" tabindex="-1"></a><span class="in">                            y = c(-33.803127, -31.093943), </span></span>
<span id="cb9-744"><a href="#cb9-744" aria-hidden="true" tabindex="-1"></a><span class="in">                            site = c("Jenolan","Yessabah"))</span></span>
<span id="cb9-745"><a href="#cb9-745" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-746"><a href="#cb9-746" aria-hidden="true" tabindex="-1"></a><span class="in">map_plot &lt;- ggplot() +</span></span>
<span id="cb9-747"><a href="#cb9-747" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_polygon(data = range_map, aes(x = long, y = lat, group = group), colour = NA, fill = "#f595a6") +</span></span>
<span id="cb9-748"><a href="#cb9-748" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_raster(data = aus_elev_df_2, aes(x = x, y = y, fill = elevation), alpha = 0.5, show.legend = F) +</span></span>
<span id="cb9-749"><a href="#cb9-749" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_gradient2(low = "white", mid = "white", high = "black", midpoint = 0) +</span></span>
<span id="cb9-750"><a href="#cb9-750" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_sf(data = aus_map, color = "#8C8C8C", fill = NA) +</span></span>
<span id="cb9-751"><a href="#cb9-751" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = resp_sites_df, aes(x = x, y = y), shape = 21, colour = "black", fill = "white", size = 3) +</span></span>
<span id="cb9-752"><a href="#cb9-752" aria-hidden="true" tabindex="-1"></a><span class="in">  xlim(143, 155) + ylim(-39.5, -24) +</span></span>
<span id="cb9-753"><a href="#cb9-753" aria-hidden="true" tabindex="-1"></a><span class="in">  coord_sf() +</span></span>
<span id="cb9-754"><a href="#cb9-754" aria-hidden="true" tabindex="-1"></a><span class="in">  theme_void()</span></span>
<span id="cb9-755"><a href="#cb9-755" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-756"><a href="#cb9-756" aria-hidden="true" tabindex="-1"></a><span class="in"># Load all files </span></span>
<span id="cb9-757"><a href="#cb9-757" aria-hidden="true" tabindex="-1"></a><span class="in">HOBO_list  &lt;- list.files(path = file.path(data_path, "data/"), pattern = "\\HOBO.csv$", full.names = T)</span></span>
<span id="cb9-758"><a href="#cb9-758" aria-hidden="true" tabindex="-1"></a><span class="in">HOBO_list2 &lt;- data.frame(file_name = HOBO_list, id = as.character(1:length(HOBO_list))) # id for joining</span></span>
<span id="cb9-759"><a href="#cb9-759" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-760"><a href="#cb9-760" aria-hidden="true" tabindex="-1"></a><span class="in">HOBO_dat &lt;- HOBO_list %&gt;% </span></span>
<span id="cb9-761"><a href="#cb9-761" aria-hidden="true" tabindex="-1"></a><span class="in">  lapply(read_csv) %&gt;% # read all files at once</span></span>
<span id="cb9-762"><a href="#cb9-762" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::bind_rows(.id = "id") %&gt;% # bind all tables into one object, and give id for each</span></span>
<span id="cb9-763"><a href="#cb9-763" aria-hidden="true" tabindex="-1"></a><span class="in">  data.frame() %&gt;%</span></span>
<span id="cb9-764"><a href="#cb9-764" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(date_time = as.POSIXct(date_time, format = "%m/%d/%Y %H:%M"),</span></span>
<span id="cb9-765"><a href="#cb9-765" aria-hidden="true" tabindex="-1"></a><span class="in">                date      = as.POSIXct(date_time, format = "%m/%d/%Y"),</span></span>
<span id="cb9-766"><a href="#cb9-766" aria-hidden="true" tabindex="-1"></a><span class="in">                time      = strftime(date_time, format = "%H:%M:%S"),</span></span>
<span id="cb9-767"><a href="#cb9-767" aria-hidden="true" tabindex="-1"></a><span class="in">                jul_day   = as.POSIXlt(date_time)$yday,</span></span>
<span id="cb9-768"><a href="#cb9-768" aria-hidden="true" tabindex="-1"></a><span class="in">                month     = format(as.Date(date_time, format = "%m/%d/%Y"),"%m"),</span></span>
<span id="cb9-769"><a href="#cb9-769" aria-hidden="true" tabindex="-1"></a><span class="in">                month_cat = format(as.Date(date_time, format = "%m/%d/%Y"),"%B"),</span></span>
<span id="cb9-770"><a href="#cb9-770" aria-hidden="true" tabindex="-1"></a><span class="in">                location = factor(location, levels = c("EXT", "deep"))) </span></span>
<span id="cb9-771"><a href="#cb9-771" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-772"><a href="#cb9-772" aria-hidden="true" tabindex="-1"></a><span class="in">temp_plot &lt;- ggplot() +</span></span>
<span id="cb9-773"><a href="#cb9-773" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = HOBO_dat %&gt;% dplyr::filter(location == "EXT"), </span></span>
<span id="cb9-774"><a href="#cb9-774" aria-hidden="true" tabindex="-1"></a><span class="in">            aes(x = jul_day, y = air_temp), size = 0.5, colour = "#f595a6") +</span></span>
<span id="cb9-775"><a href="#cb9-775" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = HOBO_dat %&gt;% dplyr::filter(location == "deep"), </span></span>
<span id="cb9-776"><a href="#cb9-776" aria-hidden="true" tabindex="-1"></a><span class="in">            aes(x = jul_day, y = air_temp), size = 1.2, colour = "black") +</span></span>
<span id="cb9-777"><a href="#cb9-777" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab("Air temperature (°C)") + xlab(NULL) +</span></span>
<span id="cb9-778"><a href="#cb9-778" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme() +</span></span>
<span id="cb9-779"><a href="#cb9-779" aria-hidden="true" tabindex="-1"></a><span class="in">  theme(legend.position = "bottom") +</span></span>
<span id="cb9-780"><a href="#cb9-780" aria-hidden="true" tabindex="-1"></a><span class="in">  facet_wrap(~ site, ncol = 1)</span></span>
<span id="cb9-781"><a href="#cb9-781" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-782"><a href="#cb9-782" aria-hidden="true" tabindex="-1"></a><span class="in">cowplot::plot_grid(map_plot, temp_plot, ncol = 2, rel_widths = c(1, 0.5))</span></span>
<span id="cb9-783"><a href="#cb9-783" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-784"><a href="#cb9-784" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-785"><a href="#cb9-785" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-786"><a href="#cb9-786" aria-hidden="true" tabindex="-1"></a><span class="fu">## Figure 2 - *T*~sk~, TMR, and TEWL</span></span>
<span id="cb9-787"><a href="#cb9-787" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-788"><a href="#cb9-788" aria-hidden="true" tabindex="-1"></a><span class="in">```{r Fig 2, fig.align='center', fig.height=6, fig.width=8, fig.show='hide'}</span></span>
<span id="cb9-789"><a href="#cb9-789" aria-hidden="true" tabindex="-1"></a><span class="in">Tsk_me &lt;- as.data.frame(ggeffects::ggpredict(Tsk_mod, terms = c("temp_exp", "pop", "tor_start [5]", "lnMass [2.7]")))</span></span>
<span id="cb9-790"><a href="#cb9-790" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-791"><a href="#cb9-791" aria-hidden="true" tabindex="-1"></a><span class="in">Tsk_plot &lt;- ggplot() +</span></span>
<span id="cb9-792"><a href="#cb9-792" aria-hidden="true" tabindex="-1"></a><span class="in">  # raw data</span></span>
<span id="cb9-793"><a href="#cb9-793" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = torpor_dat, aes(x = temp_exp, y = Tsk_C, group = ID), </span></span>
<span id="cb9-794"><a href="#cb9-794" aria-hidden="true" tabindex="-1"></a><span class="in">            colour = "lightgrey",</span></span>
<span id="cb9-795"><a href="#cb9-795" aria-hidden="true" tabindex="-1"></a><span class="in">            alpha = 0.5,</span></span>
<span id="cb9-796"><a href="#cb9-796" aria-hidden="true" tabindex="-1"></a><span class="in">            position = position_dodge(0.2)) +</span></span>
<span id="cb9-797"><a href="#cb9-797" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = torpor_dat, </span></span>
<span id="cb9-798"><a href="#cb9-798" aria-hidden="true" tabindex="-1"></a><span class="in">             aes(x = temp_exp, y = Tsk_C, shape = pop), </span></span>
<span id="cb9-799"><a href="#cb9-799" aria-hidden="true" tabindex="-1"></a><span class="in">             size = 1, position = position_dodge(0.2), alpha = 0.5) +</span></span>
<span id="cb9-800"><a href="#cb9-800" aria-hidden="true" tabindex="-1"></a><span class="in">  # summary data</span></span>
<span id="cb9-801"><a href="#cb9-801" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_errorbar(data = Tsk_me, aes(x = x, ymin = conf.low, ymax = conf.high, group = group), </span></span>
<span id="cb9-802"><a href="#cb9-802" aria-hidden="true" tabindex="-1"></a><span class="in">                width = 0.1, position = position_dodge(0.2)) +</span></span>
<span id="cb9-803"><a href="#cb9-803" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = Tsk_me, aes(x = x, y = predicted, group = group, linetype = group), position = position_dodge(0.2)) +</span></span>
<span id="cb9-804"><a href="#cb9-804" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = Tsk_me, aes(x = x, y = predicted, shape = group, fill = group), size = 3, position = position_dodge(0.2)) +</span></span>
<span id="cb9-805"><a href="#cb9-805" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_shape_manual(values = c(21, 24)) +</span></span>
<span id="cb9-806"><a href="#cb9-806" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values = c("white", "black")) +</span></span>
<span id="cb9-807"><a href="#cb9-807" aria-hidden="true" tabindex="-1"></a><span class="in">  xlab(expression(italic("T")["a"]~"(°C)")) +</span></span>
<span id="cb9-808"><a href="#cb9-808" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab(expression(italic("T")["sk"]~"(°C)")) +</span></span>
<span id="cb9-809"><a href="#cb9-809" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme()</span></span>
<span id="cb9-810"><a href="#cb9-810" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-811"><a href="#cb9-811" aria-hidden="true" tabindex="-1"></a><span class="in">MR_me &lt;- as.data.frame(ggeffects::ggpredict(MR_mod, terms = c("temp_exp", "pop", "tor_start [5]", "lnMass [2.7]"))) %&gt;%</span></span>
<span id="cb9-812"><a href="#cb9-812" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(est = exp(predicted),</span></span>
<span id="cb9-813"><a href="#cb9-813" aria-hidden="true" tabindex="-1"></a><span class="in">                low = exp(conf.low),</span></span>
<span id="cb9-814"><a href="#cb9-814" aria-hidden="true" tabindex="-1"></a><span class="in">                high = exp(conf.high))</span></span>
<span id="cb9-815"><a href="#cb9-815" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-816"><a href="#cb9-816" aria-hidden="true" tabindex="-1"></a><span class="in">MR_plot &lt;- ggplot() +</span></span>
<span id="cb9-817"><a href="#cb9-817" aria-hidden="true" tabindex="-1"></a><span class="in">  # raw data</span></span>
<span id="cb9-818"><a href="#cb9-818" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = torpor_dat, aes(x = temp_exp, y = MR_J_h, group = ID), </span></span>
<span id="cb9-819"><a href="#cb9-819" aria-hidden="true" tabindex="-1"></a><span class="in">            colour = "lightgrey",</span></span>
<span id="cb9-820"><a href="#cb9-820" aria-hidden="true" tabindex="-1"></a><span class="in">            alpha = 0.5,</span></span>
<span id="cb9-821"><a href="#cb9-821" aria-hidden="true" tabindex="-1"></a><span class="in">            position = position_dodge(0.2)) +</span></span>
<span id="cb9-822"><a href="#cb9-822" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = torpor_dat, </span></span>
<span id="cb9-823"><a href="#cb9-823" aria-hidden="true" tabindex="-1"></a><span class="in">             aes(x = temp_exp, y = MR_J_h, shape = pop), </span></span>
<span id="cb9-824"><a href="#cb9-824" aria-hidden="true" tabindex="-1"></a><span class="in">             size = 1, position = position_dodge(0.2), alpha = 0.5) +</span></span>
<span id="cb9-825"><a href="#cb9-825" aria-hidden="true" tabindex="-1"></a><span class="in">  # summary data</span></span>
<span id="cb9-826"><a href="#cb9-826" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_errorbar(data = MR_me, aes(x = x, ymin = low, ymax = high, group = group), </span></span>
<span id="cb9-827"><a href="#cb9-827" aria-hidden="true" tabindex="-1"></a><span class="in">                width = 0.1, position = position_dodge(0.2)) +</span></span>
<span id="cb9-828"><a href="#cb9-828" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = MR_me, aes(x = x, y = est, group = group, linetype = group), position = position_dodge(0.2)) +</span></span>
<span id="cb9-829"><a href="#cb9-829" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = MR_me, aes(x = x, y = est, shape = group, fill = group), size = 3, position = position_dodge(0.2)) +</span></span>
<span id="cb9-830"><a href="#cb9-830" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_shape_manual(values = c(21, 24)) +</span></span>
<span id="cb9-831"><a href="#cb9-831" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values = c("white", "black")) +</span></span>
<span id="cb9-832"><a href="#cb9-832" aria-hidden="true" tabindex="-1"></a><span class="in">  xlab(expression(italic("T")["a"]~"(°C)")) +</span></span>
<span id="cb9-833"><a href="#cb9-833" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab(expression("TMR (J h"^"-1"*")")) +</span></span>
<span id="cb9-834"><a href="#cb9-834" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_y_continuous(breaks = c(1, 10, 25, 50, 100, 200, 400, 800), trans = "sqrt") +</span></span>
<span id="cb9-835"><a href="#cb9-835" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme()</span></span>
<span id="cb9-836"><a href="#cb9-836" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-837"><a href="#cb9-837" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_me &lt;- as.data.frame(ggeffects::ggpredict(EWL_mod, terms = c("temp_exp", "pop", "tor_start [5]", "lnMass [2.7]"))) %&gt;%</span></span>
<span id="cb9-838"><a href="#cb9-838" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(est = exp(predicted),</span></span>
<span id="cb9-839"><a href="#cb9-839" aria-hidden="true" tabindex="-1"></a><span class="in">                low = exp(conf.low),</span></span>
<span id="cb9-840"><a href="#cb9-840" aria-hidden="true" tabindex="-1"></a><span class="in">                high = exp(conf.high))</span></span>
<span id="cb9-841"><a href="#cb9-841" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-842"><a href="#cb9-842" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_plot &lt;- ggplot() +</span></span>
<span id="cb9-843"><a href="#cb9-843" aria-hidden="true" tabindex="-1"></a><span class="in">  # raw data</span></span>
<span id="cb9-844"><a href="#cb9-844" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = torpor_dat, aes(x = temp_exp, y = EWL_mg_h, group = ID), </span></span>
<span id="cb9-845"><a href="#cb9-845" aria-hidden="true" tabindex="-1"></a><span class="in">            colour = "lightgrey",</span></span>
<span id="cb9-846"><a href="#cb9-846" aria-hidden="true" tabindex="-1"></a><span class="in">            alpha = 0.5,</span></span>
<span id="cb9-847"><a href="#cb9-847" aria-hidden="true" tabindex="-1"></a><span class="in">            position = position_dodge(0.2)) +</span></span>
<span id="cb9-848"><a href="#cb9-848" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = torpor_dat, </span></span>
<span id="cb9-849"><a href="#cb9-849" aria-hidden="true" tabindex="-1"></a><span class="in">             aes(x = temp_exp, y = EWL_mg_h, shape = pop), </span></span>
<span id="cb9-850"><a href="#cb9-850" aria-hidden="true" tabindex="-1"></a><span class="in">             size = 1, position = position_dodge(0.2), alpha = 0.5) +</span></span>
<span id="cb9-851"><a href="#cb9-851" aria-hidden="true" tabindex="-1"></a><span class="in">  # summary data</span></span>
<span id="cb9-852"><a href="#cb9-852" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_errorbar(data = EWL_me, aes(x = x, ymin = low, ymax = high, group = group), </span></span>
<span id="cb9-853"><a href="#cb9-853" aria-hidden="true" tabindex="-1"></a><span class="in">                width = 0.1, position = position_dodge(0.2)) +</span></span>
<span id="cb9-854"><a href="#cb9-854" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = EWL_me, aes(x = x, y = est, group = group, linetype = group), position = position_dodge(0.2)) +</span></span>
<span id="cb9-855"><a href="#cb9-855" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = EWL_me, aes(x = x, y = est, shape = group, fill = group), size = 3, position = position_dodge(0.2)) +</span></span>
<span id="cb9-856"><a href="#cb9-856" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_shape_manual(values = c(21, 24)) +</span></span>
<span id="cb9-857"><a href="#cb9-857" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values = c("white", "black")) +</span></span>
<span id="cb9-858"><a href="#cb9-858" aria-hidden="true" tabindex="-1"></a><span class="in">  xlab(expression(italic("T")["a"]~"(°C)")) +</span></span>
<span id="cb9-859"><a href="#cb9-859" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab(expression("TEWL (mg H"["2"]*"O"~"h"^"-1"*")")) +</span></span>
<span id="cb9-860"><a href="#cb9-860" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_y_continuous(breaks = c(1, 2.5, 5, 10, 20, 30), trans = "sqrt") +</span></span>
<span id="cb9-861"><a href="#cb9-861" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme()</span></span>
<span id="cb9-862"><a href="#cb9-862" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-863"><a href="#cb9-863" aria-hidden="true" tabindex="-1"></a><span class="in">MR_me2 &lt;- as.data.frame(ggeffects::ggpredict(MR_mod2, terms = c("Tsk_C", "pop", "tor_start [5]", "lnMass [2.7]"))) %&gt;%</span></span>
<span id="cb9-864"><a href="#cb9-864" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::mutate(est = exp(predicted),</span></span>
<span id="cb9-865"><a href="#cb9-865" aria-hidden="true" tabindex="-1"></a><span class="in">                low = exp(conf.low),</span></span>
<span id="cb9-866"><a href="#cb9-866" aria-hidden="true" tabindex="-1"></a><span class="in">                high = exp(conf.high))</span></span>
<span id="cb9-867"><a href="#cb9-867" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-868"><a href="#cb9-868" aria-hidden="true" tabindex="-1"></a><span class="in">MR_plot2 &lt;- ggplot() +</span></span>
<span id="cb9-869"><a href="#cb9-869" aria-hidden="true" tabindex="-1"></a><span class="in">  # raw data</span></span>
<span id="cb9-870"><a href="#cb9-870" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = torpor_dat, aes(x = Tsk_C, y = MR_J_h, group = ID), </span></span>
<span id="cb9-871"><a href="#cb9-871" aria-hidden="true" tabindex="-1"></a><span class="in">            colour = "lightgrey",</span></span>
<span id="cb9-872"><a href="#cb9-872" aria-hidden="true" tabindex="-1"></a><span class="in">            alpha = 0.5,</span></span>
<span id="cb9-873"><a href="#cb9-873" aria-hidden="true" tabindex="-1"></a><span class="in">            position = position_dodge(0.2)) +</span></span>
<span id="cb9-874"><a href="#cb9-874" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_point(data = torpor_dat, </span></span>
<span id="cb9-875"><a href="#cb9-875" aria-hidden="true" tabindex="-1"></a><span class="in">             aes(x = Tsk_C, y = MR_J_h, shape = pop), </span></span>
<span id="cb9-876"><a href="#cb9-876" aria-hidden="true" tabindex="-1"></a><span class="in">             size = 1, position = position_dodge(0.2), alpha = 0.5) +</span></span>
<span id="cb9-877"><a href="#cb9-877" aria-hidden="true" tabindex="-1"></a><span class="in">  # summary data</span></span>
<span id="cb9-878"><a href="#cb9-878" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_ribbon(data = MR_me2, aes(x = x, ymin = low, ymax = high,  group = group), alpha = 0.1) +</span></span>
<span id="cb9-879"><a href="#cb9-879" aria-hidden="true" tabindex="-1"></a><span class="in">  geom_line(data = MR_me2, aes(x = x, y = est, group = group, linetype = group)) +</span></span>
<span id="cb9-880"><a href="#cb9-880" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_shape_manual(values = c(21, 24)) +</span></span>
<span id="cb9-881"><a href="#cb9-881" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values = c("white", "black")) +</span></span>
<span id="cb9-882"><a href="#cb9-882" aria-hidden="true" tabindex="-1"></a><span class="in">  xlab(expression(italic("T")["sk"]~"(°C)")) +</span></span>
<span id="cb9-883"><a href="#cb9-883" aria-hidden="true" tabindex="-1"></a><span class="in">  ylab(expression("MR (J h"^"-1"*")")) +</span></span>
<span id="cb9-884"><a href="#cb9-884" aria-hidden="true" tabindex="-1"></a><span class="in">  mytheme()</span></span>
<span id="cb9-885"><a href="#cb9-885" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-886"><a href="#cb9-886" aria-hidden="true" tabindex="-1"></a><span class="in">cowplot::plot_grid(MR_plot + theme(legend.position = "none"), </span></span>
<span id="cb9-887"><a href="#cb9-887" aria-hidden="true" tabindex="-1"></a><span class="in">                   EWL_plot + theme(legend.position = "none"), </span></span>
<span id="cb9-888"><a href="#cb9-888" aria-hidden="true" tabindex="-1"></a><span class="in">                   Tsk_plot + theme(legend.position = "none"), </span></span>
<span id="cb9-889"><a href="#cb9-889" aria-hidden="true" tabindex="-1"></a><span class="in">                   MR_plot2 + theme(legend.position = "none"), </span></span>
<span id="cb9-890"><a href="#cb9-890" aria-hidden="true" tabindex="-1"></a><span class="in">                   nrow = 2, labels = c('a', 'b', 'c', 'd'))</span></span>
<span id="cb9-891"><a href="#cb9-891" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-892"><a href="#cb9-892" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-893"><a href="#cb9-893" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-894"><a href="#cb9-894" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-895"><a href="#cb9-895" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-896"><a href="#cb9-896" aria-hidden="true" tabindex="-1"></a><span class="fu">## Figure 3 - Phylogeny</span></span>
<span id="cb9-897"><a href="#cb9-897" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-898"><a href="#cb9-898" aria-hidden="true" tabindex="-1"></a><span class="in">```{r Fig 3, fig.align='center', fig.height=5, fig.width=8, fig.show='hide'}</span></span>
<span id="cb9-899"><a href="#cb9-899" aria-hidden="true" tabindex="-1"></a><span class="in">MR_mod_post  &lt;- posterior_samples(MR_mod) # extract posterior</span></span>
<span id="cb9-900"><a href="#cb9-900" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_mod_post &lt;- posterior_samples(EWL_mod) # extract posterior</span></span>
<span id="cb9-901"><a href="#cb9-901" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-902"><a href="#cb9-902" aria-hidden="true" tabindex="-1"></a><span class="in"># Loop to extract within ID-level posterior of intercept</span></span>
<span id="cb9-903"><a href="#cb9-903" aria-hidden="true" tabindex="-1"></a><span class="in">ID_unique  &lt;- sort(unique(torpor_dat$ID)) # pop ID names</span></span>
<span id="cb9-904"><a href="#cb9-904" aria-hidden="true" tabindex="-1"></a><span class="in">out_list   &lt;- vector(mode = 'list', length = length(ID_unique))</span></span>
<span id="cb9-905"><a href="#cb9-905" aria-hidden="true" tabindex="-1"></a><span class="in">for (j in seq_along(ID_unique)) {</span></span>
<span id="cb9-906"><a href="#cb9-906" aria-hidden="true" tabindex="-1"></a><span class="in">  out_list[[j]]                 &lt;- data.frame(matrix(NA, nrow(MR_mod_post), 1))</span></span>
<span id="cb9-907"><a href="#cb9-907" aria-hidden="true" tabindex="-1"></a><span class="in">  dat_names                     &lt;-  paste0(ID_unique[j])</span></span>
<span id="cb9-908"><a href="#cb9-908" aria-hidden="true" tabindex="-1"></a><span class="in">  names(out_list[[j]])          &lt;-  dat_names</span></span>
<span id="cb9-909"><a href="#cb9-909" aria-hidden="true" tabindex="-1"></a><span class="in">  out_list[[j]][, dat_names[1]] &lt;-  MR_mod_post[, paste0('r_ID[', ID_unique[j], ',Intercept]')]</span></span>
<span id="cb9-910"><a href="#cb9-910" aria-hidden="true" tabindex="-1"></a><span class="in">}</span></span>
<span id="cb9-911"><a href="#cb9-911" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-912"><a href="#cb9-912" aria-hidden="true" tabindex="-1"></a><span class="in">MR_Int_out  &lt;- do.call('cbind.data.frame', out_list)</span></span>
<span id="cb9-913"><a href="#cb9-913" aria-hidden="true" tabindex="-1"></a><span class="in">MR_Int_post &lt;- sample_n(MR_Int_out, 4000) # Extract 4000 random rows</span></span>
<span id="cb9-914"><a href="#cb9-914" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-915"><a href="#cb9-915" aria-hidden="true" tabindex="-1"></a><span class="in">out_list &lt;- vector(mode = 'list', length = length(ID_unique))</span></span>
<span id="cb9-916"><a href="#cb9-916" aria-hidden="true" tabindex="-1"></a><span class="in">for (j in seq_along(ID_unique)) {</span></span>
<span id="cb9-917"><a href="#cb9-917" aria-hidden="true" tabindex="-1"></a><span class="in">  out_list[[j]]                 &lt;- data.frame(matrix(NA, nrow(EWL_mod_post), 1))</span></span>
<span id="cb9-918"><a href="#cb9-918" aria-hidden="true" tabindex="-1"></a><span class="in">  dat_names                     &lt;-  paste0(ID_unique[j])</span></span>
<span id="cb9-919"><a href="#cb9-919" aria-hidden="true" tabindex="-1"></a><span class="in">  names(out_list[[j]])          &lt;-  dat_names</span></span>
<span id="cb9-920"><a href="#cb9-920" aria-hidden="true" tabindex="-1"></a><span class="in">  out_list[[j]][, dat_names[1]] &lt;-  EWL_mod_post[, paste0('r_ID[', ID_unique[j], ',Intercept]')]</span></span>
<span id="cb9-921"><a href="#cb9-921" aria-hidden="true" tabindex="-1"></a><span class="in">}</span></span>
<span id="cb9-922"><a href="#cb9-922" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-923"><a href="#cb9-923" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_Int_out  &lt;- do.call('cbind.data.frame', out_list)</span></span>
<span id="cb9-924"><a href="#cb9-924" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_Int_post &lt;- sample_n(EWL_Int_out, 4000) # Extract 4000 random rows</span></span>
<span id="cb9-925"><a href="#cb9-925" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-926"><a href="#cb9-926" aria-hidden="true" tabindex="-1"></a><span class="in"># Tidy data</span></span>
<span id="cb9-927"><a href="#cb9-927" aria-hidden="true" tabindex="-1"></a><span class="in">MR_Int_post_long &lt;- MR_Int_post %&gt;%</span></span>
<span id="cb9-928"><a href="#cb9-928" aria-hidden="true" tabindex="-1"></a><span class="in">  tidyr::pivot_longer(cols = everything(), names_to = "ID", values_to = "Intercept")</span></span>
<span id="cb9-929"><a href="#cb9-929" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-930"><a href="#cb9-930" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_Int_post_long &lt;- EWL_Int_post %&gt;%</span></span>
<span id="cb9-931"><a href="#cb9-931" aria-hidden="true" tabindex="-1"></a><span class="in">  tidyr::pivot_longer(cols = everything(), names_to = "ID", values_to = "Intercept")</span></span>
<span id="cb9-932"><a href="#cb9-932" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-933"><a href="#cb9-933" aria-hidden="true" tabindex="-1"></a><span class="in">MR_Int_post_sum &lt;- MR_Int_post_long %&gt;%</span></span>
<span id="cb9-934"><a href="#cb9-934" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::group_by(ID) %&gt;%</span></span>
<span id="cb9-935"><a href="#cb9-935" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::summarise(mean_MR = mean(Intercept))</span></span>
<span id="cb9-936"><a href="#cb9-936" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-937"><a href="#cb9-937" aria-hidden="true" tabindex="-1"></a><span class="in">EWL_Int_post_sum &lt;- EWL_Int_post_long %&gt;%</span></span>
<span id="cb9-938"><a href="#cb9-938" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::group_by(ID) %&gt;%</span></span>
<span id="cb9-939"><a href="#cb9-939" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::summarise(mean_EWL = mean(Intercept))</span></span>
<span id="cb9-940"><a href="#cb9-940" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-941"><a href="#cb9-941" aria-hidden="true" tabindex="-1"></a><span class="in">phylo_dat &lt;- torpor_dat %&gt;%</span></span>
<span id="cb9-942"><a href="#cb9-942" aria-hidden="true" tabindex="-1"></a><span class="in">  dplyr::group_by(ID_gene, ID, sex, pop) %&gt;%</span></span>
<span id="cb9-943"><a href="#cb9-943" aria-hidden="true" tabindex="-1"></a><span class="in">  summarise(mean_mass = mean(mass_g))</span></span>
<span id="cb9-944"><a href="#cb9-944" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-945"><a href="#cb9-945" aria-hidden="true" tabindex="-1"></a><span class="in">x &lt;- dplyr::as_tibble(pruned_tree_cal)</span></span>
<span id="cb9-946"><a href="#cb9-946" aria-hidden="true" tabindex="-1"></a><span class="in">y &lt;- dplyr::full_join(phylo_dat, MR_Int_post_sum, by = 'ID')</span></span>
<span id="cb9-947"><a href="#cb9-947" aria-hidden="true" tabindex="-1"></a><span class="in">y &lt;- dplyr::full_join(y, EWL_Int_post_sum, by = 'ID')</span></span>
<span id="cb9-948"><a href="#cb9-948" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-949"><a href="#cb9-949" aria-hidden="true" tabindex="-1"></a><span class="in">tree_dat  &lt;- tidytree::as.treedata(dplyr::full_join(x, y %&gt;% rename(label = ID_gene), by = 'label'))</span></span>
<span id="cb9-950"><a href="#cb9-950" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-951"><a href="#cb9-951" aria-hidden="true" tabindex="-1"></a><span class="in">tree_plot &lt;- ggtree::ggtree(tree_dat) + </span></span>
<span id="cb9-952"><a href="#cb9-952" aria-hidden="true" tabindex="-1"></a><span class="in">  ggtree::geom_tippoint(aes(shape = pop, fill = pop), size = 2, show.legend = F) +</span></span>
<span id="cb9-953"><a href="#cb9-953" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_shape_manual(values = c(21, 24)) +</span></span>
<span id="cb9-954"><a href="#cb9-954" aria-hidden="true" tabindex="-1"></a><span class="in">  scale_fill_manual(values = c("white", "black")) </span></span>
<span id="cb9-955"><a href="#cb9-955" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-956"><a href="#cb9-956" aria-hidden="true" tabindex="-1"></a><span class="in">tree_plot2 &lt;- tree_plot + ggtreeExtra::geom_fruit(geom = geom_bar,</span></span>
<span id="cb9-957"><a href="#cb9-957" aria-hidden="true" tabindex="-1"></a><span class="in">                                    mapping = aes(x = mean_MR, fill = pop),</span></span>
<span id="cb9-958"><a href="#cb9-958" aria-hidden="true" tabindex="-1"></a><span class="in">                                    colour = "black",</span></span>
<span id="cb9-959"><a href="#cb9-959" aria-hidden="true" tabindex="-1"></a><span class="in">                                    size = 0.1,</span></span>
<span id="cb9-960"><a href="#cb9-960" aria-hidden="true" tabindex="-1"></a><span class="in">                                    stat = "identity", </span></span>
<span id="cb9-961"><a href="#cb9-961" aria-hidden="true" tabindex="-1"></a><span class="in">                                    orientation = "y", </span></span>
<span id="cb9-962"><a href="#cb9-962" aria-hidden="true" tabindex="-1"></a><span class="in">                                    axis.params = list(axis = "x", text.size = 3), </span></span>
<span id="cb9-963"><a href="#cb9-963" aria-hidden="true" tabindex="-1"></a><span class="in">                                    offset = 0.3, </span></span>
<span id="cb9-964"><a href="#cb9-964" aria-hidden="true" tabindex="-1"></a><span class="in">                                    pwidth = 0.4,</span></span>
<span id="cb9-965"><a href="#cb9-965" aria-hidden="true" tabindex="-1"></a><span class="in">                                    show.legend = F)</span></span>
<span id="cb9-966"><a href="#cb9-966" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-967"><a href="#cb9-967" aria-hidden="true" tabindex="-1"></a><span class="in">tree_plot2 + ggtreeExtra::geom_fruit(geom = geom_bar,</span></span>
<span id="cb9-968"><a href="#cb9-968" aria-hidden="true" tabindex="-1"></a><span class="in">                                     mapping = aes(x = mean_EWL, fill = pop),</span></span>
<span id="cb9-969"><a href="#cb9-969" aria-hidden="true" tabindex="-1"></a><span class="in">                                     colour = "black",</span></span>
<span id="cb9-970"><a href="#cb9-970" aria-hidden="true" tabindex="-1"></a><span class="in">                                     size = 0.1,</span></span>
<span id="cb9-971"><a href="#cb9-971" aria-hidden="true" tabindex="-1"></a><span class="in">                                     stat = "identity", </span></span>
<span id="cb9-972"><a href="#cb9-972" aria-hidden="true" tabindex="-1"></a><span class="in">                                     orientation = "y", </span></span>
<span id="cb9-973"><a href="#cb9-973" aria-hidden="true" tabindex="-1"></a><span class="in">                                     axis.params = list(axis = "x", text.size = 3), </span></span>
<span id="cb9-974"><a href="#cb9-974" aria-hidden="true" tabindex="-1"></a><span class="in">                                     offset = 0.3, </span></span>
<span id="cb9-975"><a href="#cb9-975" aria-hidden="true" tabindex="-1"></a><span class="in">                                     pwidth = 0.4,</span></span>
<span id="cb9-976"><a href="#cb9-976" aria-hidden="true" tabindex="-1"></a><span class="in">                                     show.legend = F)</span></span>
<span id="cb9-977"><a href="#cb9-977" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb9-978"><a href="#cb9-978" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-979"><a href="#cb9-979" aria-hidden="true" tabindex="-1"></a>***</span>
<span id="cb9-980"><a href="#cb9-980" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-981"><a href="#cb9-981" aria-hidden="true" tabindex="-1"></a><span class="fu"># References {-}</span></span>
<span id="cb9-982"><a href="#cb9-982" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-983"><a href="#cb9-983" aria-hidden="true" tabindex="-1"></a>&lt;div id="refs"&gt;&lt;/div&gt;</span>
<span id="cb9-984"><a href="#cb9-984" aria-hidden="true" tabindex="-1"></a>&lt;br&gt;</span>
<span id="cb9-985"><a href="#cb9-985" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-986"><a href="#cb9-986" aria-hidden="true" tabindex="-1"></a>***</span>
<span id="cb9-987"><a href="#cb9-987" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-988"><a href="#cb9-988" aria-hidden="true" tabindex="-1"></a><span class="fu">## Session Information</span></span>
<span id="cb9-989"><a href="#cb9-989" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-990"><a href="#cb9-990" aria-hidden="true" tabindex="-1"></a><span class="in">```{r sessioninfo, echo = FALSE}</span></span>
<span id="cb9-991"><a href="#cb9-991" aria-hidden="true" tabindex="-1"></a><span class="in">pander::pander(sessionInfo(), locale = FALSE)</span></span>
<span id="cb9-992"><a href="#cb9-992" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span></code><button title="Copy to Clipboard" class="code-copy-button" data-in-quarto-modal=""><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->




</body></html>